const DATA = {
  data: [
    ['Database Identifiers', '', '', '', '', '', '', '', '', 'Sample collection and processing', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', 'Host information', '', '', '', '', '', '', '', '', '', '', 'Host exposure information', '', '', 'Sequencing', '', '', '', '', '', '', 'Bioinformatics and QC metrics', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', 'Pathogen diagnostic testing', '', '', '', '', '', 'Contributor acknowledgement'],
    ['specimen collector sample ID', 'PHAC sample ID', 'IRIDA sample name', 'umbrella bioproject accession', 'bioproject accession', 'biosample accession', 'SRA accession', 'GenBank accession', 'GISAID accession', 'sample collected by', 'sample collector contact email', 'sample collector contact address', 'sequence submitted by', 'sequence submitter contact email', 'sequence submitter contact address', 'sample collection date', 'geo_loc_name (country)', 'geo_loc_name (province/territory)', 'geo_loc_name (city)', 'organism', 'isolate', 'purpose of sampling', 'anatomical material', 'anatomical part', 'body product', 'environmental material', 'environmental site', 'collection device', 'collection method', 'collection protocol', 'specimen processing', 'lab host', 'passage number', 'passage method', 'biomaterial extracted', 'host (common name)', 'host (scientific name)', 'host health state', 'host health status details', 'host disease', 'host age', 'host gender', 'host origin geo_loc name (country)', 'host subject ID', 'symptom onset date', 'signs and symptoms', 'location of exposure geo_loc name (country)', 'travel history', 'exposure event', 'library ID', 'sequencing instrument', 'sequencing protocol name', 'sequencing protocol source', 'sequencing kit number', 'amplicon pcr primers filename', 'raw sequence data processing', 'sequencing depth (average)', 'assembly name', 'assembly method', 'assembly coverage breadth', 'assembly coverage depth', 'r1 fastq filename', 'r2 fastq filename', 'fasta filename', 'number base pairs', 'genome length', 'mean contig length', 'N50', 'Ns per 100 kbp', 'reference genome accession', 'consensus sequence ID', 'consensus sequence method', 'annotation feature table filename', 'bioinformatics protocol', 'gene name 1', 'diagnostic pcr protocol 1', 'diagnostic pcr Ct value 1', 'gene name 2', 'diagnostic pcr protocol 2', 'diagnostic pcr Ct value 2', 'authors'],
  ],
  columns: [
    {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {},
  ],
};