<!doctype html>

<html lang="en">
<head>
  <meta charset="utf-8">
  <title>Guidelines for Biosample Metadata Collection and Submission</title>
  <meta name="description" content="Guidelines for submitting metadata regarding biosamples and their associated cases to sequence repositories.">
  <meta name="author" content="Hsiao Public Health Bioinformatics, Vancouver, BC, Canada">
  <link rel="stylesheet" href="./doc.css">

</head>

<body>
	
	<h2>Guidelines for Biosample metadata collection and submission</h2>
	<hr size="2"/>
	<p>Guidelines for submitting metadata regarding biosamples and their associated cases to sequence repositories.</p>

	<table>
		<thead>
			<tr>
				<th class="label">Field</th>
				<th class="description">Description</th>
				<th class="guidance">Guidance</th>
				<th class="example">Examples</th>
				<th class="data_status">Data Status</th>
			</tr>
		</thead>
		<tbody>
			<!-- python search and replace does this part -->	
  			
						<tr class="section">
							<td colspan="5"><h3>Database Identifiers</h3></td>
						</tr>
						
						<tr>
							<td class="label">specimen collector sample ID</td>
							<td>The user-defined name for the sample.</td>
							<td>Store the collector sample ID. If this number is considered identifiable information, provide an alternative ID. Be sure to store the key that maps between the original and alternative IDs for traceability and follow up if necessary. Every collector sample ID from a single submitter must be unique. It can have any format, but we suggest that you make it concise, unique and consistent within your lab.</td>
							<td>prov_rona_99</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">umbrella bioproject accession</td>
							<td>The INSDC accession number assigned to the umbrella BioProject for the Canadian SARS-CoV-2 sequencing effort.</td>
							<td>Store the umbrella BioProject accession by selecting it from the picklist in the template. The umbrella BioProject accession will be identical for all CanCOGen submitters. Different provinces will have their own BioProjects, however these BioProjects will be linked under one umbrella BioProject.</td>
							<td>PRJNA623807</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">bioproject accession</td>
							<td>The INSDC accession number of the BioProject(s) to which the BioSample belongs.</td>
							<td>Store the BioProject accession number. BioProjects are an organizing tool that links together raw sequence data, assemblies, and their associated metadata. Each province will be assigned a different bioproject accession number by the National Microbiology Lab. A valid NCBI BioProject accession has prefix PRJN e.g., PRJNA12345, and is created once at the beginning of a new sequencing project. </td>
							<td>PRJNA12345</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">biosample accession</td>
							<td>The identifier assigned to a BioSample in INSDC archives.</td>
							<td>Store the accession returned from the BioSample submission. NCBI BioSamples will have the prefix SAMN.</td>
							<td>SAMN14180202</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">SRA accession</td>
							<td>The Sequence Read Archive (SRA) identifier linking raw read data, methodological metadata and quality control metrics submitted to the INSDC.</td>
							<td>Store the accession assigned to the submitted "run". NCBI-SRA accessions start with SRR.</td>
							<td>SRR11177792</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">GenBank/ENA/DDBJ accession</td>
							<td>The GenBank identifier assigned to the sequence in the INSDC archives.</td>
							<td>Store the accession returned from a GenBank submission (viral genome assembly).</td>
							<td>MN908947.3</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">GISAID accession</td>
							<td>The GISAID accession number assigned to the sequence.</td>
							<td>Store the accession returned from the GISAID submission.</td>
							<td>EPI_ISL_123456</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">GISAID virus name</td>
							<td>The user-defined GISAID virus name assigned to the sequence.</td>
							<td>GISAID virus names should be in the format "hCoV-19/Country/Identifier/year".</td>
							<td>hCoV-19/Canada/prov_rona_99/2020</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">host specimen voucher</td>
							<td>Identifier for the physical specimen.</td>
							<td>Include a URI (Uniform Resource Identifier) in the form of a URL providing a direct link to the physical host specimen. If the specimen was destroyed in the process of analysis, electronic images (e-vouchers) are an adequate substitute for a physical host voucher specimen. If a URI is not available, a museum-provided globally unique identifier (GUID) can be used. If no persistent unique identifier is available, follow the INSDC guidance for populating the voucher_specimen attribute using standard triplets for institution codes (i.e., /specimen_voucher="[<institution-code>:[<collection-code>:]]<specimen_id>"): http://www.insdc.org/controlled-vocabulary-specimenvoucher-qualifier</td>
							<td>URI example: http://portal.vertnet.org/o/fmnh/mammals?id=33e55cfe-330b-40d9-aaae-8d042cba7542, INSDC triplet example: UAM:Mamm:52179</td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Sample collection and processing</h3></td>
						</tr>
						
						<tr>
							<td class="label">sample collected by</td>
							<td>The name of the agency that collected the original sample.</td>
							<td>The name of the sample collector should be written out in full, (with minor exceptions) and be consistent across multple submissions e.g. Public Health Agency of Canada, Public Health Ontario, BC Centre for Disease Control. The sample collector specified is at the discretion of the data provider (i.e. may be hospital, provincial public health lab, or other).</td>
							<td>Public Health Agency of Canada</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">sample collector contact email</td>
							<td>The email address of the contact responsible for follow-up regarding the sample.</td>
							<td>The email address can represent a specific individual or lab e.g. johnnyblogs@lab.ca, or RespLab@lab.ca</td>
							<td>johnnyblogs@lab.ca</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sample collector contact address</td>
							<td>The mailing address of the agency submitting the sample.</td>
							<td>The mailing address should be in the format: Street number and name, City, Province/Territory, Postal Code, Country</td>
							<td>655 Lab St, Vancouver, British Columbia, V5N 2A2, Canada</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sequence submitted by</td>
							<td>The name of the agency that generated the sequence.</td>
							<td>The name of the agency should be written out in full, (with minor exceptions) and be consistent across multple submissions. If submitting specimens rather than sequencing data, please put the "National Microbiology Laboratory (NML)".</td>
							<td>Centers for Disease Control and Prevention</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">sequence submitter contact email</td>
							<td>The email address of the contact responsible for follow-up regarding the sequence.</td>
							<td>The email address can represent a specific individual or lab e.g. johnnyblogs@lab.ca, or RespLab@lab.ca</td>
							<td>RespLab@lab.ca</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sequence submitter contact address</td>
							<td>The mailing address of the agency submitting the sequence.</td>
							<td>The mailing address should be in the format: Street number and name, City, Province/Territory, Postal Code, Country</td>
							<td>123 Sunnybrooke St, Toronto, Ontario, M4P 1L6, Canada</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sample collection date</td>
							<td>The date on which the sample was collected.</td>
							<td>Sample collection date is critical for surveillance and many types of analyses. Required granularity includes year, month and day. If this date is considered identifiable information, it is acceptable to add "jitter" by adding or subtracting a calendar day (acceptable by GISAID). Alternatively, ”received date” may be used as a substitute. The date should be provided in ISO 8601 standard format "YYYY-MM-DD".</td>
							<td>2020-03-19</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">sample received date</td>
							<td>The date on which the sample was received.</td>
							<td>ISO 8601 standard "YYYY-MM-DD".</td>
							<td>2020-03-20</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">geo_loc_name (country)</td>
							<td>The country where the sample was collected.</td>
							<td>Provide the country name from the controlled vocabulary provided.</td>
							<td>South Africa</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">geo_loc_name (state/province/territory)</td>
							<td>The state/province/territory where the sample was collected.</td>
							<td>Provide the state/province/territory name from the GAZ geography ontology. Search for geography terms here: https://www.ebi.ac.uk/ols/ontologies/gaz</td>
							<td>Western Cape</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">geo_loc name (county/region)</td>
							<td>The county/region of origin of the sample.</td>
							<td>Provide the county/region name from the GAZ geography ontology. Search for geography terms here: https://www.ebi.ac.uk/ols/ontologies/gaz</td>
							<td>Derbyshire</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">geo_loc_name (city)</td>
							<td>The city where the sample was collected.</td>
							<td>Provide the city name. Use this look-up service to identify the standardized term: https://www.ebi.ac.uk/ols/ontologies/gaz</td>
							<td>Vancouver</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">geo_loc latitude</td>
							<td>The latitude coordinates of the geographical location of sample collection.</td>
							<td>Provide latitude coordinates if available. Do not use the centre of the city/region/province/state/country or the location of your agency as a proxy, as this implicates a real location and is misleading. Specify as degrees latitude in format "d[d.dddd] N|S".</td>
							<td>38.98 N</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">geo_loc longitude</td>
							<td>The longitude coordinates of the geographical location of sample collection.</td>
							<td>Provide longitude coordinates if available. Do not use the centre of the city/region/province/state/country or the location of your agency as a proxy, as this implicates a real location and is misleading. Specify as degrees longitude in format "d[dd.dddd] W|E".</td>
							<td>77.11 W</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">organism</td>
							<td>Taxonomic name of the organism.</td>
							<td>Use "Severe acute respiratory syndrome coronavirus 2". This value is provided in the template.</td>
							<td>Severe acute respiratory syndrome coronavirus 2</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">isolate</td>
							<td>Identifier of the specific isolate.</td>
							<td>This identifier should be an unique, indexed, alpha-numeric ID within your laboratory. If submitted to the INSDC, the "isolate" name is propagated throughtout different databases. As such, structure the "isolate" name to be ICTV/INSDC compliant in the following format: "SARS-CoV-2/host/country/sampleID/date".</td>
							<td>SARS-CoV-2/human/USA/CA-CDPH-001/2020</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">culture collection</td>
							<td>The name of the source collection and unique culture identifier. </td>
							<td>Format: "<institution-code>:[<collection-code>:]<culture_id>". For more information, see http://www.insdc.org/controlled-vocabulary-culturecollection-qualifier.</td>
							<td>/culture_collection="ATCC:26370"</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">purpose of sampling</td>
							<td>The reason that the sample was collected.</td>
							<td>The reason a sample was collected may provide information about potential biases in sampling strategy. Provide the purpose of sampling from the picklist in the template. Most likely, the sample was collected for Diagnostic testing. The reason why a sample was originally collected may differ from the reason why it was selected for sequencing, which should be indicated in the "purpose of sequencing" field. </td>
							<td>Diagnostic testing</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">purpose of sampling details</td>
							<td>The description of why the sample was collected, providing specific details.</td>
							<td>Provide an expanded description of why the sample was collected using free text. The description may include the importance of the sample for a particular public health investigation/surveillance activity/research question. If details are not available, provide a null value.</td>
							<td>Screening of bat specimens in museum collections.</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sample plan name</td>
							<td>The name of the sample plan implemented for sample collection.</td>
							<td>Provide the name and version of the sample plan outlining the sample strategy.</td>
							<td>CanCOGeN Sampling Strategy 1.0</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sample collected in quarantine</td>
							<td>Whether the sample was collected from an individual in quarantine.</td>
							<td>Whether a sample was collected under quarantine conditions (e.g. self-quarantining, medically isolated, staying at a quarantine hotel) can inform public health measure assessments. Use the picklist provided in the template.</td>
							<td>Yes</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">anatomical material</td>
							<td>A substance obtained from an anatomical part of an organism e.g. tissue, blood.</td>
							<td>Provide a descriptor if an anatomical material was sampled. Use the picklist provided in the template. If a desired term is missing from the picklist, contact emma.griffiths@bccdc.ca. If not applicable, do not leave blank. Choose a null value. </td>
							<td>Blood</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">anatomical part</td>
							<td>An anatomical part of an organism e.g. oropharynx.</td>
							<td>Provide a descriptor if an anatomical part was sampled. Use the picklist provided in the template. If a desired term is missing from the picklist, contact emma.griffiths@bccdc.ca. If not applicable, do not leave blank. Choose a null value. </td>
							<td>Nasopharynx</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">body product</td>
							<td>A substance excreted/secreted from an organism e.g. feces, urine, sweat.</td>
							<td>Provide a descriptor if a body product was sampled. Use the picklist provided in the template.  If a desired term is missing from the picklist, contact emma.griffiths@bccdc.ca. If not applicable, do not leave blank. Choose a null value. </td>
							<td>Feces</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">environmental material</td>
							<td>A substance obtained from the natural or man-made environment e.g. soil, water, sewage.</td>
							<td>Provide a descriptor if an environmental material was sampled. Use the picklist provided in the template.  If a desired term is missing from the picklist, contact emma.griffiths@bccdc.ca. If not applicable, do not leave blank. Choose a null value. </td>
							<td>Face mask</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">environmental site</td>
							<td>An environmental location may describe a site in the natural or built environment e.g. contact surface, metal can, hospital, wet market, bat cave.</td>
							<td>Provide a descriptor if an environmental site was sampled. Use the picklist provided in the template.  If a desired term is missing from the picklist, contact emma.griffiths@bccdc.ca. If not applicable, do not leave blank. Choose a null value. </td>
							<td>Hospital room</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">collection device</td>
							<td>The instrument or container used to collect the sample e.g. swab.</td>
							<td>Provide a descriptor if a device was used for sampling. Use the picklist provided in the template. If a desired term is missing from the picklist, contact emma.griffiths@bccdc.ca. If not applicable, do not leave blank. Choose a null value. </td>
							<td>Swab</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">collection method</td>
							<td>The process used to collect the sample e.g. phlebotamy, necropsy.</td>
							<td>Provide a descriptor if a collection method was used for sampling. Use the picklist provided in the template.  If a desired term is missing from the picklist, contact emma.griffiths@bccdc.ca. If not applicable, do not leave blank. Choose a null value. </td>
							<td>Bronchoalveolar lavage (BAL)</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">collection protocol</td>
							<td>The name and version of a particular protocol used for sampling.</td>
							<td>Free text.</td>
							<td>SC2SamplingProtocol 1.2</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">specimen processing</td>
							<td>Any processing applied to the sample during or after receiving the sample.</td>
							<td>Critical for interpreting data. Select all the applicable processes from the pick list. If virus was passaged, include information in "lab host", "passage number", and "passage method" fields. If none of the processes in the pick list apply, put "not applicable".</td>
							<td>Virus passage</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">lab host</td>
							<td>Name and description of the laboratory host used to propagate the source organism or material from which the sample was obtained.</td>
							<td>Type of cell line used for propagation. Provide the name of the cell line using the picklist in the template. If not passaged, put "not applicable".</td>
							<td>Vero E6 cell line</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">passage number</td>
							<td>Number of passages.</td>
							<td>Provide number of known passages. If not passaged, put "not applicable"</td>
							<td>3</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">passage method</td>
							<td>Description of how organism was passaged.</td>
							<td>Free text. Provide a very short description (<10 words). If not passaged, put "not applicable".</td>
							<td>AVL buffer+30%EtOH lysate received from Respiratory Lab. P3 passage in Vero-1 via bioreactor large-scale batch passage. P3 batch derived from the SP-2/reference lab strain.</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">biomaterial extracted</td>
							<td>The biomaterial extracted from samples for the purpose of sequencing.</td>
							<td>Provide the biomaterial extracted from the picklist in the template.</td>
							<td>RNA (total)</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">data abstraction details</td>
							<td>A description of how any data elements were altered to preserve patient privacy.</td>
							<td>If applicable, provide a description of how each data element was abstracted. </td>
							<td>Jitter added to publicly shared collection dates to prevent re-identifiability.</td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Host Information</h3></td>
						</tr>
						
						<tr>
							<td class="label">host (common name)</td>
							<td>The commonly used name of the host.</td>
							<td>Common name or scientific name are required if there was a host. Both can be provided, if known. Use terms from the pick lists in the template. Common name e.g. human, bat. If the sample was environmental, put "not applicable.</td>
							<td>Human</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">host (scientific name)</td>
							<td>The taxonomic, or scientific name of the host.</td>
							<td>Common name or scientific name are required if there was a host. Both can be provided, if known. Use terms from the pick lists in the template. Scientific name e.g. Homo sapiens, If the sample was environmental, put "not applicable</td>
							<td>Homo sapiens</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">host health state</td>
							<td>Health status of the host at the time of sample collection.</td>
							<td>If known, select a descriptor from the pick list provided in the template.</td>
							<td>Asymptomatic</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">host health status details</td>
							<td>Further details pertaining to the health or disease status of the host at time of collection.</td>
							<td>If known, select a descriptor from the pick list provided in the template.</td>
							<td>Hospitalized (ICU)</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">host disease</td>
							<td>The name of the disease experienced by the host.</td>
							<td>Select "COVID-19" from the pick list provided in the template.</td>
							<td>COVID-19</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">host health outcome</td>
							<td>Disease outcome in the host.</td>
							<td>If known, select a descriptor from the pick list provided in the template.</td>
							<td>Recovered</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">host age</td>
							<td>Age of host at the time of sampling.</td>
							<td>Enter the age of the host in years. If not available, provide a null value. If there is not host, put "Not Applicable".</td>
							<td>79</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">host age unit</td>
							<td>The unit used to measure the host age, in either months or years.</td>
							<td>Indicate whether the host age is in months or years. Age indicated in months will be binned to the 0 - 9 year age bin. </td>
							<td>years</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">host age bin</td>
							<td>Age of host at the time of sampling, expressed as an age group.</td>
							<td>Select the corresponding host age bin from the pick list provided in the template. If not available, provide a null value.</td>
							<td></td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">host gender</td>
							<td>The gender of the host at the time of sample collection.</td>
							<td>Select the corresponding host gender from the pick list provided in the template. If not available, provide a null value. If there is no host, put "Not Applicable".</td>
							<td>male</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">host residence geo_loc name (country)</td>
							<td>The country of residence of the host.</td>
							<td>Select the country name from pick list provided in the template.</td>
							<td>Canada</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">host ethnicity</td>
							<td>The self-identified ethnicity(ies) of the host.</td>
							<td>If known, provide the self-identified ethnicity or ethnicities of the host as a free text description. This is highly sensitive information which must be treated respectfully and carefully when sharing. The information may have implications for equitable access and benefit sharing. Consult your privacy officer, data steward and/or cultural services representative.</td>
							<td>Indigenous, European</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">host subject ID</td>
							<td>A unique identifier by which each host can be referred to e.g. #131</td>
							<td>Provide the host identifier. Should be a unique, user-defined identifier.</td>
							<td>BCxy123</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">symptom onset date</td>
							<td>The date on which the symptoms began or were first noted.</td>
							<td>ISO 8601 standard "YYYY-MM-DD".</td>
							<td>16-03-2020</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">signs and symptoms</td>
							<td>A perceived change in function or sensation, (loss, disturbance or appearance) indicative of a disease, reported by a patient or clinician.</td>
							<td>Select all of the symptoms experienced by the host from the pick list.</td>
							<td>Cough, Fever, Chills</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">pre-existing conditions and risk factors</td>
							<td>Patient pre-existing conditions and risk factors. Pre-existing condition: A medical condition that existed prior to the current infection. Risk Factor: A variable associated with an increased risk of disease or infection.</td>
							<td>Select all of the pre-existing conditions and risk factors experienced by the host from the pick list. If the desired term is missing, contact the curation team.</td>
							<td>Asthma</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">complications</td>
							<td>Patient medical complications that are believed to have occurred as a result of host disease.</td>
							<td>Select all of the complications experienced by the host from the pick list. If the desired term is missing, contact the curation team.</td>
							<td>Acute Respiratory Failure</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Host vaccination information</h3></td>
						</tr>
						
						<tr>
							<td class="label">host vaccination status</td>
							<td>The vaccination status of the host (fully vaccinated, partially vaccinated, or not vaccinated).</td>
							<td>Select the vaccination status of the host from the pick list.</td>
							<td>Fully Vaccinated</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">vaccine name</td>
							<td>The name of the vaccine.</td>
							<td>Free text. Provide the name of the vaccine.</td>
							<td>BNT162b2, Pfizer-BioNTech</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">number of vaccine doses received</td>
							<td>The number of doses of the vaccine recived by the host.</td>
							<td>Record how many doses of the vaccine the host has received.</td>
							<td>2</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">first dose vaccination date</td>
							<td>The date the host was first vaccinated.</td>
							<td>Provide the vaccination date in ISO 8601 standard format "YYYY-MM-DD".</td>
							<td>2021-02-26</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">last dose vaccination date</td>
							<td>The date the host received their last dose of vaccine.</td>
							<td>Provide the date that the last dose of the vaccine was administered. Provide the last dose vaccination date in ISO 8601 standard format "YYYY-MM-DD".</td>
							<td>2021-04-09</td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Host exposure information</h3></td>
						</tr>
						
						<tr>
							<td class="label">location of exposure geo_loc name (country)</td>
							<td>The country where the host was likely exposed to the causative agent of the illness.</td>
							<td>Select the country name from pick list provided in the template.</td>
							<td>Canada</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">destination of most recent travel (city)</td>
							<td>The name of the city that was the destination of most recent travel.</td>
							<td>Provide the name of the city that the host travelled to. Use this look-up service to identify the standardized term: https://www.ebi.ac.uk/ols/ontologies/gaz</td>
							<td>New York City</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">destination of most recent travel (state/province/territory)</td>
							<td>The name of the province that was the destination of most recent travel.</td>
							<td>Provide the name of the state/province/territory that the host travelled to. Use this look-up service to identify the standardized term: https://www.ebi.ac.uk/ols/ontologies/gaz</td>
							<td>California</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">destination of most recent travel (country)</td>
							<td>The name of the country that was the destination of most recent travel.</td>
							<td>Provide the name of the country that the host travelled to. Use this look-up service to identify the standardized term: https://www.ebi.ac.uk/ols/ontologies/gaz</td>
							<td>United Kingdom</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">most recent travel departure date</td>
							<td>The date of a person's most recent departure from their primary residence (at that time) on a journey to one or more other locations.</td>
							<td>Provide the travel departure date.</td>
							<td>2020-03-16</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">most recent travel return date</td>
							<td>The date of a person's most recent return to some residence from a journey originating at that residence.</td>
							<td>Provide the travel return date.</td>
							<td>2020-04-26</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">travel history</td>
							<td>Travel history in last six months.</td>
							<td>Specify the countries (and more granular locations if known, separated by a comma) travelled in the last six months; can include multiple travels. Separate multiple travel events with a semi-colon. List most recent travel first.</td>
							<td>Canada, Vancouver; USA, Seattle; Italy, Milan</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">exposure event</td>
							<td>Event leading to exposure.</td>
							<td>Select an exposure event from the pick list provided in the template. If the desired term is missing, contact the curation team.</td>
							<td>Convention</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">exposure contact level</td>
							<td>The exposure transmission contact type.</td>
							<td>Select direct or indirect exposure from the pick-list.</td>
							<td>Direct</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">host role</td>
							<td>The role of the host in relation to the exposure setting.</td>
							<td>Select the host's personal role(s) from the pick list provided in the template. If the desired term is missing, contact the curation team.</td>
							<td>Patient</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">exposure setting</td>
							<td>The setting leading to exposure.</td>
							<td>Select the host exposure setting(s) from the pick list provided in the template. If a desired term is missing, contact the curation team.</td>
							<td>Healthcare Setting</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">exposure details</td>
							<td>Additional host exposure information.</td>
							<td>Free text description of the exposure.</td>
							<td>Host role - Other: Bus Driver</td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Host reinfection information</h3></td>
						</tr>
						
						<tr>
							<td class="label">prior SARS-CoV-2 infection</td>
							<td>Whether there was prior SARS-CoV-2 infection.</td>
							<td>If known, provide infromation about whether the individual had a previous SARS-CoV-2 infection. Select a value from the pick list.</td>
							<td>Yes</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">prior SARS-CoV-2 infection isolate</td>
							<td>The identifier of the isolate found in the prior SARS-CoV-2 infection.</td>
							<td>Provide the isolate name of the most recent prior infection. Structure the "isolate" name to be ICTV/INSDC compliant in the following format: "SARS-CoV-2/host/country/sampleID/date".</td>
							<td>SARS-CoV-2/human/USA/CA-CDPH-001/2020</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">prior SARS-CoV-2 infection date</td>
							<td>The date of diagnosis of the prior SARS-CoV-2 infection.</td>
							<td>Provide the date that the most recent prior infection was diagnosed. Provide the prior SARS-CoV-2 infection date in ISO 8601 standard format "YYYY-MM-DD".</td>
							<td>2021-01-23</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">prior SARS-CoV-2 antiviral treatment</td>
							<td>Whether there was prior SARS-CoV-2 treatment with an antiviral agent.</td>
							<td>If known, provide infromation about whether the individual had a previous SARS-CoV-2 antiviral treatment. Select a value from the pick list.</td>
							<td>No prior antiviral treatment</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">prior SARS-CoV-2 antiviral treatment agent</td>
							<td>The name of the antiviral treatment agent administered during the prior SARS-CoV-2 infection.</td>
							<td>Provide the name of the antiviral treatment agent administered during the most recent prior infection. If no treatment was administered, put "No treatment". If multiple antiviral agents were administered, list them all separated by commas.</td>
							<td>Remdesivir</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">prior SARS-CoV-2 antiviral treatment date</td>
							<td>The date treatment was first administered during the prior SARS-CoV-2 infection.</td>
							<td>Provide the date that the antiviral treatment agent was first administered during the most recenrt prior infection. Provide the prior SARS-CoV-2 treatment date in ISO 8601 standard format "YYYY-MM-DD".</td>
							<td>2021-01-28</td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Sequencing</h3></td>
						</tr>
						
						<tr>
							<td class="label">purpose of sequencing</td>
							<td>The reason that the sample was sequenced.</td>
							<td>The reason why a sample was originally collected may differ from the reason why it was selected for sequencing. The reason a sample was sequenced may provide information about potential biases in sequencing strategy. Provide the purpose of sequencing from the picklist in the template. The reason for sample collection should be indicated in the "purpose of sampling" field. </td>
							<td>Baseline surveillance (random sampling)</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">purpose of sequencing details</td>
							<td>The description of why the sample was sequenced providing specific details.</td>
							<td>Provide an expanded description of why the sample was sequenced using free text. The description may include the importance of the sequences for a particular public health investigation/surveillance activity/research question. Suggested standardized descriotions include: Screened for S gene target failure (S dropout), Screened for mink variants, Screened for B.1.1.7 variant, Screened for B.1.135 variant, Screened for P.1 variant, Screened due to travel history, Screened due to close contact with infected individual, Assessing public health control measures, Determining early introductions and spread, Investigating airline-related exposures, Investigating temporary foreign worker, Investigating remote regions, Investigating health care workers, Investigating schools/universities, Investigating reinfection.
</td>
							<td>Screened for S gene target failure (S dropout)</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">sequencing date</td>
							<td>The date the sample was sequenced.</td>
							<td>ISO 8601 standard "YYYY-MM-DD".</td>
							<td>2020-06-22</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">library ID</td>
							<td>The user-specified identifier for the library prepared for sequencing.</td>
							<td>The library name should be unique, and can be an autogenerated ID from your LIMS, or modification of the isolate ID.</td>
							<td>XYZ_123345</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">amplicon size</td>
							<td>The length of the amplicon generated by PCR amplification.</td>
							<td>Provide the amplicon size, including the units.</td>
							<td>300bp</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">library preparation kit</td>
							<td>The name of the DNA library preparation kit used to generate the library being sequenced.</td>
							<td>Provide the name of the library preparation kit used.</td>
							<td>Nextera XT</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">flow cell barcode</td>
							<td>The barcode of the flow cell used for sequencing.</td>
							<td>Provide the barcode of the flow cell used for sequencing the sample.</td>
							<td>FAB06069</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sequencing instrument</td>
							<td>The model of the sequencing instrument used.</td>
							<td>Select a sequencing instrument from the picklist provided in the template.</td>
							<td>MinIon</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">sequencing protocol name</td>
							<td>The name and version number of the sequencing protocol used.</td>
							<td>Provide the name and version of the sequencing protocol e.g. 1D_DNA_MinION</td>
							<td>https://www.protocols.io/view/covid-19-artic-v3-illumina-library-construction-an-bibtkann</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sequencing protocol</td>
							<td>The protocol used to generate the sequence.</td>
							<td>Provide a free text description of the methods and materials used to generate the sequence. Suggested text, fill in information where indicated.: "Viral sequencing was performed following a tiling amplicon strategy using the <fill in> primer scheme. Sequencing was performed using a <fill in> sequencing instrument. Libraries were prepared using <fill in> library kit. "</td>
							<td>Genomes were generated through amplicon sequencing of 1200 bp amplicons with Freed schema primers. Libraries were created using Illumina DNA Prep kits, and sequence data was produced using Miseq Micro v2 (500 cycles) sequencing kits.</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sequencing kit number</td>
							<td>The manufacturer's kit number.</td>
							<td>Alphanumeric value.</td>
							<td>AB456XYZ789</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">amplicon pcr primer scheme</td>
							<td>The specifications of the primers (primer sequences, binding positions, fragment size generated etc) used to generate the amplicons to be sequenced.</td>
							<td>Provide the name and version of the primer scheme used to generate the amplicons for sequencing.</td>
							<td>https://github.com/joshquick/artic-ncov2019/blob/master/primer_schemes/nCoV-2019/V3/nCoV-2019.tsv</td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Bioinformatics and QC metrics</h3></td>
						</tr>
						
						<tr>
							<td class="label">raw sequence data processing method</td>
							<td>The names of the software and version number used for raw data processing such as removing barcodes, adapter trimming, filtering etc.</td>
							<td>Provide the software name followed by the version e.g. Trimmomatic v. 0.38, Porechop v. 0.2.3</td>
							<td>Porechop 0.2.3</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">dehosting method</td>
							<td>The method used to remove host reads from the pathogen sequence.</td>
							<td>Provide the name and version number of the software used to remove host reads.</td>
							<td>Nanostripper</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">consensus sequence name</td>
							<td>The name of the consensus sequence.</td>
							<td>Provide the name and version number of the consensus sequence.</td>
							<td>ncov123assembly3</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">consensus sequence filename</td>
							<td>The name of the consensus sequence file.</td>
							<td>Provide the name and version number of the consensus sequence FASTA file.</td>
							<td>ncov123assembly.fasta</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">consensus sequence filepath</td>
							<td>The filepath of the consesnsus sequence file.</td>
							<td>Provide the filepath of the consensus sequence FASTA file.</td>
							<td>/User/Documents/RespLab/Data/ncov123assembly.fasta</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">consensus sequence software name</td>
							<td>The name of software used to generate the consensus sequence.</td>
							<td>Provide the name of the software used to generate the consensus sequence.</td>
							<td>Ivar</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">consensus sequence software version</td>
							<td>The version of the software used to generate the consensus sequence.</td>
							<td>Provide the version of the software used to generate the consensus sequence.</td>
							<td>1.3</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">breadth of coverage value</td>
							<td>The percentage of the reference genome covered by the sequenced data, to a prescribed depth.</td>
							<td>Provide value as a percent.</td>
							<td>95%</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">depth of coverage value</td>
							<td>The average number of reads representing a given nucleotide in the reconstructed sequence.</td>
							<td>Provide value as a fold of coverage.</td>
							<td>400x</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">depth of coverage threshold</td>
							<td>The threshold used as a cut-off for the depth of coverage.</td>
							<td>Provide the threshold fold coverage.</td>
							<td>100x</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">r1 fastq filename</td>
							<td>The user-specified filename of the r1 FASTQ file.</td>
							<td>Provide the r1 FASTQ filename.</td>
							<td>ABC123_S1_L001_R1_001.fastq.gz</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">r2 fastq filename</td>
							<td>The user-specified filename of the r2 FASTQ file.</td>
							<td>Provide the r2 FASTQ filename.</td>
							<td>ABC123_S1_L001_R2_001.fastq.gz</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">r1 fastq filepath</td>
							<td>The location of the r1 FASTQ file within a user's file system.</td>
							<td>Provide the filepath for the r1 FASTQ file. This information aids in data management. </td>
							<td>/User/Documents/RespLab/Data/ABC123_S1_L001_R1_001.fastq.gz</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">r2 fastq filepath</td>
							<td>The location of the r2 FASTQ file within a user's file system.</td>
							<td>Provide the filepath for the r2 FASTQ file. This information aids in data management. </td>
							<td>/User/Documents/RespLab/Data/ABC123_S1_L001_R2_001.fastq.gz</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">fast5 filename</td>
							<td>The user-specified filename of the FAST5 file.</td>
							<td>Provide the FAST5 filename.</td>
							<td>rona123assembly.fast5</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">fast5 filepath</td>
							<td>The location of the FAST5 file within a user's file system.</td>
							<td>Provide the filepath for the FAST5 file. This information aids in data management. </td>
							<td>/User/Documents/RespLab/Data/rona123assembly.fast5</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">number of base pairs sequenced</td>
							<td>The number of total base pairs generated by the sequencing process.</td>
							<td>Provide a numerical value (no need to include units).</td>
							<td>387566</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">consensus genome length</td>
							<td>Size of the reconstructed genome described as the number of base pairs.</td>
							<td>Provide a numerical value (no need to include units).</td>
							<td>38677</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">Ns per 100 kbp</td>
							<td>The number of N symbols present in the consensus fasta sequence, per 100kbp of sequence.</td>
							<td>Provide a numerical value (no need to include units).</td>
							<td>330</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">reference genome accession</td>
							<td>A persistent, unique identifier of a genome database entry.</td>
							<td>Provide the accession number of the reference genome.</td>
							<td>NC_045512.2</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">bioinformatics protocol</td>
							<td>The name and version number of the bioinformatics protocol used.</td>
							<td>Further details regarding the methods used to process raw data, and/or generate assemblies, and/or generate consensus sequences can be provided in an SOP or protocol. Provide the name and version number of the protocol.</td>
							<td>https://www.protocols.io/groups/cphln-sarscov2-sequencing-consortium/members</td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Lineage and Variant information</h3></td>
						</tr>
						
						<tr>
							<td class="label">lineage/clade name</td>
							<td>The name of the lineage or clade.</td>
							<td>Provide the Pangolin or Nextstrain lineage/clade name.</td>
							<td>B.1.1.7</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">lineage/clade analysis software name</td>
							<td>The name of the software used to determine the lineage/clade.</td>
							<td>Provide the name of the software used to determine the lineage/clade.</td>
							<td>Pangolin</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">lineage/clade analysis software version</td>
							<td>The version of the software used to determine the lineage/clade.</td>
							<td>Provide the version of the software used ot determine the lineage/clade.</td>
							<td>2.1.10</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">variant designation</td>
							<td>The variant classification of the lineage/clade i.e. variant, variant of concern.</td>
							<td>If the lineage/clade is considered a Variant of Concern, select Variant of Concern from the pick list. If the lineage/clade contains mutations of concern (mutations that increase transmission, clincal severity, or other epidemiological fa ctors) but it not a global Variant of Concern, select Variant. If the lineage/clade does not contain mutations of concern, leave blank.</td>
							<td>Variant of Concern</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">variant evidence details</td>
							<td>Details about the evidence used to make the variant determination.</td>
							<td>Provide the assay and list the set of lineage-defining mutations used to make the variant determination. If there are mutations of interest/concern observed in addition to lineage-defining mutations, describe those here.</td>
							<td>RT-qPCR TaqPath assay: S gene target failure</td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Pathogen diagnostic testing</h3></td>
						</tr>
						
						<tr>
							<td class="label">gene name 1</td>
							<td>The name of the gene used in the diagnostic RT-PCR test.</td>
							<td>Provide the full name of the gene used in the test. The gene symbol (short form of gene name) can also be provided. Standardized gene names and symbols can be found in the Gene Ontology using this look-up service: https://bit.ly/2Sq1LbI</td>
							<td>E (orf4)</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">diagnostic pcr protocol 1</td>
							<td>The name and version number of the protocol used for diagnostic marker amplification.</td>
							<td>The name and version number of the protocol used for carrying out a diagnostic PCR test. This information can be compared to sequence data for evaluation of performance and quality control.</td>
							<td>EGenePCRTest 2</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">diagnostic pcr Ct value 1</td>
							<td>The Ct value result from a diagnostic SARS-CoV-2 RT-PCR test.</td>
							<td>Provide the CT value of the sample from the diagnostic RT-PCR test.</td>
							<td>21</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">gene name 2</td>
							<td>The name of the gene used in the diagnostic RT-PCR test.</td>
							<td>Provide the full name of another gene used in an RT-PCR test. The gene symbol (short form of gene name) can also be provided. Standardized gene names and symbols can be found in the Gene Ontology using this look-up service: https://bit.ly/2Sq1LbI</td>
							<td>nsp12 (RdRp)</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">diagnostic pcr protocol 2</td>
							<td>The name and version number of the protocol used for diagnostic marker amplification.</td>
							<td>The name and version number of the protocol used for carrying out a second diagnostic PCR test. This information can be compared to sequence data for evaluation of performance and quality control.</td>
							<td>RdRpGenePCRTest 3</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">diagnostic pcr Ct value 2</td>
							<td>The Ct value result from a diagnostic SARS-CoV-2 RT-PCR test.</td>
							<td>Provide the CT value of the sample from the second diagnostic RT-PCR test.</td>
							<td>36</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">gene name 3</td>
							<td>The name of the gene used in the diagnostic RT-PCR test.</td>
							<td>Provide the full name of another gene used in an RT-PCR test. The gene symbol (short form of gene name) can also be provided. Standardized gene names and symbols can be found in the Gene Ontology using this look-up service: https://bit.ly/2Sq1LbI</td>
							<td>nsp12 (RdRp)</td>
							<td>['Not Applicable', 'Missing', 'Not Collected', 'Not Provided', 'Restricted Access']</td>
						</tr>

						
						<tr>
							<td class="label">diagnostic pcr protocol 3</td>
							<td>The name and version number of the protocol used for diagnostic marker amplification.</td>
							<td>The name and version number of the protocol used for carrying out a second diagnostic PCR test. This information can be compared to sequence data for evaluation of performance and quality control.</td>
							<td>RdRpGenePCRTest 3</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">diagnostic pcr Ct value 3</td>
							<td>The Ct value result from a diagnostic SARS-CoV-2 RT-PCR test.</td>
							<td>Provide the CT value of the sample from the second diagnostic RT-PCR test.</td>
							<td>30</td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>NCBI SRA information</h3></td>
						</tr>
						
						<tr>
							<td class="label">title</td>
							<td>Short description that will identify  the dataset on public pages.</td>
							<td>Format: {methodology} of {organism}: {sample info}</td>
							<td>Genomic sequencing of SARS-CoV-2: Nasopharynx (NP), Swab</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">library_strategy</td>
							<td>See NCBI SRA template for details.</td>
							<td>Provide the library strategy by selecting a value from the pick list. For amplicon sequencing select "AMPLICON". </td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">library_source</td>
							<td>See NCBI SRA template for details.</td>
							<td>Provide the library source by selecting a value from the pick list. For amplicon sequencing select "Viral RNA". </td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">library_selection</td>
							<td>See NCBI SRA template for details.</td>
							<td>Provide the library selection by selecting a value from the pick list. For amplicon sequencing select "PCR". </td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">library_layout</td>
							<td>See NCBI SRA template for details.</td>
							<td>Provide the library layout by selecting a value from the pick list. For Illumina instruments, select "PAIRED". For Oxford Nanopore instruments, select "SINGLE".</td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">filetype</td>
							<td>See NCBI SRA template for details.</td>
							<td>Provide the filetype by selecting a value from the pick list. </td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">filename</td>
							<td>See NCBI SRA template for details.</td>
							<td>Provide the appropriate filename recorded in the Bioinformatics and QC metrics section. If sequence data is "paired", provide the second filename under "Filename 2".</td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">filename2</td>
							<td>See NCBI SRA template for details.</td>
							<td>Provide the appropriate filename recorded in the Bioinformatics and QC metrics section.</td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Contributor acknowledgement</h3></td>
						</tr>
						
						<tr>
							<td class="label">authors</td>
							<td>Names of individuals contributing to the processes of sample collection, sequence generation, analysis, and data submission.</td>
							<td>Include the first and last names of all individuals that should be attributed, separated by a comma.</td>
							<td>Tejinder Singh, Fei Hu, Joe Blogs</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">DataHarmonizer provenance</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>data status</h3></td>
						</tr>
						
						<tr>
							<td class="label">Not Applicable</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">Missing</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">Not Collected</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">Not Provided</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">Restricted Access</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">prior SARS-CoV-2 infection</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						</table>

		</tbody>
	</table>
</body>
</html>