"use strict";(this.webpackChunkschemas=this.webpackChunkschemas||[]).push([[816],{816:e=>{e.exports=JSON.parse('{"name":"mixs","comments":["slot titles that are associated with more than one slot name/SCN: host sex"],"source":"https://github.com/GenomicsStandardsConsortium/mixs/raw/issue-610-temp-mixs-xlsx-home/mixs/excel/mixs_v6.xlsx","id":"https://w3id.org/mixs","version":"v6.2.0","prefixes":{"linkml":{"prefix_prefix":"linkml","prefix_reference":"https://w3id.org/linkml/"},"MIXS":{"prefix_prefix":"MIXS","prefix_reference":"https://w3id.org/mixs/"},"xsd":{"prefix_prefix":"xsd","prefix_reference":"http://www.w3.org/2001/XMLSchema#"},"shex":{"prefix_prefix":"shex","prefix_reference":"http://www.w3.org/ns/shex#"},"schema":{"prefix_prefix":"schema","prefix_reference":"http://schema.org/"}},"default_prefix":"MIXS","default_range":"string","subsets":{"combination_classes":{"name":"combination_classes","from_schema":"https://w3id.org/mixs"},"sequencing":{"name":"sequencing","from_schema":"https://w3id.org/mixs"},"environment":{"name":"environment","from_schema":"https://w3id.org/mixs"},"nucleic acid sequence source":{"name":"nucleic acid sequence source","from_schema":"https://w3id.org/mixs"},"investigation":{"name":"investigation","from_schema":"https://w3id.org/mixs"},"ethics":{"name":"ethics","from_schema":"https://w3id.org/mixs"},"example":{"name":"example","from_schema":"https://w3id.org/mixs"}},"types":{"string":{"name":"string","description":"A character string","notes":["In RDF serializations, a slot with range of string is treated as a literal or type xsd:string.   If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"string\\"."],"from_schema":"https://w3id.org/mixs","exact_mappings":["schema:Text"],"base":"str","uri":"xsd:string"},"integer":{"name":"integer","description":"An integer","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"integer\\"."],"from_schema":"https://w3id.org/mixs","exact_mappings":["schema:Integer"],"base":"int","uri":"xsd:integer"},"boolean":{"name":"boolean","description":"A binary (true or false) value","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"boolean\\"."],"from_schema":"https://w3id.org/mixs","exact_mappings":["schema:Boolean"],"base":"Bool","uri":"xsd:boolean","repr":"bool"},"float":{"name":"float","description":"A real number that conforms to the xsd:float specification","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"float\\"."],"from_schema":"https://w3id.org/mixs","exact_mappings":["schema:Float"],"base":"float","uri":"xsd:float"},"datetime":{"name":"datetime","description":"The combination of a date and time","notes":["If you are authoring schemas in LinkML YAML, the type is referenced with the lower case \\"datetime\\"."],"from_schema":"https://w3id.org/mixs","exact_mappings":["schema:DateTime"],"base":"XSDDateTime","uri":"xsd:dateTime","repr":"str"}},"enums":{"NEG_CONT_TYPE_ENUM":{"name":"NEG_CONT_TYPE_ENUM","from_schema":"https://w3id.org/mixs","permissible_values":{"DNA-free PCR mix":{"text":"DNA-free PCR mix"},"distilled water":{"text":"distilled water"},"empty collection device":{"text":"empty collection device"},"empty collection tube":{"text":"empty collection tube"},"phosphate buffer":{"text":"phosphate buffer"},"sterile swab":{"text":"sterile swab"},"sterile syringe":{"text":"sterile syringe"}}},"REL_TO_OXYGEN_ENUM":{"name":"REL_TO_OXYGEN_ENUM","from_schema":"https://w3id.org/mixs","permissible_values":{"aerobe":{"text":"aerobe"},"anaerobe":{"text":"anaerobe"},"facultative":{"text":"facultative"},"microaerophilic":{"text":"microaerophilic"},"microanaerobe":{"text":"microanaerobe"},"obligate aerobe":{"text":"obligate aerobe"},"obligate anaerobe":{"text":"obligate anaerobe"}}},"LIB_LAYOUT_ENUM":{"name":"LIB_LAYOUT_ENUM","from_schema":"https://w3id.org/mixs","permissible_values":{"other":{"text":"other"},"paired":{"text":"paired"},"single":{"text":"single"},"vector":{"text":"vector"}}},"OXY_STAT_SAMP_ENUM":{"name":"OXY_STAT_SAMP_ENUM","from_schema":"https://w3id.org/mixs","permissible_values":{"aerobic":{"text":"aerobic"},"anaerobic":{"text":"anaerobic"},"other":{"text":"other"}}},"SAMP_CAPT_STATUS_ENUM":{"name":"SAMP_CAPT_STATUS_ENUM","from_schema":"https://w3id.org/mixs","permissible_values":{"active surveillance in response to an outbreak":{"text":"active surveillance in response to an outbreak"},"active surveillance not initiated by an outbreak":{"text":"active surveillance not initiated by an outbreak"},"farm sample":{"text":"farm sample"},"market sample":{"text":"market sample"},"other":{"text":"other"}}},"SAMP_DIS_STAGE_ENUM":{"name":"SAMP_DIS_STAGE_ENUM","from_schema":"https://w3id.org/mixs","permissible_values":{"dissemination":{"text":"dissemination"},"growth and reproduction":{"text":"growth and reproduction"},"infection":{"text":"infection"},"inoculation":{"text":"inoculation"},"other":{"text":"other"},"penetration":{"text":"penetration"}}},"SEDIMENT_TYPE_ENUM":{"name":"SEDIMENT_TYPE_ENUM","from_schema":"https://w3id.org/mixs","permissible_values":{"biogenous":{"text":"biogenous"},"cosmogenous":{"text":"cosmogenous"},"hydrogenous":{"text":"hydrogenous"},"lithogenous":{"text":"lithogenous"}}},"TIDAL_STAGE_ENUM":{"name":"TIDAL_STAGE_ENUM","from_schema":"https://w3id.org/mixs","permissible_values":{"ebb tide":{"text":"ebb tide"},"flood tide":{"text":"flood tide"},"high tide":{"text":"high tide"},"low tide":{"text":"low tide"}}},"DamageTreatmentEnum":{"name":"DamageTreatmentEnum","from_schema":"https://w3id.org/mixs","permissible_values":{"none":{"text":"none"},"partial-udg":{"text":"partial-udg"},"full-udg":{"text":"full-udg"},"enriched":{"text":"enriched"},"other":{"text":"other"}}},"LibTypeEnum":{"name":"LibTypeEnum","from_schema":"https://w3id.org/mixs","permissible_values":{"amplicon":{"text":"amplicon"},"enriched":{"text":"enriched"},"shotgun":{"text":"shotgun"}}},"BioCulturalLabel":{"name":"BioCulturalLabel","from_schema":"https://w3id.org/mixs","permissible_values":{"BC P":{"text":"BC P"},"BC MC":{"text":"BC MC"},"BC CL":{"text":"BC CL"},"BC CV":{"text":"BC CV"},"BC CNV":{"text":"BC CNV"},"BC R":{"text":"BC R"},"BC CB":{"text":"BC CB"},"BC OC":{"text":"BC OC"},"BC O":{"text":"BC O"},"BC NC":{"text":"BC NC"}}},"LibStrand":{"name":"LibStrand","from_schema":"https://w3id.org/mixs","permissible_values":{"single":{"text":"single"},"double":{"text":"double"}}},"SampAgeUnit":{"name":"SampAgeUnit","from_schema":"https://w3id.org/mixs","permissible_values":{"BCE":{"text":"BCE"},"CE":{"text":"CE"},"BP":{"text":"BP"},"cal BP":{"text":"cal BP"},"cal BCE":{"text":"cal BCE"},"cal CE":{"text":"cal CE"},"ka":{"text":"ka"},"Ma":{"text":"Ma"}}},"SampAgeInferMethod":{"name":"SampAgeInferMethod","from_schema":"https://w3id.org/mixs","permissible_values":{"radiocarbon dating":{"text":"radiocarbon dating"},"optically stimulated infrared luminescence":{"text":"optically stimulated infrared luminescence"},"amino acid racemisation":{"text":"amino acid racemisation"},"electron spin resonance":{"text":"electron spin resonance"},"uranium thorium":{"text":"uranium thorium"},"stratigraphic inference":{"text":"stratigraphic inference"},"relative dating":{"text":"relative dating"},"historical records":{"text":"historical records"},"other":{"text":"other"}}},"MsMethod":{"name":"MsMethod","from_schema":"https://w3id.org/mixs","permissible_values":{"IMRS":{"text":"IMRS"},"AMS":{"text":"AMS"}}},"CalibCurve":{"name":"CalibCurve","from_schema":"https://w3id.org/mixs","permissible_values":{"IntCal98":{"text":"IntCal98"},"IntCal04":{"text":"IntCal04"},"IntCal09":{"text":"IntCal09"},"IntCal13":{"text":"IntCal13"},"IntCal20":{"text":"IntCal20"},"SHCal04":{"text":"SHCal04"},"SHCal13":{"text":"SHCal13"},"SHCal20":{"text":"SHCal20"},"Bomb04NH1":{"text":"Bomb04NH1"},"Bomb04NH2":{"text":"Bomb04NH2"},"Bomb04NH3":{"text":"Bomb04NH3"},"Bomb04SH":{"text":"Bomb04SH"},"Bomb13NH1":{"text":"Bomb13NH1"},"Bomb13NH2":{"text":"Bomb13NH2"},"Bomb13NH3":{"text":"Bomb13NH3"},"Bomb13SH12":{"text":"Bomb13SH12"},"Bomb13SH3":{"text":"Bomb13SH3"},"Bomb21NH1":{"text":"Bomb21NH1"},"Bomb21NH2":{"text":"Bomb21NH2"},"Bomb21NH3":{"text":"Bomb21NH3"},"Bomb21SH12":{"text":"Bomb21SH12"},"Bomb21SH3":{"text":"Bomb21SH3"},"Kueppers04":{"text":"Kueppers04"},"Marine98":{"text":"Marine98"},"Marine04":{"text":"Marine04"},"Marine09":{"text":"Marine09"},"Marine13":{"text":"Marine13"},"Marine20":{"text":"Marine20"},"CalPal2007_HULU":{"text":"CalPal2007_HULU"}}},"ConfidenceSigma":{"name":"ConfidenceSigma","from_schema":"https://w3id.org/mixs","permissible_values":{"1":{"text":"1"},"2":{"text":"2"}}},"RadiocarbonCalibUnit":{"name":"RadiocarbonCalibUnit","from_schema":"https://w3id.org/mixs","permissible_values":{"cal BP":{"text":"cal BP"},"cal AD":{"text":"cal AD"},"cal BC":{"text":"cal BC"},"cal CE":{"text":"cal CE"},"cal BCE":{"text":"cal BCE"}}}},"slots":{"adapters":{"name":"adapters","annotations":{"Expected_value":{"tag":"Expected_value","value":"adapter A and B sequence"}},"description":"Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters","title":"adapters","examples":[{"value":"AATGATACGGCGACCACCGAGATCTACACGCT;CAAGCAGAAGACGGCATACGAGAT"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{dna};{dna}","slot_uri":"MIXS:0000048","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"]},"alkalinity":{"name":"alkalinity","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milliequivalent per liter, milligram per liter"}},"description":"Alkalinity, the ability of a solution to neutralize acids to the equivalence point of carbonate or bicarbonate","title":"alkalinity","examples":[{"value":"50 milligram per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["alkalinity"],"slot_uri":"MIXS:0000421","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"alkyl_diethers":{"name":"alkyl_diethers","description":"Concentration of alkyl diethers","title":"alkyl diethers","examples":[{"value":"0.005 mole per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000490","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"alt":{"name":"alt","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth\'s surface. In this context, the altitude measurement is the vertical distance between the earth\'s surface above sea level and the sampled position in the air","title":"altitude","examples":[{"value":"100 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000094","domain_of":["Mims","HostAssociated","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"aminopept_act":{"name":"aminopept_act","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"mole per liter per hour"}},"description":"Measurement of aminopeptidase activity","title":"aminopeptidase activity","examples":[{"value":"0.269 mole per liter per hour"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000172","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"ammonium":{"name":"ammonium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of ammonium in the sample","title":"ammonium","examples":[{"value":"1.5 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000427","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"ances_data":{"name":"ances_data","description":"Information about either pedigree or other ancestral information description (e.g. parental variety in case of mutant or selection), e.g. A/3*B (meaning [(A x B) x B] x B)","title":"ancestral data","examples":[{"value":"A/3*B"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000247","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"annot":{"name":"annot","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of tool or pipeline used, or annotation source description"}},"description":"Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter","title":"annotation","examples":[{"value":"prokka"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:0000059","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"]},"assembly_name":{"name":"assembly_name","annotations":{"Expected_value":{"tag":"Expected_value","value":"name and version of assembly"}},"description":"Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community","title":"assembly name","examples":[{"value":"HuRef, JCVI_ISG_i3_1.0"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text} {text}","slot_uri":"MIXS:0000057","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"]},"assembly_qual":{"name":"assembly_qual","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling   90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling < 90% of the expected genome or replicon sequence, or for which no genome size could be estimated","title":"assembly quality","examples":[{"value":"High-quality draft genome"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["quality"],"string_serialization":"[Finished genome|High-quality draft genome|Medium-quality draft genome|Low-quality draft genome|Genome fragment(s)]","slot_uri":"MIXS:0000056","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"]},"assembly_software":{"name":"assembly_software","description":"Tool(s) used for assembly, including version number and parameters","title":"assembly software","examples":[{"value":"metaSPAdes;3.11.0;kmer set 21,33,55,77,99,121, default parameters otherwise"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["software"],"slot_uri":"MIXS:0000058","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"associated_resource":{"name":"associated_resource","annotations":{"Expected_value":{"tag":"Expected_value","value":"reference to resource"}},"description":"A related resource that is referenced, cited, or otherwise associated to the sequence","title":"relevant electronic resources","examples":[{"value":"http://www.earthmicrobiome.org/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["resource"],"string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000091","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"recommended":true,"multivalued":true},"bacteria_carb_prod":{"name":"bacteria_carb_prod","description":"Measurement of bacterial carbon production","title":"bacterial carbon production","examples":[{"value":"2.53 microgram per liter per hour"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","production"],"slot_uri":"MIXS:0000173","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"biol_stat":{"name":"biol_stat","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"The level of genome modification","title":"biological status","examples":[{"value":"natural"}],"from_schema":"https://w3id.org/mixs","keywords":["status"],"string_serialization":"[wild|natural|semi-natural|inbred line|breeder\'s line|hybrid|clonal selection|mutant]","slot_uri":"MIXS:0000858","domain_of":["HostAssociated","MimsHostAssociated"]},"biomass":{"name":"biomass","annotations":{"Expected_value":{"tag":"Expected_value","value":"biomass type;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"ton, kilogram, gram"}},"description":"Amount of biomass; should include the name for the part of biomass measured, e.g. Microbial, total. Can include multiple measurements","title":"biomass","examples":[{"value":"total;20 gram"}],"from_schema":"https://w3id.org/mixs","keywords":["biomass"],"string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000174","domain_of":["Sediment","MimsSediment"],"multivalued":true},"bishomohopanol":{"name":"bishomohopanol","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"microgram per liter, microgram per gram"}},"description":"Concentration of bishomohopanol","title":"bishomohopanol","examples":[{"value":"14 microgram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000175","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"blood_press_diast":{"name":"blood_press_diast","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"millimeter mercury"}},"description":"Resting diastolic blood pressure, measured as mm mercury","title":"host blood pressure diastolic","from_schema":"https://w3id.org/mixs","keywords":["host","host.","pressure"],"slot_uri":"MIXS:0000258","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"blood_press_syst":{"name":"blood_press_syst","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"millimeter mercury"}},"description":"Resting systolic blood pressure, measured as mm mercury","title":"host blood pressure systolic","from_schema":"https://w3id.org/mixs","keywords":["host","host.","pressure"],"slot_uri":"MIXS:0000259","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"bromide":{"name":"bromide","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"parts per million"}},"description":"Concentration of bromide","title":"bromide","examples":[{"value":"0.05 parts per million"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000176","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"calcium":{"name":"calcium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, micromole per liter, parts per million"}},"description":"Concentration of calcium in the sample","title":"calcium","examples":[{"value":"0.2 micromole per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000432","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"carb_nitro_ratio":{"name":"carb_nitro_ratio","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement value"}},"description":"Ratio of amount or concentrations of carbon to nitrogen. Used for quality control value in proteinaceous samples for radiocarbon dating.","title":"carbon/nitrogen ratio","examples":[{"value":"0.417361111"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","nitrogen","ratio"],"string_serialization":"{float}:{float}","slot_uri":"MIXS:0000310","domain_of":["Sediment","RadiocarbonDating","MimsSediment"],"range":"float","required":false,"recommended":true,"multivalued":false},"chem_administration":{"name":"chem_administration","annotations":{"Expected_value":{"tag":"Expected_value","value":"CHEBI;timestamp"}},"description":"List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi","title":"chemical administration","examples":[{"value":"agar [CHEBI:2509];2018-05-11T20:00Z"}],"from_schema":"https://w3id.org/mixs","keywords":["administration"],"string_serialization":"{termLabel} [{termID}];{timestamp}","slot_uri":"MIXS:0000751","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"multivalued":true},"chloride":{"name":"chloride","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, parts per million"}},"description":"Concentration of chloride in the sample","title":"chloride","examples":[{"value":"5000 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000429","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"chlorophyll":{"name":"chlorophyll","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per cubic meter, microgram per liter"}},"description":"Concentration of chlorophyll","title":"chlorophyll","examples":[{"value":"5 milligram per cubic meter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000177","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"collection_date":{"name":"collection_date","description":"The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant","title":"collection date","examples":[{"value":"2013-03-25T12:42:31+01:00"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["date"],"slot_uri":"MIXS:0000011","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"datetime","required":true},"density":{"name":"density","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"gram per cubic meter, gram per cubic centimeter"}},"description":"Density of the sample, which is its mass per unit volume (aka volumetric mass density)","title":"density","examples":[{"value":"1000 kilogram per cubic meter"}],"from_schema":"https://w3id.org/mixs","keywords":["density"],"slot_uri":"MIXS:0000435","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"depth":{"name":"depth","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"The vertical distance below local surface. For sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples","title":"depth","in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["depth"],"slot_uri":"MIXS:0000018","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diether_lipids":{"name":"diether_lipids","annotations":{"Expected_value":{"tag":"Expected_value","value":"diether lipid name;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"nanogram per liter"}},"description":"Concentration of diether lipids; can include multiple types of diether lipids","title":"diether lipids","examples":[{"value":"0.2 nanogram per liter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000178","domain_of":["Sediment","MimsSediment"],"multivalued":true},"diss_carb_dioxide":{"name":"diss_carb_dioxide","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter"}},"description":"Concentration of dissolved carbon dioxide in the sample or liquid portion of the sample","title":"dissolved carbon dioxide","examples":[{"value":"5 milligram per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","dissolved"],"slot_uri":"MIXS:0000436","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_hydrogen":{"name":"diss_hydrogen","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of dissolved hydrogen","title":"dissolved hydrogen","examples":[{"value":"0.3 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["dissolved"],"slot_uri":"MIXS:0000179","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_inorg_carb":{"name":"diss_inorg_carb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"microgram per liter, milligram per liter, parts per million"}},"description":"Dissolved inorganic carbon concentration in the sample, typically measured after filtering the sample using a 0.45 micrometer filter","title":"dissolved inorganic carbon","examples":[{"value":"2059 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","dissolved","inorganic"],"slot_uri":"MIXS:0000434","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_org_carb":{"name":"diss_org_carb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter"}},"description":"Concentration of dissolved organic carbon in the sample, liquid portion of the sample, or aqueous phase of the fluid","title":"dissolved organic carbon","examples":[{"value":"197 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","dissolved","organic"],"slot_uri":"MIXS:0000433","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_org_nitro":{"name":"diss_org_nitro","description":"Dissolved organic nitrogen concentration measured as; total dissolved nitrogen - NH4 - NO3 - NO2","title":"dissolved organic nitrogen","examples":[{"value":"0.05 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["dissolved","nitrogen","organic"],"slot_uri":"MIXS:0000162","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_oxygen":{"name":"diss_oxygen","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per kilogram, milligram per liter"}},"description":"Concentration of dissolved oxygen","title":"dissolved oxygen","examples":[{"value":"175 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","keywords":["dissolved","oxygen"],"slot_uri":"MIXS:0000119","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"elev":{"name":"elev","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth\'s surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit","title":"elevation","examples":[{"value":"100 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["elevation"],"slot_uri":"MIXS:0000093","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"env_broad_scale":{"name":"env_broad_scale","description":"Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS","title":"broad-scale environmental context","examples":[{"value":"rangeland biome [ENVO:01000247]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["context","environmental"],"slot_uri":"MIXS:0000012","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\]$","structured_pattern":{"syntax":"^{termLabel} \\\\[{termID}\\\\]$","interpolated":true,"partial_match":true}},"env_local_scale":{"name":"env_local_scale","annotations":{"Expected_value":{"tag":"Expected_value","value":"Environmental entities having causal influences upon the entity at time of sampling"}},"description":"Report the entity or entities which are in the sample or specimen s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS","title":"local environmental context","examples":[{"value":"hillside [ENVO:01000333]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["context","environmental"],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000013","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"required":true},"env_medium":{"name":"env_medium","description":"Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of \'environmental material\' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top)","title":"environmental medium","examples":[{"value":"bluegrass field soil [ENVO:00005789]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["environmental"],"slot_uri":"MIXS:0000014","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\]$","structured_pattern":{"syntax":"^{termLabel} \\\\[{termID}\\\\]$","interpolated":true,"partial_match":true}},"experimental_factor":{"name":"experimental_factor","annotations":{"Expected_value":{"tag":"Expected_value","value":"text or EFO and/or OBI"}},"description":"Variable aspects of an experiment design that can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI)","title":"experimental factor","examples":[{"value":"time series design [EFO:0001779]"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["experimental","factor"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000008","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^\\\\S+.*\\\\S+ \\\\[[a-zA-Z]{2,}:\\\\d+\\\\]$"},"feat_pred":{"name":"feat_pred","description":"Method used to predict UViGs features such as ORFs, integration site, etc","title":"feature prediction","examples":[{"value":"Prodigal;2.6.3;default parameters"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["feature","predict"],"slot_uri":"MIXS:0000061","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"genetic_mod":{"name":"genetic_mod","annotations":{"Expected_value":{"tag":"Expected_value","value":"PMID, DOI, URL or text"}},"description":"Genetic modifications of the genome of an organism, which may occur naturally by spontaneous mutation, or be introduced by some experimental means, e.g. specification of a transgene or the gene knocked-out or details of transient transfection","title":"genetic modification","examples":[{"value":"aox1A transgenic"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000859","domain_of":["HostAssociated","MimsHostAssociated"]},"geo_loc_name":{"name":"geo_loc_name","description":"The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)","title":"geographic location (country and/or sea,region)","examples":[{"value":"USA: Maryland, Bethesda"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["geographic","location"],"slot_uri":"MIXS:0000010","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+): ([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+), ([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{text}: {text}, {text}$","interpolated":true,"partial_match":true}},"glucosidase_act":{"name":"glucosidase_act","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"mol per liter per hour"}},"description":"Measurement of glucosidase activity","title":"glucosidase activity","examples":[{"value":"5 mol per liter per hour"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000137","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"gravidity":{"name":"gravidity","description":"Whether or not subject is gravid, and if yes date due or date post-conception, specifying which is used","title":"gravidity","examples":[{"value":"yes;due date:2018-05-11"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{boolean};{timestamp}","slot_uri":"MIXS:0000875","domain_of":["HostAssociated","MimsHostAssociated"]},"host_age":{"name":"host_age","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"year, day, hour"}},"description":"Age of host at the time of sampling; relevant scale depends on species and study, e.g. Could be seconds for amoebae or centuries for trees","title":"host age","from_schema":"https://w3id.org/mixs","keywords":["age","host","host."],"slot_uri":"MIXS:0000255","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_body_habitat":{"name":"host_body_habitat","description":"Original body habitat where the sample was obtained from","title":"host body habitat","from_schema":"https://w3id.org/mixs","keywords":["body","habitat","host","host."],"slot_uri":"MIXS:0000866","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_body_product":{"name":"host_body_product","annotations":{"Expected_value":{"tag":"Expected_value","value":"FMA or UBERON"}},"description":"Substance produced by the body, e.g. Stool, mucus, where the sample was obtained from. Use terms from the foundational model of anatomy ontology (fma) or Uber-anatomy ontology (UBERON)","title":"host body product","examples":[{"value":"mucus [FMA:66938]"}],"from_schema":"https://w3id.org/mixs","keywords":["body","host","host.","product"],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000888","domain_of":["HostAssociated","MimsHostAssociated"]},"host_body_site":{"name":"host_body_site","annotations":{"Expected_value":{"tag":"Expected_value","value":"FMA or UBERON"}},"description":"Name of body site where the sample was obtained from, such as a specific organ or tissue (tongue, lung etc...). Use terms from the foundational model of anatomy ontology (fma) or the Uber-anatomy ontology (UBERON)","title":"host body site","from_schema":"https://w3id.org/mixs","keywords":["body","host","host.","site"],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000867","domain_of":["HostAssociated","MimsHostAssociated"]},"host_body_temp":{"name":"host_body_temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Core body temperature of the host when sample was collected","title":"host body temperature","from_schema":"https://w3id.org/mixs","keywords":["body","host","host.","temperature"],"slot_uri":"MIXS:0000274","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_color":{"name":"host_color","description":"The color of host","title":"host color","from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000260","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_common_name":{"name":"host_common_name","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":""}},"description":"Common name of the host","title":"host common name","from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000248","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_diet":{"name":"host_diet","description":"Type of diet depending on the host, for animals omnivore, herbivore etc., for humans high-fat, meditteranean etc.; can include multiple diet types","title":"host diet","from_schema":"https://w3id.org/mixs","keywords":["diet","host","host."],"slot_uri":"MIXS:0000869","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"host_disease_stat":{"name":"host_disease_stat","annotations":{"Expected_value":{"tag":"Expected_value","value":"disease name or Disease Ontology term"}},"description":"List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text","title":"host disease status","in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["disease","host","host.","status"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000031","domain_of":["HostAssociated","MimsHostAssociated"]},"host_dry_mass":{"name":"host_dry_mass","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"kilogram, gram"}},"description":"Measurement of dry mass","title":"host dry mass","examples":[{"value":"500 gram"}],"from_schema":"https://w3id.org/mixs","keywords":["dry","host","host.","mass"],"slot_uri":"MIXS:0000257","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_fam_rel":{"name":"host_fam_rel","annotations":{"Expected_value":{"tag":"Expected_value","value":"relationship type;arbitrary identifier"}},"description":"Relationships to other hosts in the same study; can include multiple relationships","title":"host family relationship","from_schema":"https://w3id.org/mixs","keywords":["family","host","host.","relationship"],"string_serialization":"{text};{text}","slot_uri":"MIXS:0000872","domain_of":["HostAssociated","MimsHostAssociated"],"multivalued":true},"host_genotype":{"name":"host_genotype","description":"Observed genotype","title":"host genotype","from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000365","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_growth_cond":{"name":"host_growth_cond","description":"Literature reference giving growth conditions of the host","title":"host growth conditions","examples":[{"value":"https://academic.oup.com/icesjms/article/68/2/349/617247"}],"from_schema":"https://w3id.org/mixs","keywords":["condition","growth","host","host."],"slot_uri":"MIXS:0000871","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$|([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}|{text}$","interpolated":true,"partial_match":true}},"host_height":{"name":"host_height","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"centimeter, millimeter, meter"}},"description":"The height of subject","title":"host height","from_schema":"https://w3id.org/mixs","keywords":["height","host","host."],"slot_uri":"MIXS:0000264","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_last_meal":{"name":"host_last_meal","annotations":{"Expected_value":{"tag":"Expected_value","value":"content;duration"}},"description":"Content of last meal and time since feeding; can include multiple values","title":"host last meal","from_schema":"https://w3id.org/mixs","keywords":["host","host."],"string_serialization":"{text};{duration}","slot_uri":"MIXS:0000870","domain_of":["HostAssociated","MimsHostAssociated"],"multivalued":true},"host_length":{"name":"host_length","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"centimeter, millimeter, meter"}},"description":"The length of subject","title":"host length","examples":[{"value":"1 meter"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","length"],"slot_uri":"MIXS:0000256","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_life_stage":{"name":"host_life_stage","annotations":{"Expected_value":{"tag":"Expected_value","value":"stage"}},"description":"Description of life stage of host","title":"host life stage","from_schema":"https://w3id.org/mixs","keywords":["host","host.","life"],"string_serialization":"{text}","slot_uri":"MIXS:0000251","domain_of":["HostAssociated","MimsHostAssociated"]},"host_phenotype":{"name":"host_phenotype","annotations":{"Expected_value":{"tag":"Expected_value","value":"PATO or HP"}},"description":"Phenotype of human or other host. Use terms from the phenotypic quality ontology (pato) or the Human Phenotype Ontology (HP)","title":"host phenotype","from_schema":"https://w3id.org/mixs","keywords":["host","host."],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000874","domain_of":["HostAssociated","MimsHostAssociated"]},"host_sex":{"name":"host_sex","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Gender or physical sex of the host","title":"host sex","comments":["example of non-binary from Excel sheets does not match any of the enumerated values"],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"string_serialization":"[female|hermaphrodite|non-binary|male|transgender|transgender (female to male)|transgender (male to female)\\n|undeclared]","slot_uri":"MIXS:0000811","domain_of":["HostAssociated","MimsHostAssociated"]},"host_shape":{"name":"host_shape","description":"Morphological shape of host","title":"host shape","examples":[{"value":"round"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000261","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_subject_id":{"name":"host_subject_id","description":"A unique identifier by which each subject can be referred to, de-identified","title":"host subject id","from_schema":"https://w3id.org/mixs","keywords":["host","host.","identifier"],"slot_uri":"MIXS:0000861","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_subspecf_genlin":{"name":"host_subspecf_genlin","annotations":{"Expected_value":{"tag":"Expected_value","value":"Genetic lineage below lowest rank of NCBI taxonomy, which is subspecies, e.g. serovar, biotype, ecotype, variety, cultivar"}},"description":"Information about the genetic distinctness of the host organism below the subspecies level e.g., serovar, serotype, biotype, ecotype, variety, cultivar, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123","title":"host subspecific genetic lineage","examples":[{"value":"serovar:Newport, variety:glabrum, cultivar: Red Delicious"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","lineage"],"string_serialization":"{rank name}:{text}","slot_uri":"MIXS:0001318","domain_of":["HostAssociated","MimsHostAssociated"],"multivalued":true},"host_substrate":{"name":"host_substrate","description":"The growth substrate of the host","title":"host substrate","examples":[{"value":"rock"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000252","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_symbiont":{"name":"host_symbiont","annotations":{"Expected_value":{"tag":"Expected_value","value":"species name or common name"}},"description":"The taxonomic name of the organism(s) found living in mutualistic, commensalistic, or parasitic symbiosis with the specific host. The sampled symbiont can have its own symbionts. For example, parasites may have hyperparasites (=parasites of the parasite)","title":"observed host symbionts","from_schema":"https://w3id.org/mixs","keywords":["host","host.","observed","symbiosis"],"string_serialization":"{text}","slot_uri":"MIXS:0001298","domain_of":["HostAssociated","MimsHostAssociated"],"multivalued":true},"host_taxid":{"name":"host_taxid","annotations":{"Expected_value":{"tag":"Expected_value","value":"NCBI taxon identifier"}},"description":"NCBI taxon id of the host, e.g. 9606","title":"host taxid","from_schema":"https://w3id.org/mixs","keywords":["host","host.","taxon"],"slot_uri":"MIXS:0000250","domain_of":["HostAssociated","MimsHostAssociated"]},"host_tot_mass":{"name":"host_tot_mass","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"kilogram, gram"}},"description":"Total mass of the host at collection, the unit depends on host","title":"host total mass","from_schema":"https://w3id.org/mixs","keywords":["host","host.","mass","total"],"slot_uri":"MIXS:0000263","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"lat_lon":{"name":"lat_lon","annotations":{"Expected_value":{"tag":"Expected_value","value":"decimal degrees,  limit to 8 decimal points"}},"description":"The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system","title":"geographic location (latitude and longitude)","examples":[{"value":"50.586825 6.408977"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["geographic","location"],"string_serialization":"{float} {float}","slot_uri":"MIXS:0000009","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"required":true,"pattern":"^(-?((?:[0-8]?[0-9](?:\\\\.\\\\d{0,8})?)|90)) -?[0-9]+(?:\\\\.[0-9]{0,8})?$|^-?(1[0-7]{1,2})$","structured_pattern":{"syntax":"^{lat} {lon}$","interpolated":true,"partial_match":true}},"lib_layout":{"name":"lib_layout","description":"Specify whether to expect single, paired, or other configuration of reads","title":"library layout","examples":[{"value":"paired"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"slot_uri":"MIXS:0000041","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"LIB_LAYOUT_ENUM"},"lib_reads_seqd":{"name":"lib_reads_seqd","description":"Total number of clones sequenced from the library","title":"library reads sequenced","examples":[{"value":"20"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"slot_uri":"MIXS:0000040","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer"},"lib_screen":{"name":"lib_screen","annotations":{"Expected_value":{"tag":"Expected_value","value":"screening strategy name"}},"description":"Specific enrichment or screening methods applied before and/or after creating libraries","title":"library screening strategy","examples":[{"value":"enriched, screened, normalized"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"string_serialization":"{text}","slot_uri":"MIXS:0000043","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"]},"lib_size":{"name":"lib_size","description":"Total number of clones in the library prepared for the project","title":"library size","examples":[{"value":"50"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","size"],"slot_uri":"MIXS:0000039","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer"},"lib_vector":{"name":"lib_vector","annotations":{"Expected_value":{"tag":"Expected_value","value":"vector"}},"description":"Cloning vector type(s) used in construction of libraries","title":"library vector","examples":[{"value":"Bacteriophage P1"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"string_serialization":"{text}","slot_uri":"MIXS:0000042","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"]},"magnesium":{"name":"magnesium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"mole per liter, milligram per liter, parts per million, micromole per kilogram"}},"description":"Concentration of magnesium in the sample","title":"magnesium","examples":[{"value":"52.8 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000431","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"mean_frict_vel":{"name":"mean_frict_vel","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter per second"}},"description":"Measurement of mean friction velocity","title":"mean friction velocity","examples":[{"value":"0.5 meter per second"}],"from_schema":"https://w3id.org/mixs","keywords":["mean","velocity"],"slot_uri":"MIXS:0000498","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"mean_peak_frict_vel":{"name":"mean_peak_frict_vel","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter per second"}},"description":"Measurement of mean peak friction velocity","title":"mean peak friction velocity","examples":[{"value":"1 meter per second"}],"from_schema":"https://w3id.org/mixs","keywords":["mean","peak","velocity"],"slot_uri":"MIXS:0000502","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"methane":{"name":"methane","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, parts per billion, parts per million"}},"description":"Methane (gas) amount or concentration at the time of sampling","title":"methane","from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000101","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"mid":{"name":"mid","description":"Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to specifically tag unique samples in a sequencing run. Sequence should be reported in uppercase letters","title":"multiplex identifiers","examples":[{"value":"GTGAATAT"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["identifier"],"slot_uri":"MIXS:0000047","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[ACGTRKSYMWBHDVN]+$","structured_pattern":{"syntax":"^{ambiguous_nucleotides}$","interpolated":true,"partial_match":true}},"misc_param":{"name":"misc_param","annotations":{"Expected_value":{"tag":"Expected_value","value":"parameter name;measurement value"}},"description":"Any other measurement performed or parameter collected, that is not listed here","title":"miscellaneous parameter","examples":[{"value":"Bicarbonate ion concentration;2075 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","keywords":["parameter"],"string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000752","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"multivalued":true},"n_alkanes":{"name":"n_alkanes","annotations":{"Expected_value":{"tag":"Expected_value","value":"n-alkane name;measurement value"}},"description":"Concentration of n-alkanes; can include multiple n-alkanes","title":"n-alkanes","examples":[{"value":"n-hexadecane;100 milligram per liter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000503","domain_of":["Sediment","MimsSediment"],"multivalued":true},"neg_cont_type":{"name":"neg_cont_type","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration or text"}},"description":"The substance or equipment used as a negative control in an investigation","title":"negative control type","in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["type"],"slot_uri":"MIXS:0001321","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"NEG_CONT_TYPE_ENUM","recommended":true},"nitrate":{"name":"nitrate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of nitrate in the sample","title":"nitrate","examples":[{"value":"65 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrate"],"slot_uri":"MIXS:0000425","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"nitrite":{"name":"nitrite","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of nitrite in the sample","title":"nitrite","examples":[{"value":"0.5 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrite"],"slot_uri":"MIXS:0000426","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"nitro":{"name":"nitro","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of nitrogen (total)","title":"nitrogen","examples":[{"value":"4.2 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrogen"],"slot_uri":"MIXS:0000504","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"nucl_acid_amp":{"name":"nucl_acid_amp","description":"A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids","title":"nucleic acid amplification","examples":[{"value":"https://phylogenomics.me/protocols/16s-pcr-protocol/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000038","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"nucl_acid_ext":{"name":"nucl_acid_ext","description":"A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample","title":"nucleic acid extraction","examples":[{"value":"https://mobio.com/media/wysiwyg/pdfs/protocols/12888.pdf"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000037","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"number_contig":{"name":"number_contig","description":"Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG","title":"number of contigs","examples":[{"value":"40"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["number"],"slot_uri":"MIXS:0000060","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer"},"org_carb":{"name":"org_carb","description":"Concentration of organic carbon","title":"organic carbon","examples":[{"value":"1.5 microgram per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","organic"],"slot_uri":"MIXS:0000508","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"org_matter":{"name":"org_matter","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"microgram per liter"}},"description":"Concentration of organic matter","title":"organic matter","examples":[{"value":"1.75 milligram per cubic meter"}],"from_schema":"https://w3id.org/mixs","keywords":["organic"],"slot_uri":"MIXS:0000204","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"org_nitro":{"name":"org_nitro","description":"Concentration of organic nitrogen","title":"organic nitrogen","examples":[{"value":"4 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrogen","organic"],"slot_uri":"MIXS:0000205","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"organism_count":{"name":"organism_count","annotations":{"Expected_value":{"tag":"Expected_value","value":"organism name;measurement value;enumeration"}},"description":"Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)","title":"organism count","from_schema":"https://w3id.org/mixs","keywords":["count","organism"],"string_serialization":"{text};{float} {unit};[ATP|MPN|qPCR|other]","slot_uri":"MIXS:0000103","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"multivalued":true},"oxy_stat_samp":{"name":"oxy_stat_samp","description":"Oxygenation status of sample","title":"oxygenation status of sample","examples":[{"value":"aerobic"}],"from_schema":"https://w3id.org/mixs","keywords":["oxygen","sample","status"],"slot_uri":"MIXS:0000753","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"OXY_STAT_SAMP_ENUM"},"part_org_carb":{"name":"part_org_carb","description":"Concentration of particulate organic carbon","title":"particulate organic carbon","examples":[{"value":"1.92 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","organic","particle","particulate"],"slot_uri":"MIXS:0000515","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"particle_class":{"name":"particle_class","annotations":{"Expected_value":{"tag":"Expected_value","value":"particle name;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"micrometer"}},"description":"Particles are classified, based on their size, into six general categories:clay, silt, sand, gravel, cobbles, and boulders; should include amount of particle preceded by the name of the particle type; can include multiple values","title":"particle classification","from_schema":"https://w3id.org/mixs","keywords":["classification","particle"],"string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000206","domain_of":["Sediment","MimsSediment"],"multivalued":true},"perturbation":{"name":"perturbation","annotations":{"Expected_value":{"tag":"Expected_value","value":"perturbation type name;perturbation interval and duration"}},"description":"Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types","title":"perturbation","examples":[{"value":"antibiotic addition;R2/2018-05-11T14:30Z/2018-05-11T19:30Z/P1H30M"}],"from_schema":"https://w3id.org/mixs","keywords":["perturbation"],"string_serialization":"{text};{Rn/start_time/end_time/duration}","slot_uri":"MIXS:0000754","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"multivalued":true},"petroleum_hydrocarb":{"name":"petroleum_hydrocarb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of petroleum hydrocarbon","title":"petroleum hydrocarbon","examples":[{"value":"0.05 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["hydrocarbon","petroleum"],"slot_uri":"MIXS:0000516","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"ph":{"name":"ph","description":"Ph measurement of the sample, or liquid portion of sample, or aqueous phase of the fluid","title":"pH","examples":[{"value":"7.2"}],"from_schema":"https://w3id.org/mixs","keywords":["ph"],"slot_uri":"MIXS:0001001","domain_of":["Sediment","MimsSediment"],"range":"float"},"phaeopigments":{"name":"phaeopigments","annotations":{"Expected_value":{"tag":"Expected_value","value":"phaeopigment name;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per cubic meter"}},"description":"Concentration of phaeopigments; can include multiple phaeopigments","title":"phaeopigments","examples":[{"value":"2.5 milligram per cubic meter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000180","domain_of":["Sediment","MimsSediment"],"multivalued":true},"phosphate":{"name":"phosphate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of phosphate","title":"phosphate","examples":[{"value":"0.7 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["phosphate"],"slot_uri":"MIXS:0000505","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"phosplipid_fatt_acid":{"name":"phosplipid_fatt_acid","annotations":{"Expected_value":{"tag":"Expected_value","value":"phospholipid fatty acid name;measurement value"}},"description":"Concentration of phospholipid fatty acids; can include multiple values","title":"phospholipid fatty acid","examples":[{"value":"2.98 milligram per liter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000181","domain_of":["Sediment","MimsSediment"],"multivalued":true},"porosity":{"name":"porosity","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement value or range"},"Preferred_unit":{"tag":"Preferred_unit","value":"percentage"}},"description":"Porosity of deposited sediment is volume of voids divided by the total volume of sample","title":"porosity","from_schema":"https://w3id.org/mixs","keywords":["porosity"],"string_serialization":"{float} - {float} {unit}","slot_uri":"MIXS:0000211","domain_of":["Sediment","MimsSediment"]},"pos_cont_type":{"name":"pos_cont_type","description":"The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive","title":"positive control type","in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["type"],"string_serialization":"{term} or {text}","slot_uri":"MIXS:0001322","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"recommended":true},"potassium":{"name":"potassium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, parts per million"}},"description":"Concentration of potassium in the sample","title":"potassium","examples":[{"value":"463 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000430","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"pressure":{"name":"pressure","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"atmosphere"}},"description":"Pressure to which the sample is subject to, in atmospheres","title":"pressure","examples":[{"value":"50 atmosphere"}],"from_schema":"https://w3id.org/mixs","keywords":["pressure"],"slot_uri":"MIXS:0000412","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"project_name":{"name":"project_name","description":"Name of the project within which the sequencing was organized","title":"project name","examples":[{"value":"Forest soil metagenome"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["project"],"slot_uri":"MIXS:0000092","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"redox_potential":{"name":"redox_potential","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"millivolt"}},"description":"Redox potential, measured relative to a hydrogen cell, indicating oxidation or reduction potential","title":"redox potential","examples":[{"value":"300 millivolt"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000182","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"ref_biomaterial":{"name":"ref_biomaterial","description":"Primary publication if isolated before genome publication; otherwise, primary genome report","title":"reference for biomaterial","examples":[{"value":"doi:10.1016/j.syapm.2018.01.009"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000025","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"ref_db":{"name":"ref_db","annotations":{"Expected_value":{"tag":"Expected_value","value":"names, versions, and references of databases"}},"description":"List of database(s) used for ORF annotation, along with version number and reference to website or publication","title":"reference database(s)","examples":[{"value":"pVOGs;5;http://dmk-brain.ecn.uiowa.edu/pVOGs/ Grazziotin et al. 2017 doi:10.1093/nar/gkw975"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["database"],"string_serialization":"{database};{version};{reference}","slot_uri":"MIXS:0000062","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"]},"rel_to_oxygen":{"name":"rel_to_oxygen","description":"Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments","title":"relationship to oxygen","examples":[{"value":"aerobe"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["oxygen","relationship"],"slot_uri":"MIXS:0000015","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"REL_TO_OXYGEN_ENUM"},"salinity":{"name":"salinity","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"practical salinity unit, percentage"}},"description":"The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater","title":"salinity","examples":[{"value":"25 practical salinity unit"}],"from_schema":"https://w3id.org/mixs","keywords":["salinity"],"slot_uri":"MIXS:0000183","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_capt_status":{"name":"samp_capt_status","description":"Reason for the sample","title":"sample capture status","examples":[{"value":"farm sample"}],"from_schema":"https://w3id.org/mixs","keywords":["sample","status"],"slot_uri":"MIXS:0000860","domain_of":["HostAssociated","MimsHostAssociated"],"range":"SAMP_CAPT_STATUS_ENUM"},"samp_collect_device":{"name":"samp_collect_device","annotations":{"Expected_value":{"tag":"Expected_value","value":"device name"}},"description":"The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094)","title":"sample collection device","in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["device","sample"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000002","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"]},"samp_collect_method":{"name":"samp_collect_method","description":"The method employed for collecting the sample","title":"sample collection method","in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["method","sample"],"slot_uri":"MIXS:0001225","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$|([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}|{text}$","interpolated":true,"partial_match":true}},"samp_dis_stage":{"name":"samp_dis_stage","description":"Stage of the disease at the time of sample collection, e.g. inoculation, penetration, infection, growth and reproduction, dissemination of pathogen","title":"sample disease stage","examples":[{"value":"infection"}],"from_schema":"https://w3id.org/mixs","keywords":["disease","sample"],"slot_uri":"MIXS:0000249","domain_of":["HostAssociated","MimsHostAssociated"],"range":"SAMP_DIS_STAGE_ENUM"},"samp_mat_process":{"name":"samp_mat_process","description":"A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed","title":"sample material processing","examples":[{"value":"filtering of seawater, storing samples in ethanol"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["material","process","sample"],"slot_uri":"MIXS:0000016","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"samp_name":{"name":"samp_name","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":""}},"description":"A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name","title":"sample name","examples":[{"value":"ISDsoil1"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["sample"],"slot_uri":"MIXS:0001107","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"samp_size":{"name":"samp_size","description":"The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected","title":"amount or size of sample collected","examples":[{"value":"5 liter"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["sample","size"],"slot_uri":"MIXS:0000001","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_store_dur":{"name":"samp_store_dur","description":"Duration for which the sample was stored. Indicate the duration for which the sample was stored written in ISO 8601 format","title":"sample storage duration","examples":[{"value":"P1Y6M"}],"from_schema":"https://w3id.org/mixs","keywords":["duration","period","sample","storage"],"slot_uri":"MIXS:0000116","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^P(?:(?:\\\\d+D|\\\\d+M(?:\\\\d+D)?|\\\\d+Y(?:\\\\d+M(?:\\\\d+D)?)?)(?:T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S))?|T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S)|\\\\d+W)$","structured_pattern":{"syntax":"^{duration}$","interpolated":true,"partial_match":true}},"samp_store_loc":{"name":"samp_store_loc","annotations":{"Expected_value":{"tag":"Expected_value","value":"location name"}},"description":"Location at which sample was stored, usually name of a specific freezer/room","title":"sample storage location","examples":[{"value":"Freezer no:5"}],"from_schema":"https://w3id.org/mixs","keywords":["location","sample","storage"],"string_serialization":"{text}","slot_uri":"MIXS:0000755","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"]},"samp_store_temp":{"name":"samp_store_temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Temperature at which sample was stored, e.g. -80 degree Celsius","title":"sample storage temperature","examples":[{"value":"-80 degree Celsius"}],"from_schema":"https://w3id.org/mixs","keywords":["sample","storage","temperature"],"slot_uri":"MIXS:0000110","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_taxon_id":{"name":"samp_taxon_id","description":"NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use \'synthetic metagenome  for mock community/positive controls, or \'blank sample\' for negative controls","title":"taxonomy ID of DNA sample","examples":[{"value":"Gut Metagenome [NCBITaxon:749906]"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dna","identifier","sample","taxon"],"slot_uri":"MIXS:0001320","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[NCBITaxon:\\\\d+\\\\]$","structured_pattern":{"syntax":"^{text} \\\\[{NCBItaxon_id}\\\\]$","interpolated":true,"partial_match":true}},"samp_vol_we_dna_ext":{"name":"samp_vol_we_dna_ext","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milliliter, gram, milligram, square centimeter"}},"description":"Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001)","title":"sample volume or weight for DNA extraction","examples":[{"value":"1500 milliliter"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["dna","sample","volume","weight"],"slot_uri":"MIXS:0000111","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sediment_type":{"name":"sediment_type","description":"Information about the sediment type based on major constituents","title":"sediment type","examples":[{"value":"biogenous"}],"from_schema":"https://w3id.org/mixs","keywords":["sediment","type"],"slot_uri":"MIXS:0001078","domain_of":["Sediment","MimsSediment"],"range":"SEDIMENT_TYPE_ENUM"},"seq_meth":{"name":"seq_meth","description":"Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103)","title":"sequencing method","examples":[{"value":"454 Genome Sequencer FLX [OBI:0000702]"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["method"],"slot_uri":"MIXS:0000050","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)|(([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\])$","structured_pattern":{"syntax":"^{text}|({termLabel} \\\\[{termID}\\\\])$","interpolated":true,"partial_match":true}},"silicate":{"name":"silicate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of silicate","title":"silicate","examples":[{"value":"0.05 micromole per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000184","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sim_search_meth":{"name":"sim_search_meth","description":"Tool used to compare ORFs with database, along with version and cutoffs used","title":"similarity search method","examples":[{"value":"HMMER3;3.1b2;hmmsearch, cutoff of 50 on score"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["method"],"slot_uri":"MIXS:0000063","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"size_frac":{"name":"size_frac","annotations":{"Expected_value":{"tag":"Expected_value","value":"filter size value range"}},"description":"Filtering pore size used in sample preparation","title":"size fraction selected","examples":[{"value":"0-0.22 micrometer"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["fraction","size"],"string_serialization":"{float}-{float} {unit}","slot_uri":"MIXS:0000017","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"]},"sodium":{"name":"sodium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, parts per million"}},"description":"Sodium concentration in the sample","title":"sodium","examples":[{"value":"10.5 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000428","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sop":{"name":"sop","annotations":{"Expected_value":{"tag":"Expected_value","value":"reference to SOP"}},"description":"Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences","title":"relevant standard operating procedures","examples":[{"value":"http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/its/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["procedures"],"string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000090","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"multivalued":true},"source_mat_id":{"name":"source_mat_id","annotations":{"Expected_value":{"tag":"Expected_value","value":"for cultures of microorganisms: identifiers for two culture collections; for other material a unique arbitrary identifer"}},"description":"A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain \'UAM:Herps:14\' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2)","title":"source material identifiers","examples":[{"value":"MPI012345"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["identifier","material","source"],"string_serialization":"{text}","slot_uri":"MIXS:0000026","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"multivalued":true},"sulfate":{"name":"sulfate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of sulfate in the sample","title":"sulfate","examples":[{"value":"5 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["sulfate"],"slot_uri":"MIXS:0000423","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sulfide":{"name":"sulfide","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of sulfide in the sample","title":"sulfide","examples":[{"value":"2 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["sulfide"],"slot_uri":"MIXS:0000424","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tax_class":{"name":"tax_class","description":"Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes","title":"taxonomic classification","examples":[{"value":"vConTACT vContact2 (references from NCBI RefSeq v83, genus rank classification, default parameters)"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["classification","taxon"],"slot_uri":"MIXS:0000064","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"temp":{"name":"temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Temperature of the sample at the time of sampling","title":"temperature","examples":[{"value":"25 degree Celsius"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["temperature"],"slot_uri":"MIXS:0000113","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tidal_stage":{"name":"tidal_stage","description":"Stage of tide","title":"tidal stage","examples":[{"value":"high tide"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000750","domain_of":["Sediment","MimsSediment"],"range":"TIDAL_STAGE_ENUM"},"tot_carb":{"name":"tot_carb","description":"Total carbon content","title":"total carbon","from_schema":"https://w3id.org/mixs","keywords":["carbon","total"],"slot_uri":"MIXS:0000525","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tot_depth_water_col":{"name":"tot_depth_water_col","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Measurement of total depth of water column","title":"total depth of water column","examples":[{"value":"500 meter"}],"from_schema":"https://w3id.org/mixs","keywords":["depth","total","water"],"slot_uri":"MIXS:0000634","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tot_nitro_content":{"name":"tot_nitro_content","description":"Total nitrogen content of the sample","title":"total nitrogen content","examples":[{"value":"35 milligrams Nitrogen per kilogram of soil"}],"from_schema":"https://w3id.org/mixs","keywords":["content","nitrogen","total"],"slot_uri":"MIXS:0000530","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tot_org_carb":{"name":"tot_org_carb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"gram Carbon per kilogram sample material"}},"description":"Total organic carbon content","title":"total organic carbon","examples":[{"value":"0.02"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","organic","total"],"slot_uri":"MIXS:0000533","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"turbidity":{"name":"turbidity","description":"Measure of the amount of cloudiness or haziness in water caused by individual particles","title":"turbidity","examples":[{"value":"0.3 nephelometric turbidity units"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000191","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"water_content":{"name":"water_content","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"gram per gram or cubic centimeter per cubic centimeter"}},"description":"Water content measurement","title":"water content","from_schema":"https://w3id.org/mixs","keywords":["content","water"],"slot_uri":"MIXS:0000185","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"collection_context_name":{"name":"collection_context_name","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of institution"}},"description":"Name of where sample originated and is typically stored. Typically will be \'owning institution\'.","title":"context name where location was collected","examples":[{"value":"Natural History Museum London"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["storage"],"slot_uri":"MIXS:999999901","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"cultural_era":{"name":"cultural_era","annotations":{"Expected_value":{"tag":"Expected_value","value":"chronotology or PeriodO term; text"}},"description":"The cultural era approximating the period in which the individual lived from https://chronontology.dainst.org/ or PeriodO if possible. If not available, follow INSDC standard for reporting missing data (https://www.insdc.org/submitting-standards/missing-value-reporting/).","title":"cultural era or period","examples":[{"value":"Copper Age [Chronotology: NW6hofAScJSE]"},{"value":"Upper Middle Palaeolthic"},{"value":"Not collected"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient","age"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:999999902","domain_of":["Ancient","MimsHostAssociatedAncient"],"required":true,"recommended":true,"multivalued":false},"damage_treatment":{"name":"damage_treatment","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Indication of whether characteristic ancient DNA damage has been  altered or removed from a DNA extract in a laboratory","title":"Damage treatment type","examples":[{"value":"none"},{"value":"partial-udg"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"slot_uri":"MIXS:999999903","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"DamageTreatmentEnum","required":true,"multivalued":true},"dna_extraction_date":{"name":"dna_extraction_date","description":"The date when the nucleic acid extraction was started from the sample material. In case no exact time is available, the date can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant","title":"date of extraction of nucleic acids from sample","examples":[{"value":"2023-12-01"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999904","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"datetime","required":false,"recommended":true},"ethical_authority":{"name":"ethical_authority","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of authority providing ethical approval"}},"description":"Name of the authorit(ies) or institution(s) that awarded sampling and  analysis (e.g. human remains) and/or export permission  (e.g. animal remains)","title":"ethical authority","examples":[{"value":"University of Copenhagen"},{"value":"Federal Foreign Office (Germany)"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["ethics","location"],"slot_uri":"MIXS:999999905","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"ethical_date":{"name":"ethical_date","description":"Date of award of ethical/export permission. The date can be right truncated  i.e. all of these are valid times: 2008-01-23; 2008-01; 2008;  Except: 2008-01; 2008 all are ISO8601 compliant","title":"date of ethical approval","examples":[{"value":"2023-12-01"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999906","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"datetime","required":false,"recommended":true},"ethical_id":{"name":"ethical_id","description":"The permissions code, ID, document, or similar provided by any authority  (socal, legal, academic etc.) associated with approval of the analysis of  this particular sample.","title":"ethical permit or approval ID","examples":[{"value":"DE-123-JK"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"slot_uri":"MIXS:999999906","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"experimental_procedures":{"name":"experimental_procedures","description":"Provide a DOI or URL to refer to the paper where the field report, nucleic acid extraction,  library construction, and other procedures are explained in more detail, e.g. the paper reporting the data.","title":"experimental procedures","examples":[{"value":"doi:10.1093/nar/gkr771"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999907","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_concentration":{"name":"lib_concentration","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement value in number of ng per l"}},"description":"Concentration of library in ng per l, as inferred by qPCR, digital PCR, Qubit etc.","title":"library concentration","examples":[{"value":"2"},{"value":"0.5"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{float}","slot_uri":"MIXS:999999908","domain_of":["Ancient","MimsHostAssociatedAncient"],"multivalued":true},"lib_index_polymerase":{"name":"lib_index_polymerase","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of polymerase used during library indexing"}},"description":"The DNA polymerase enzyme used for indexing DNA libraries","title":"library index polymerase","examples":[{"value":"Agilent PfuTurbo Cx HotStart"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["polymerase","library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999909","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true},"lib_preparation_protocol":{"name":"lib_preparation_protocol","description":"Citation(s) for the DNA library preparation protocol","title":"library preparation protocols","examples":[{"value":"doi:10.1093/nar/gkr771"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999907","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_reamplification_polymerase":{"name":"lib_reamplification_polymerase","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of polymerase used during library indexing"}},"description":"The name of the DNA polymerase enzyme used for reamplifying DNA libraries post-indexing.","title":"library reamplification polymerase","examples":[{"value":"KAPA HiFi HotStart Uracil+"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["polymerase","library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999909","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true},"lib_type":{"name":"lib_type","description":"The type of library created, i.e., amplicon, enriched, or shotgun.  Amplicon based is a library that has been amplified to target a specific single region of a genome (e.g. a specific gene).  An enriched library is a library has had a particular genome or genome regions \'captured\' or enriched typically via baits or probes. A shotgun library has had no type of targeted enrichment for a particular gene or genome.","title":"library type","examples":[{"value":"amplicon"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"slot_uri":"MIXS:999999910","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"LibTypeEnum","required":false},"neg_cont_status":{"name":"neg_cont_status","description":"Specify whether the sample is a negative control or not.","title":"negative control status","examples":[{"value":"true"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["control"],"slot_uri":"MIXS:999999911","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"boolean"},"num_capture_amp_cycles":{"name":"num_capture_amp_cycles","description":"Number of amplification cycles after capture enrichment","title":"number of reamplification cycles after capture","examples":[{"value":"12"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999912","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer","multivalued":true},"num_reamp_cycles":{"name":"num_reamp_cycles","description":"Number of amplification cycles after library indexing PCR. If capture data, this refers to amplifications prior to the capture experiments.","title":"number of reamplification cycles","examples":[{"value":"12"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999913","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer"},"preservational_treatment":{"name":"preservational_treatment","description":"Description of any treatment applied to samples for the purpose of  maximising collection preservation that may influence downstream  DNA recovery or library construction, such as storage fluid or  reconstructive glue","title":"preservational treatment","examples":[{"value":"stored in formalin"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999914","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"prev_pubs":{"name":"prev_pubs","description":"Any publications that report data from the same body/skeleton/individual","title":"previous publications","examples":[{"value":"doi:10.1016/j.jas.2015.02.0181"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999915","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"recovery_date":{"name":"recovery_date","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"year"}},"description":"Date of excavation or retrieval from burial or depositional context, if known. In case no exact time is available, the date can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant. If excavations were done during a longer period, report its midpoint at a month or year level.","title":"date of retrieval from depositional context","examples":[{"value":"2023"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999916","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"datetime","required":false,"recommended":true},"samp_decontam_pretreat":{"name":"samp_decontam_pretreat","description":"Method(s) employed for surface decontamination of samples of external  modern DNA; Treatment used on the samples. Depends on the sample type.  More relevant for bones than environmental samples. E.g. buffers, EDTA, etc.","title":"sample decontamination pretreatment","examples":[{"value":"doi:10.1016/j.jas.2015.02.0181"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999917","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"samp_age_range_youngest_limit":{"name":"samp_age_range_youngest_limit","annotations":{"Expected_value":{"tag":"Expected_value","value":"age value corresponding to unit"}},"description":"The youngest possible age of the sample. Youngest refers to the time closest to present day. If a sample has only a single age value, the value from samp_age_range_oldest_limit can be repeated. Sample age refers to a period of time in the past, such as when an organism was living and then died, not the biological age.","title":"sample age range youngest limit","examples":[{"value":"100000"},{"value":"1700"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999918","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer","required":true,"recommended":true},"samp_age_range_youngest_unit":{"name":"samp_age_range_youngest_unit","description":"The unit of the youngest age in the sample age range.","title":"sample age range youngest unit","examples":[{"value":"cal BP"},{"value":"CE"},{"value":"Ma"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999919","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"SampAgeUnit","required":true,"recommended":true,"multivalued":true},"samp_age_range_oldest_limit":{"name":"samp_age_range_oldest_limit","annotations":{"Expected_value":{"tag":"Expected_value","value":"age value corresponding to unit"}},"description":"The oldest possible age of the sample. Oldest refers to the time furthest from present day. If a sample has only a single age value, the value from samp_age_range_youngest_limit can be repeated. Sample age refers to a period of time in the past, such as when an organism was living and then died, not the biological age.","title":"sample age range oldest limit","examples":[{"value":"120000"},{"value":"1900"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999920","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer","required":true,"recommended":true},"samp_age_range_oldest_unit":{"name":"samp_age_range_oldest_unit","description":"The unit of the oldest age in sample age range.","title":"sample age range oldest unit","examples":[{"value":"cal BP"},{"value":"CE"},{"value":"Ma"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999921","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"SampAgeUnit","required":true,"recommended":true,"multivalued":true},"samp_age_range_inference_methods":{"name":"samp_age_range_inference_methods","description":"The method used to infer the sample age. This corresponds to a fixed list of most commonly used dating methods, such as 14C, OSL, stratigraphic inference, historical records, other etc. If the method is not in the list, please provide a description in samp_age_range_inference_description.","title":"sample age inference methods","examples":[{"value":"radiocarbon dating, calibrated with OxCal v4.3"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999922","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"SampAgeInferMethod","required":true,"recommended":true,"multivalued":true},"samp_age_range_inference_description":{"name":"samp_age_range_inference_description","description":"Additional general information about the inference method used to infer the sample age that can be useful for  understanding how the sample age was estimated. This can be things such as whether radiocarbon age estimated on AMS or conventional radiocarbon dating, or which calibration confidence interval was used.  It can also be used to describe what artefacts were used to age the stratigraphic layer of historical contexts.","title":"sample age inference description","examples":[{"value":"radiocarbon dating, calibrated with OxCal v4.3 with 95% confidence interval"},{"value":"based on proxy dating from other bone samples of stratigraphic layer"},{"value":"a coin found in the burial was from the 3rd century"},{"value":"age taken from previous publication Doe et al. 2019"},{"value":"radiocarbon age ID: OxA-12345"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999923","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":false},"samp_alt_lab_ids":{"name":"samp_alt_lab_ids","description":"An alternative sample or material ID related to the sample not already covered by terms samp_name and source_mat_id, including from associated other non-genetic analyses of the same sample (e.g. museum IDs, internal lab sample ID from sampling such a bone drilling). Can be specified multiple times for different ID contexts.","title":"alternative sample IDs","examples":[{"value":"ABC_24"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:999999924","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"storage_conditions":{"name":"storage_conditions","description":"General conditions in which the material (before sampling for DNA analysis) was stored in long-term collection storage, that may influenced DNA recovery or library construction. For example, specify temperature, humidity, presence of microbial overgrowth etc..","title":"storage conditions","examples":[{"value":"climate-controlled"},{"value":"Stored in the museum from recovery_date to collection_date"},{"value":"Stored at -20oC until dna_extraction_date"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:999999925","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string"},"palaeopath_status":{"name":"palaeopath_status","annotations":{"Expected_value":{"tag":"Expected_value","value":"description of health related observation on ancient remains"}},"description":"Describe briefly any relevant palaeopathological or health-related observations of the remains of the individual under study.","title":"palaeopathology status.","examples":[{"value":"Osteoporosis."},{"value":"Parasites found in pelvic area."},{"value":"Caries on right upper molar."}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["palaeopathology","host health","ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999926","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":false},"biocultural_label":{"name":"biocultural_label","annotations":{"Expected_value":{"tag":"Expected_value","value":"Relevant biocultural label from https://localcontexts.org/labels/biocultural-labels/"}},"description":"Relevant biocultural labels that apply to the sample under study, based on https://localcontexts.org/label/bc-provenance/. These help indicate how indigenous peoples and similar communities view in which contexts this data can be reused.","title":"biocultural label","examples":[{"value":"BC R"},{"value":"BC MC"}],"in_subset":["ethics"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"slot_uri":"MIXS:999999927","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"BioCulturalLabel","required":false,"recommended":false,"multivalued":true},"ethics_perm_scope":{"name":"ethics_perm_scope","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the scope of ethical permissions for data use"}},"description":"Description of the original scope and permissions of the research on the genetic material, as was approved by the ethical authority (e.g. bacteria only, DNA only, bacteria and human, no host read analysis allowed).","title":"ethical Permissions & Scope","examples":[{"value":"Defined scope only includes the study of bacterial sequences and any human sequence is not covered under the agreement."}],"in_subset":["ethics"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"string_serialization":"{text}","slot_uri":"MIXS:999999928","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"host_body_preservation_state":{"name":"host_body_preservation_state","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the preservation of the sampled (ancient) organism/host"}},"description":"Description of the state of the sampled (ancient) organism/host as originally preserved in the burial environment, e.g. if the the sample was artificially or naturally (fully/partially) mummified at the time of death.","title":"ancient body preservation state","examples":[{"value":"complete artificial mummification."},{"value":"natural partial mummification."},{"value":"fully skeletonised."}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999929","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":false},"lib_strandedness":{"name":"lib_strandedness","annotations":{"Expected_value":{"tag":"Expected_value","value":"DNA library strandedness"}},"description":"The strandedness of the original template DNA molecules used for constructing the sequencing library","title":"DNA strandedness in library creation","examples":[{"value":"single"},{"value":"double"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"slot_uri":"MIXS:999999930","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"LibStrand","required":false,"recommended":true,"multivalued":true},"enrichment_probe_set":{"name":"enrichment_probe_set","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of probe set used for target enrichment/library selection"}},"description":"Description of target enrichment probe designs used (e.g., species included, sequences, type, company). This can include custom kits (please provide a general description and DOI if available) or commercially available kit (provide ID and company).","title":"enrichment probe design description","examples":[{"value":"Custom probe design (70 bp biotinylated RNA probes) covering 500 full E. Coli genomes; company TE-Science"},{"value":"Commercial kit ABC001-EC from company TE-Science, purchased 2020"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","enrichment"],"string_serialization":"{text}","slot_uri":"MIXS:999999931","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":true},"mid_config":{"name":"mid_config","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the indexing configuration of the library"}},"description":"Index/barcode/primer configuration used during library building for sequencing. This includes information such as the number, type and location of indexes, the index/primer kit/list, or if \'inline\' barcodes or UMIs were ligated directly onto the template molecules.","title":"multiplex identifiers or indexing configuration of libraries","examples":[{"value":"dual index with single internal barcode."},{"value":"UDI index sequences."}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999932","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"c14_date":{"name":"c14_date","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"years before present (yr BP)"}},"description":"The uncalibrated date from the laboratory measurement. Should be a conventional 14C date (i.e., 14C yr BP) NOT in AD/BC format.  This is typically the \'raw\' date reported by the radiocarbon lab, in Before Present (BP) notation.","title":"Radiocarbon date","examples":[{"value":"1453"},{"value":"12560"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999971","domain_of":["RadiocarbonDating"],"range":"integer","required":true,"recommended":true,"multivalued":false},"c14_date_sd":{"name":"c14_date_sd","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"years before present (yr BP)"}},"description":"The one standard deviation around the \'raw\' C14 Date measurement, normally indicated as a  after the main date. Must be in the same format (i.e., y BP).","title":"Radiocarbon date standard deviation","examples":[{"value":"120"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999972","domain_of":["RadiocarbonDating"],"range":"integer","required":true,"recommended":true,"multivalued":false},"c14_date_lab_code":{"name":"c14_date_lab_code","annotations":{"Expected_value":{"tag":"Expected_value","value":"unique identifier"}},"description":"Unique laboratory identification number which is a combination of a lab identifier prefix, and a number. The prefix should be derived from: https://radiocarbon.webhost.uits.arizona.edu/laboratories.","title":"Laboratory code for the date determination","examples":[{"value":"OxA-1539"},{"value":"KN-1369"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{text}-{integer}","slot_uri":"MIXS:99999973","domain_of":["RadiocarbonDating"],"range":"string","required":true,"recommended":true,"multivalued":false},"sample_material":{"name":"sample_material","annotations":{"Expected_value":{"tag":"Expected_value","value":"text"}},"description":"Material of the sample used to extract carbon used for radiocarbon dating measurements. Use ontology terms where possible, e.g. from UBERON for anatomical parts, or ENVO for other organic samples.","title":"Radiocarbon dating sample material","examples":[{"value":"charcoal [ENVO_01000560]"},{"value":"molar tooth [UBERON_0003655]"},{"value":"wood fragment"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:99999974","domain_of":["RadiocarbonDating"],"range":"string","required":false,"recommended":true,"multivalued":false},"delta_13_c":{"name":"delta_13_c","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"per mil ()"}},"description":"The delta carbon-13 value of the sample (13C), which is the ratio of the stable isotope 13C to 12C, expressed in per mil () notation. Used as a quality control measurement.","title":"Delta 13C value","examples":[{"value":"-25.3"},{"value":"-20.1"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{float}","slot_uri":"MIXS:99999976","domain_of":["RadiocarbonDating"],"required":true,"recommended":true,"multivalued":false},"delta_13_c_method":{"name":"delta_13_c_method","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Which spectrophotometry method was used to measure the delta carbon-13 value, either with  Isotope Ratio Mass Spectrometer (IRMS) or Accelerated Mass Spectrometer (AMS).","title":"Delta 13C measurement method","examples":[{"value":"ams"},{"value":"imrs"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{text}","slot_uri":"MIXS:99999977","domain_of":["RadiocarbonDating"],"range":"MsMethod","required":false,"recommended":true},"carbon_perc":{"name":"carbon_perc","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"percent (%)"}},"description":"The percentage of carbon in a non-proteinaceous sample used for dating (such as charcoal), expressed as a percentage (%). Used as a quality control measurement.","title":"Carbon percentage","examples":[{"value":"50.3"},{"value":"45.1"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{float} {unit}","slot_uri":"MIXS:99999978","domain_of":["RadiocarbonDating"],"range":"float","required":true,"recommended":true,"multivalued":false},"calib_age_youngest":{"name":"calib_age_youngest","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"calibrated years before present (cal BP)"}},"description":"Upper (youngest) date in the range of a calibrated age. Calendar timescale to be specified: AD/BC, CE/BCE, cal BP. The abbreviation BP is to be used for uncalibrated 14C determinations, cal BP must be used here.","title":"Radiocarbon calibration age upper limit","examples":[{"value":"1449"},{"value":"12560"},{"value":"1500"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999979","domain_of":["RadiocarbonDating"],"range":"integer","required":false,"recommended":true,"multivalued":true},"calib_age_median":{"name":"calib_age_median","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"calibrated years before present (cal BP)"}},"description":"Median date in the range of a calibrated age. Calendar timescale to be specified: AD/BC, CE/BCE, cal BP. The abbreviation BP is to be used for uncalibrated 14C determinations, cal BP must be used here.","title":"Radiocarbon calibration age upper limit","examples":[{"value":"1450"},{"value":"15700"},{"value":"1473"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999979","domain_of":["RadiocarbonDating"],"range":"integer","required":false,"recommended":true,"multivalued":true},"calib_age_oldest":{"name":"calib_age_oldest","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"calibrated years before present (cal BP)"}},"description":"Lower (oldest) date in the range of a calibrated age. Calendar timescale to be specified: AD/BC, CE/BCE, cal BP. Note: the abbreviation \'BP\' is to be used for uncalibrated 14C determinations, cal BP must be used here.","title":"Radiocarbon calibration age lower limit","examples":[{"value":"1451"},{"value":"18560"},{"value":"1453"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999980","domain_of":["RadiocarbonDating"],"range":"integer","required":false,"recommended":true,"multivalued":true},"calib_age_unit":{"name":"calib_age_unit","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Unit of the calibrated median date and calibrated range, i.e., whether in  cal AD/BC, cal CE/BCE, cal BP. Note the abbreviation BP is to be used for uncalibrated 14C determinations only, therefore cal BP must be used here.","title":"Unit of radiocarbon date calibration","examples":[{"value":"cal BP"},{"value":"cal AD"},{"value":"cal BCE"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{text}","slot_uri":"MIXS:99999981","domain_of":["RadiocarbonDating"],"range":"RadiocarbonCalibUnit","required":false,"recommended":true,"multivalued":true},"calib_age_sigma":{"name":"calib_age_sigma","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Confidence level of the reported calibrated age range, typically in 2 (95.4%) or 1 (68.2%) levels of significance (sigma).","title":"Radiocarbon calibration age confidence level","examples":[{"value":"1"},{"value":"2"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999982","domain_of":["RadiocarbonDating"],"range":"ConfidenceSigma","required":false,"recommended":true,"multivalued":true},"calib_curve":{"name":"calib_curve","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Name of calibration curve used during radiocarbon date calibration. If a Marine curve is selected, the localised reservoir offset should be specified.","title":"Radiocarbon calibration curve name","examples":[{"value":"IntCal20"},{"value":"SHCal20"},{"value":"Bomb21NH3"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{text}","slot_uri":"MIXS:99999983","domain_of":["RadiocarbonDating"],"range":"CalibCurve","required":false,"recommended":true,"multivalued":true},"calib_software":{"name":"calib_software","annotations":{"Expected_value":{"tag":"Expected_value","value":"text"}},"description":"The name of the software used to generated the calibrated version of the laboratory measurement date.","title":"Radiocarbon calibration software","examples":[{"value":"OxCal"},{"value":"CALIB"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{text}","slot_uri":"MIXS:99999984","domain_of":["RadiocarbonDating"],"range":"string","required":false,"recommended":true,"multivalued":true},"calib_version":{"name":"calib_version","annotations":{"Expected_value":{"tag":"Expected_value","value":"version"}},"description":"Version of the calibration software used for calibration.","title":"Radiocarbon calibration software version","examples":[{"value":"1.0.3"},{"value":"4.4"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{text}","slot_uri":"MIXS:99999985","domain_of":["RadiocarbonDating"],"range":"string","required":false,"recommended":true,"multivalued":true},"calib_settings":{"name":"calib_settings","annotations":{"Expected_value":{"tag":"Expected_value","value":"text"}},"description":"Settings used with the calibration software, including citations to age models used (when applicable).","title":"Radiocarbon calibration software settings","examples":[{"value":"Bayesian model"},{"value":"-m 10"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{text}","slot_uri":"MIXS:99999986","domain_of":["RadiocarbonDating"],"range":"string","required":false,"recommended":true,"multivalued":true},"calib_citation":{"name":"calib_citation","annotations":{"Expected_value":{"tag":"Expected_value","value":"text"}},"description":"The DOI, URL or citation information of the publication where the calibration software was originally described.","title":"Radiocarbon calibration software citation","examples":[{"value":"doi:10.1017/S0033822200033865"},{"value":"Ramsey 2009, Radiocarbon"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"slot_uri":"MIXS:99999987","domain_of":["RadiocarbonDating"],"range":"string","required":false,"recommended":true,"multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$|([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}|{text}$"}},"localised_reservoir_offset_value":{"name":"localised_reservoir_offset_value","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"years before present (yr BP)"}},"description":"The Delta R (R) value used modify the calibration curve to account for regional/localised marine reservoir effect.","title":"Reservoir offset","examples":[{"value":"49 yr BP"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999988","domain_of":["RadiocarbonDating"],"range":"integer","required":false,"recommended":false,"multivalued":true},"localised_reservoir_offset_sd":{"name":"localised_reservoir_offset_sd","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"years before present (yr BP)"}},"description":"The standard deviation of averaged Delta R (R) values used modify the calibration curve to account for regional/localised marine reservoir effect.","title":"Reservoir offset standard deviation","examples":[{"value":"3 yr BP"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999989","domain_of":["RadiocarbonDating"],"range":"integer","required":false,"recommended":false,"multivalued":true},"mims_host_associated_ancient_data":{"name":"mims_host_associated_ancient_data","description":"Data that comply with Ancient combined with MimsHostAssociated","title":"MimsHostAssociatedAncient data","from_schema":"https://w3id.org/mixs","domain":"MixsCompliantData","slot_uri":"MIXS:mims_hostassociated_ancient_data","domain_of":["MixsCompliantData"],"range":"MimsHostAssociatedAncient","multivalued":true,"inlined":true,"inlined_as_list":true},"mims_sediment_ancient_data":{"name":"mims_sediment_ancient_data","description":"Data that comply with Ancient combined with MimsSediment","title":"MimsSedimentAncient data","from_schema":"https://w3id.org/mixs","domain":"MixsCompliantData","slot_uri":"MIXS:mims_sediment_ancient_data","domain_of":["MixsCompliantData"],"range":"MimsSedimentAncient","multivalued":true,"inlined":true,"inlined_as_list":true}},"classes":{"dh_interface":{"name":"dh_interface","description":"A DataHarmonizer interface","from_schema":"https://w3id.org/mixs"},"MInAS-checklists":{"name":"MInAS-checklists","description":"The Minimum Information about any Ancient Sequence (MInAS) project aims to develop standardised metadata reporting schemes of ancient DNA samples and sequencing data via community-based consensus and training. This checklist contains relevant aDNA MIxS combinations plus the ancient and radiocarbon extension","from_schema":"https://w3id.org/mixs","is_a":"dh_interface"},"Mims":{"name":"Mims","description":"Metagenome or Environmental","title":"MIMS","from_schema":"https://w3id.org/mixs","aliases":["mims"],"is_a":"Checklist","mixin":true,"slot_usage":{"adapters":{"name":"adapters","recommended":true},"alt":{"name":"alt","recommended":true},"annot":{"name":"annot","recommended":true},"assembly_name":{"name":"assembly_name","recommended":true},"assembly_qual":{"name":"assembly_qual","recommended":true},"assembly_software":{"name":"assembly_software","recommended":true},"depth":{"name":"depth","examples":[{"value":"10 meter"}],"recommended":true},"elev":{"name":"elev","recommended":true},"experimental_factor":{"name":"experimental_factor","recommended":true},"feat_pred":{"name":"feat_pred"},"lib_layout":{"name":"lib_layout","recommended":true},"lib_reads_seqd":{"name":"lib_reads_seqd","recommended":true},"lib_screen":{"name":"lib_screen","recommended":true},"lib_size":{"name":"lib_size","recommended":true},"lib_vector":{"name":"lib_vector","recommended":true},"mid":{"name":"mid","recommended":true},"nucl_acid_amp":{"name":"nucl_acid_amp","recommended":true},"nucl_acid_ext":{"name":"nucl_acid_ext","recommended":true},"number_contig":{"name":"number_contig","recommended":true},"ref_biomaterial":{"name":"ref_biomaterial"},"ref_db":{"name":"ref_db"},"rel_to_oxygen":{"name":"rel_to_oxygen"},"samp_collect_device":{"name":"samp_collect_device","examples":[{"value":"swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713]"}],"recommended":true},"samp_collect_method":{"name":"samp_collect_method","examples":[{"value":"swabbing"}],"recommended":true},"samp_mat_process":{"name":"samp_mat_process","recommended":true},"samp_size":{"name":"samp_size","recommended":true},"samp_vol_we_dna_ext":{"name":"samp_vol_we_dna_ext"},"sim_search_meth":{"name":"sim_search_meth"},"size_frac":{"name":"size_frac"},"sop":{"name":"sop","recommended":true},"source_mat_id":{"name":"source_mat_id","recommended":true},"tax_class":{"name":"tax_class"},"temp":{"name":"temp","recommended":true}},"attributes":{"samp_name":{"name":"samp_name","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":""}},"description":"A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name","title":"sample name","examples":[{"value":"ISDsoil1"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["sample"],"slot_uri":"MIXS:0001107","alias":"samp_name","owner":"Mims","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"size_frac":{"name":"size_frac","annotations":{"Expected_value":{"tag":"Expected_value","value":"filter size value range"}},"description":"Filtering pore size used in sample preparation","title":"size fraction selected","examples":[{"value":"0-0.22 micrometer"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["fraction","size"],"string_serialization":"{float}-{float} {unit}","slot_uri":"MIXS:0000017","alias":"size_frac","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"lib_screen":{"name":"lib_screen","annotations":{"Expected_value":{"tag":"Expected_value","value":"screening strategy name"}},"description":"Specific enrichment or screening methods applied before and/or after creating libraries","title":"library screening strategy","examples":[{"value":"enriched, screened, normalized"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"string_serialization":"{text}","slot_uri":"MIXS:0000043","alias":"lib_screen","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"ref_db":{"name":"ref_db","annotations":{"Expected_value":{"tag":"Expected_value","value":"names, versions, and references of databases"}},"description":"List of database(s) used for ORF annotation, along with version number and reference to website or publication","title":"reference database(s)","examples":[{"value":"pVOGs;5;http://dmk-brain.ecn.uiowa.edu/pVOGs/ Grazziotin et al. 2017 doi:10.1093/nar/gkw975"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["database"],"string_serialization":"{database};{version};{reference}","slot_uri":"MIXS:0000062","alias":"ref_db","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"nucl_acid_amp":{"name":"nucl_acid_amp","description":"A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids","title":"nucleic acid amplification","examples":[{"value":"https://phylogenomics.me/protocols/16s-pcr-protocol/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000038","alias":"nucl_acid_amp","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_size":{"name":"lib_size","description":"Total number of clones in the library prepared for the project","title":"library size","examples":[{"value":"50"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","size"],"slot_uri":"MIXS:0000039","alias":"lib_size","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer","recommended":true},"mid":{"name":"mid","description":"Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to specifically tag unique samples in a sequencing run. Sequence should be reported in uppercase letters","title":"multiplex identifiers","examples":[{"value":"GTGAATAT"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["identifier"],"slot_uri":"MIXS:0000047","alias":"mid","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[ACGTRKSYMWBHDVN]+$","structured_pattern":{"syntax":"^{ambiguous_nucleotides}$","interpolated":true,"partial_match":true}},"assembly_name":{"name":"assembly_name","annotations":{"Expected_value":{"tag":"Expected_value","value":"name and version of assembly"}},"description":"Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community","title":"assembly name","examples":[{"value":"HuRef, JCVI_ISG_i3_1.0"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text} {text}","slot_uri":"MIXS:0000057","alias":"assembly_name","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"temp":{"name":"temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Temperature of the sample at the time of sampling","title":"temperature","examples":[{"value":"25 degree Celsius"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["temperature"],"slot_uri":"MIXS:0000113","alias":"temp","owner":"Mims","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"nucl_acid_ext":{"name":"nucl_acid_ext","description":"A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample","title":"nucleic acid extraction","examples":[{"value":"https://mobio.com/media/wysiwyg/pdfs/protocols/12888.pdf"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000037","alias":"nucl_acid_ext","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"samp_size":{"name":"samp_size","description":"The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected","title":"amount or size of sample collected","examples":[{"value":"5 liter"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["sample","size"],"slot_uri":"MIXS:0000001","alias":"samp_size","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"alt":{"name":"alt","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth\'s surface. In this context, the altitude measurement is the vertical distance between the earth\'s surface above sea level and the sampled position in the air","title":"altitude","examples":[{"value":"100 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000094","alias":"alt","owner":"Mims","domain_of":["Mims","HostAssociated","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"source_mat_id":{"name":"source_mat_id","annotations":{"Expected_value":{"tag":"Expected_value","value":"for cultures of microorganisms: identifiers for two culture collections; for other material a unique arbitrary identifer"}},"description":"A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain \'UAM:Herps:14\' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2)","title":"source material identifiers","examples":[{"value":"MPI012345"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["identifier","material","source"],"string_serialization":"{text}","slot_uri":"MIXS:0000026","alias":"source_mat_id","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true},"samp_vol_we_dna_ext":{"name":"samp_vol_we_dna_ext","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milliliter, gram, milligram, square centimeter"}},"description":"Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001)","title":"sample volume or weight for DNA extraction","examples":[{"value":"1500 milliliter"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["dna","sample","volume","weight"],"slot_uri":"MIXS:0000111","alias":"samp_vol_we_dna_ext","owner":"Mims","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"lib_reads_seqd":{"name":"lib_reads_seqd","description":"Total number of clones sequenced from the library","title":"library reads sequenced","examples":[{"value":"20"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"slot_uri":"MIXS:0000040","alias":"lib_reads_seqd","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer","recommended":true},"rel_to_oxygen":{"name":"rel_to_oxygen","description":"Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments","title":"relationship to oxygen","examples":[{"value":"aerobe"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["oxygen","relationship"],"slot_uri":"MIXS:0000015","alias":"rel_to_oxygen","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"REL_TO_OXYGEN_ENUM"},"samp_collect_device":{"name":"samp_collect_device","annotations":{"Expected_value":{"tag":"Expected_value","value":"device name"}},"description":"The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094)","title":"sample collection device","examples":[{"value":"swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713]"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["device","sample"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000002","alias":"samp_collect_device","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"number_contig":{"name":"number_contig","description":"Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG","title":"number of contigs","examples":[{"value":"40"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["number"],"slot_uri":"MIXS:0000060","alias":"number_contig","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer","recommended":true},"lib_layout":{"name":"lib_layout","description":"Specify whether to expect single, paired, or other configuration of reads","title":"library layout","examples":[{"value":"paired"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"slot_uri":"MIXS:0000041","alias":"lib_layout","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"LIB_LAYOUT_ENUM","recommended":true},"assembly_qual":{"name":"assembly_qual","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling   90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling < 90% of the expected genome or replicon sequence, or for which no genome size could be estimated","title":"assembly quality","examples":[{"value":"High-quality draft genome"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["quality"],"string_serialization":"[Finished genome|High-quality draft genome|Medium-quality draft genome|Low-quality draft genome|Genome fragment(s)]","slot_uri":"MIXS:0000056","alias":"assembly_qual","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"ref_biomaterial":{"name":"ref_biomaterial","description":"Primary publication if isolated before genome publication; otherwise, primary genome report","title":"reference for biomaterial","examples":[{"value":"doi:10.1016/j.syapm.2018.01.009"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000025","alias":"ref_biomaterial","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"project_name":{"name":"project_name","description":"Name of the project within which the sequencing was organized","title":"project name","examples":[{"value":"Forest soil metagenome"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["project"],"slot_uri":"MIXS:0000092","alias":"project_name","owner":"Mims","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"lib_vector":{"name":"lib_vector","annotations":{"Expected_value":{"tag":"Expected_value","value":"vector"}},"description":"Cloning vector type(s) used in construction of libraries","title":"library vector","examples":[{"value":"Bacteriophage P1"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"string_serialization":"{text}","slot_uri":"MIXS:0000042","alias":"lib_vector","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"adapters":{"name":"adapters","annotations":{"Expected_value":{"tag":"Expected_value","value":"adapter A and B sequence"}},"description":"Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters","title":"adapters","examples":[{"value":"AATGATACGGCGACCACCGAGATCTACACGCT;CAAGCAGAAGACGGCATACGAGAT"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{dna};{dna}","slot_uri":"MIXS:0000048","alias":"adapters","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"neg_cont_type":{"name":"neg_cont_type","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration or text"}},"description":"The substance or equipment used as a negative control in an investigation","title":"negative control type","in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["type"],"slot_uri":"MIXS:0001321","alias":"neg_cont_type","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"NEG_CONT_TYPE_ENUM","recommended":true},"assembly_software":{"name":"assembly_software","description":"Tool(s) used for assembly, including version number and parameters","title":"assembly software","examples":[{"value":"metaSPAdes;3.11.0;kmer set 21,33,55,77,99,121, default parameters otherwise"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["software"],"slot_uri":"MIXS:0000058","alias":"assembly_software","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"annot":{"name":"annot","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of tool or pipeline used, or annotation source description"}},"description":"Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter","title":"annotation","examples":[{"value":"prokka"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:0000059","alias":"annot","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"pos_cont_type":{"name":"pos_cont_type","description":"The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive","title":"positive control type","in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["type"],"string_serialization":"{term} or {text}","slot_uri":"MIXS:0001322","alias":"pos_cont_type","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"feat_pred":{"name":"feat_pred","description":"Method used to predict UViGs features such as ORFs, integration site, etc","title":"feature prediction","examples":[{"value":"Prodigal;2.6.3;default parameters"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["feature","predict"],"slot_uri":"MIXS:0000061","alias":"feat_pred","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"env_local_scale":{"name":"env_local_scale","annotations":{"Expected_value":{"tag":"Expected_value","value":"Environmental entities having causal influences upon the entity at time of sampling"}},"description":"Report the entity or entities which are in the sample or specimen s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS","title":"local environmental context","examples":[{"value":"hillside [ENVO:01000333]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["context","environmental"],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000013","alias":"env_local_scale","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"samp_mat_process":{"name":"samp_mat_process","description":"A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed","title":"sample material processing","examples":[{"value":"filtering of seawater, storing samples in ethanol"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["material","process","sample"],"slot_uri":"MIXS:0000016","alias":"samp_mat_process","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"sim_search_meth":{"name":"sim_search_meth","description":"Tool used to compare ORFs with database, along with version and cutoffs used","title":"similarity search method","examples":[{"value":"HMMER3;3.1b2;hmmsearch, cutoff of 50 on score"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["method"],"slot_uri":"MIXS:0000063","alias":"sim_search_meth","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"depth":{"name":"depth","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"The vertical distance below local surface. For sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples","title":"depth","examples":[{"value":"10 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["depth"],"slot_uri":"MIXS:0000018","alias":"depth","owner":"Mims","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_collect_method":{"name":"samp_collect_method","description":"The method employed for collecting the sample","title":"sample collection method","examples":[{"value":"swabbing"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["method","sample"],"slot_uri":"MIXS:0001225","alias":"samp_collect_method","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$|([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}|{text}$","interpolated":true,"partial_match":true}},"env_medium":{"name":"env_medium","description":"Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of \'environmental material\' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top)","title":"environmental medium","examples":[{"value":"bluegrass field soil [ENVO:00005789]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["environmental"],"slot_uri":"MIXS:0000014","alias":"env_medium","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\]$","structured_pattern":{"syntax":"^{termLabel} \\\\[{termID}\\\\]$","interpolated":true,"partial_match":true}},"samp_taxon_id":{"name":"samp_taxon_id","description":"NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use \'synthetic metagenome  for mock community/positive controls, or \'blank sample\' for negative controls","title":"taxonomy ID of DNA sample","examples":[{"value":"Gut Metagenome [NCBITaxon:749906]"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dna","identifier","sample","taxon"],"slot_uri":"MIXS:0001320","alias":"samp_taxon_id","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[NCBITaxon:\\\\d+\\\\]$","structured_pattern":{"syntax":"^{text} \\\\[{NCBItaxon_id}\\\\]$","interpolated":true,"partial_match":true}},"geo_loc_name":{"name":"geo_loc_name","description":"The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)","title":"geographic location (country and/or sea,region)","examples":[{"value":"USA: Maryland, Bethesda"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["geographic","location"],"slot_uri":"MIXS:0000010","alias":"geo_loc_name","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+): ([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+), ([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{text}: {text}, {text}$","interpolated":true,"partial_match":true}},"collection_date":{"name":"collection_date","description":"The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant","title":"collection date","examples":[{"value":"2013-03-25T12:42:31+01:00"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["date"],"slot_uri":"MIXS:0000011","alias":"collection_date","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"datetime","required":true},"seq_meth":{"name":"seq_meth","description":"Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103)","title":"sequencing method","examples":[{"value":"454 Genome Sequencer FLX [OBI:0000702]"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["method"],"slot_uri":"MIXS:0000050","alias":"seq_meth","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)|(([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\])$","structured_pattern":{"syntax":"^{text}|({termLabel} \\\\[{termID}\\\\])$","interpolated":true,"partial_match":true}},"lat_lon":{"name":"lat_lon","annotations":{"Expected_value":{"tag":"Expected_value","value":"decimal degrees,  limit to 8 decimal points"}},"description":"The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system","title":"geographic location (latitude and longitude)","examples":[{"value":"50.586825 6.408977"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["geographic","location"],"string_serialization":"{float} {float}","slot_uri":"MIXS:0000009","alias":"lat_lon","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^(-?((?:[0-8]?[0-9](?:\\\\.\\\\d{0,8})?)|90)) -?[0-9]+(?:\\\\.[0-9]{0,8})?$|^-?(1[0-7]{1,2})$","structured_pattern":{"syntax":"^{lat} {lon}$","interpolated":true,"partial_match":true}},"elev":{"name":"elev","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth\'s surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit","title":"elevation","examples":[{"value":"100 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["elevation"],"slot_uri":"MIXS:0000093","alias":"elev","owner":"Mims","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"env_broad_scale":{"name":"env_broad_scale","description":"Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS","title":"broad-scale environmental context","examples":[{"value":"rangeland biome [ENVO:01000247]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["context","environmental"],"slot_uri":"MIXS:0000012","alias":"env_broad_scale","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\]$","structured_pattern":{"syntax":"^{termLabel} \\\\[{termID}\\\\]$","interpolated":true,"partial_match":true}},"tax_class":{"name":"tax_class","description":"Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes","title":"taxonomic classification","examples":[{"value":"vConTACT vContact2 (references from NCBI RefSeq v83, genus rank classification, default parameters)"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["classification","taxon"],"slot_uri":"MIXS:0000064","alias":"tax_class","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"experimental_factor":{"name":"experimental_factor","annotations":{"Expected_value":{"tag":"Expected_value","value":"text or EFO and/or OBI"}},"description":"Variable aspects of an experiment design that can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI)","title":"experimental factor","examples":[{"value":"time series design [EFO:0001779]"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["experimental","factor"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000008","alias":"experimental_factor","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true,"pattern":"^\\\\S+.*\\\\S+ \\\\[[a-zA-Z]{2,}:\\\\d+\\\\]$"},"associated_resource":{"name":"associated_resource","annotations":{"Expected_value":{"tag":"Expected_value","value":"reference to resource"}},"description":"A related resource that is referenced, cited, or otherwise associated to the sequence","title":"relevant electronic resources","examples":[{"value":"http://www.earthmicrobiome.org/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["resource"],"string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000091","alias":"associated_resource","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true},"sop":{"name":"sop","annotations":{"Expected_value":{"tag":"Expected_value","value":"reference to SOP"}},"description":"Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences","title":"relevant standard operating procedures","examples":[{"value":"http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/its/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["procedures"],"string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000090","alias":"sop","owner":"Mims","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true}},"class_uri":"MIXS:0010007"},"HostAssociated":{"name":"HostAssociated","description":"host-associated extension","title":"host-associated","from_schema":"https://w3id.org/mixs","is_a":"Extension","slot_usage":{"alt":{"name":"alt","recommended":true},"chem_administration":{"name":"chem_administration"},"depth":{"name":"depth","examples":[{"value":"10 meter"}],"recommended":true},"elev":{"name":"elev","recommended":true},"gravidity":{"name":"gravidity"},"host_age":{"name":"host_age","examples":[{"value":"10 days"}]},"host_body_habitat":{"name":"host_body_habitat","examples":[{"value":"nasopharynx"}]},"host_body_site":{"name":"host_body_site","examples":[{"value":"gill [UBERON:0002535]"}]},"host_body_temp":{"name":"host_body_temp","examples":[{"value":"15 degree Celsius"}]},"host_color":{"name":"host_color"},"host_common_name":{"name":"host_common_name","examples":[{"value":"human"}]},"host_diet":{"name":"host_diet","examples":[{"value":"herbivore"}]},"host_disease_stat":{"name":"host_disease_stat","examples":[{"value":"rabies [DOID:11260]"}]},"host_fam_rel":{"name":"host_fam_rel","examples":[{"value":"offspring;Mussel25"}]},"host_genotype":{"name":"host_genotype","examples":[{"value":"C57BL/6"}]},"host_growth_cond":{"name":"host_growth_cond"},"host_height":{"name":"host_height","examples":[{"value":"0.1 meter"}]},"host_last_meal":{"name":"host_last_meal","examples":[{"value":"corn feed;P2H"}]},"host_length":{"name":"host_length"},"host_life_stage":{"name":"host_life_stage","examples":[{"value":"adult"}]},"host_phenotype":{"name":"host_phenotype","examples":[{"value":"elongated [PATO:0001154]"}]},"host_shape":{"name":"host_shape"},"host_subject_id":{"name":"host_subject_id","examples":[{"value":"MPI123"}]},"host_substrate":{"name":"host_substrate"},"host_symbiont":{"name":"host_symbiont","examples":[{"value":"flukeworms"}]},"host_taxid":{"name":"host_taxid","examples":[{"value":"7955"}],"string_serialization":"{NCBI taxid}"},"host_tot_mass":{"name":"host_tot_mass","examples":[{"value":"2500 gram"}]},"organism_count":{"name":"organism_count","examples":[{"value":"total prokaryotes;3.5e7 cells per milliliter;qPCR"}]},"oxy_stat_samp":{"name":"oxy_stat_samp"},"perturbation":{"name":"perturbation"},"samp_store_dur":{"name":"samp_store_dur"},"samp_store_loc":{"name":"samp_store_loc"},"samp_vol_we_dna_ext":{"name":"samp_vol_we_dna_ext"},"temp":{"name":"temp"}},"attributes":{"samp_name":{"name":"samp_name","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":""}},"description":"A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name","title":"sample name","examples":[{"value":"ISDsoil1"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["sample"],"slot_uri":"MIXS:0001107","alias":"samp_name","owner":"HostAssociated","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"project_name":{"name":"project_name","description":"Name of the project within which the sequencing was organized","title":"project name","examples":[{"value":"Forest soil metagenome"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["project"],"slot_uri":"MIXS:0000092","alias":"project_name","owner":"HostAssociated","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"alt":{"name":"alt","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth\'s surface. In this context, the altitude measurement is the vertical distance between the earth\'s surface above sea level and the sampled position in the air","title":"altitude","examples":[{"value":"100 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000094","alias":"alt","owner":"HostAssociated","domain_of":["Mims","HostAssociated","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"depth":{"name":"depth","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"The vertical distance below local surface. For sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples","title":"depth","examples":[{"value":"10 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["depth"],"slot_uri":"MIXS:0000018","alias":"depth","owner":"HostAssociated","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"elev":{"name":"elev","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth\'s surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit","title":"elevation","examples":[{"value":"100 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["elevation"],"slot_uri":"MIXS:0000093","alias":"elev","owner":"HostAssociated","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"ances_data":{"name":"ances_data","description":"Information about either pedigree or other ancestral information description (e.g. parental variety in case of mutant or selection), e.g. A/3*B (meaning [(A x B) x B] x B)","title":"ancestral data","examples":[{"value":"A/3*B"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000247","alias":"ances_data","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"biol_stat":{"name":"biol_stat","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"The level of genome modification","title":"biological status","examples":[{"value":"natural"}],"from_schema":"https://w3id.org/mixs","keywords":["status"],"string_serialization":"[wild|natural|semi-natural|inbred line|breeder\'s line|hybrid|clonal selection|mutant]","slot_uri":"MIXS:0000858","alias":"biol_stat","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"genetic_mod":{"name":"genetic_mod","annotations":{"Expected_value":{"tag":"Expected_value","value":"PMID, DOI, URL or text"}},"description":"Genetic modifications of the genome of an organism, which may occur naturally by spontaneous mutation, or be introduced by some experimental means, e.g. specification of a transgene or the gene knocked-out or details of transient transfection","title":"genetic modification","examples":[{"value":"aox1A transgenic"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000859","alias":"genetic_mod","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_common_name":{"name":"host_common_name","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":""}},"description":"Common name of the host","title":"host common name","examples":[{"value":"human"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000248","alias":"host_common_name","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"samp_capt_status":{"name":"samp_capt_status","description":"Reason for the sample","title":"sample capture status","examples":[{"value":"farm sample"}],"from_schema":"https://w3id.org/mixs","keywords":["sample","status"],"slot_uri":"MIXS:0000860","alias":"samp_capt_status","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"SAMP_CAPT_STATUS_ENUM"},"samp_dis_stage":{"name":"samp_dis_stage","description":"Stage of the disease at the time of sample collection, e.g. inoculation, penetration, infection, growth and reproduction, dissemination of pathogen","title":"sample disease stage","examples":[{"value":"infection"}],"from_schema":"https://w3id.org/mixs","keywords":["disease","sample"],"slot_uri":"MIXS:0000249","alias":"samp_dis_stage","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"SAMP_DIS_STAGE_ENUM"},"host_taxid":{"name":"host_taxid","annotations":{"Expected_value":{"tag":"Expected_value","value":"NCBI taxon identifier"}},"description":"NCBI taxon id of the host, e.g. 9606","title":"host taxid","examples":[{"value":"7955"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","taxon"],"string_serialization":"{NCBI taxid}","slot_uri":"MIXS:0000250","alias":"host_taxid","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_subject_id":{"name":"host_subject_id","description":"A unique identifier by which each subject can be referred to, de-identified","title":"host subject id","examples":[{"value":"MPI123"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","identifier"],"slot_uri":"MIXS:0000861","alias":"host_subject_id","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_age":{"name":"host_age","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"year, day, hour"}},"description":"Age of host at the time of sampling; relevant scale depends on species and study, e.g. Could be seconds for amoebae or centuries for trees","title":"host age","examples":[{"value":"10 days"}],"from_schema":"https://w3id.org/mixs","keywords":["age","host","host."],"slot_uri":"MIXS:0000255","alias":"host_age","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_life_stage":{"name":"host_life_stage","annotations":{"Expected_value":{"tag":"Expected_value","value":"stage"}},"description":"Description of life stage of host","title":"host life stage","examples":[{"value":"adult"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","life"],"string_serialization":"{text}","slot_uri":"MIXS:0000251","alias":"host_life_stage","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_sex":{"name":"host_sex","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Gender or physical sex of the host","title":"host sex","comments":["example of non-binary from Excel sheets does not match any of the enumerated values"],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"string_serialization":"[female|hermaphrodite|non-binary|male|transgender|transgender (female to male)|transgender (male to female)\\n|undeclared]","slot_uri":"MIXS:0000811","alias":"host_sex","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_disease_stat":{"name":"host_disease_stat","annotations":{"Expected_value":{"tag":"Expected_value","value":"disease name or Disease Ontology term"}},"description":"List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text","title":"host disease status","examples":[{"value":"rabies [DOID:11260]"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["disease","host","host.","status"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000031","alias":"host_disease_stat","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"chem_administration":{"name":"chem_administration","annotations":{"Expected_value":{"tag":"Expected_value","value":"CHEBI;timestamp"}},"description":"List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi","title":"chemical administration","examples":[{"value":"agar [CHEBI:2509];2018-05-11T20:00Z"}],"from_schema":"https://w3id.org/mixs","keywords":["administration"],"string_serialization":"{termLabel} [{termID}];{timestamp}","slot_uri":"MIXS:0000751","alias":"chem_administration","owner":"HostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"host_body_habitat":{"name":"host_body_habitat","description":"Original body habitat where the sample was obtained from","title":"host body habitat","examples":[{"value":"nasopharynx"}],"from_schema":"https://w3id.org/mixs","keywords":["body","habitat","host","host."],"slot_uri":"MIXS:0000866","alias":"host_body_habitat","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_body_site":{"name":"host_body_site","annotations":{"Expected_value":{"tag":"Expected_value","value":"FMA or UBERON"}},"description":"Name of body site where the sample was obtained from, such as a specific organ or tissue (tongue, lung etc...). Use terms from the foundational model of anatomy ontology (fma) or the Uber-anatomy ontology (UBERON)","title":"host body site","examples":[{"value":"gill [UBERON:0002535]"}],"from_schema":"https://w3id.org/mixs","keywords":["body","host","host.","site"],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000867","alias":"host_body_site","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_body_product":{"name":"host_body_product","annotations":{"Expected_value":{"tag":"Expected_value","value":"FMA or UBERON"}},"description":"Substance produced by the body, e.g. Stool, mucus, where the sample was obtained from. Use terms from the foundational model of anatomy ontology (fma) or Uber-anatomy ontology (UBERON)","title":"host body product","examples":[{"value":"mucus [FMA:66938]"}],"from_schema":"https://w3id.org/mixs","keywords":["body","host","host.","product"],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000888","alias":"host_body_product","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_tot_mass":{"name":"host_tot_mass","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"kilogram, gram"}},"description":"Total mass of the host at collection, the unit depends on host","title":"host total mass","examples":[{"value":"2500 gram"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","mass","total"],"slot_uri":"MIXS:0000263","alias":"host_tot_mass","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_height":{"name":"host_height","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"centimeter, millimeter, meter"}},"description":"The height of subject","title":"host height","examples":[{"value":"0.1 meter"}],"from_schema":"https://w3id.org/mixs","keywords":["height","host","host."],"slot_uri":"MIXS:0000264","alias":"host_height","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_length":{"name":"host_length","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"centimeter, millimeter, meter"}},"description":"The length of subject","title":"host length","examples":[{"value":"1 meter"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","length"],"slot_uri":"MIXS:0000256","alias":"host_length","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_diet":{"name":"host_diet","description":"Type of diet depending on the host, for animals omnivore, herbivore etc., for humans high-fat, meditteranean etc.; can include multiple diet types","title":"host diet","examples":[{"value":"herbivore"}],"from_schema":"https://w3id.org/mixs","keywords":["diet","host","host."],"slot_uri":"MIXS:0000869","alias":"host_diet","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"host_last_meal":{"name":"host_last_meal","annotations":{"Expected_value":{"tag":"Expected_value","value":"content;duration"}},"description":"Content of last meal and time since feeding; can include multiple values","title":"host last meal","examples":[{"value":"corn feed;P2H"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"string_serialization":"{text};{duration}","slot_uri":"MIXS:0000870","alias":"host_last_meal","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"host_growth_cond":{"name":"host_growth_cond","description":"Literature reference giving growth conditions of the host","title":"host growth conditions","examples":[{"value":"https://academic.oup.com/icesjms/article/68/2/349/617247"}],"from_schema":"https://w3id.org/mixs","keywords":["condition","growth","host","host."],"slot_uri":"MIXS:0000871","alias":"host_growth_cond","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$|([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}|{text}$","interpolated":true,"partial_match":true}},"host_substrate":{"name":"host_substrate","description":"The growth substrate of the host","title":"host substrate","examples":[{"value":"rock"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000252","alias":"host_substrate","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_fam_rel":{"name":"host_fam_rel","annotations":{"Expected_value":{"tag":"Expected_value","value":"relationship type;arbitrary identifier"}},"description":"Relationships to other hosts in the same study; can include multiple relationships","title":"host family relationship","examples":[{"value":"offspring;Mussel25"}],"from_schema":"https://w3id.org/mixs","keywords":["family","host","host.","relationship"],"string_serialization":"{text};{text}","slot_uri":"MIXS:0000872","alias":"host_fam_rel","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"host_subspecf_genlin":{"name":"host_subspecf_genlin","annotations":{"Expected_value":{"tag":"Expected_value","value":"Genetic lineage below lowest rank of NCBI taxonomy, which is subspecies, e.g. serovar, biotype, ecotype, variety, cultivar"}},"description":"Information about the genetic distinctness of the host organism below the subspecies level e.g., serovar, serotype, biotype, ecotype, variety, cultivar, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123","title":"host subspecific genetic lineage","examples":[{"value":"serovar:Newport, variety:glabrum, cultivar: Red Delicious"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","lineage"],"string_serialization":"{rank name}:{text}","slot_uri":"MIXS:0001318","alias":"host_subspecf_genlin","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"host_genotype":{"name":"host_genotype","description":"Observed genotype","title":"host genotype","examples":[{"value":"C57BL/6"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000365","alias":"host_genotype","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_phenotype":{"name":"host_phenotype","annotations":{"Expected_value":{"tag":"Expected_value","value":"PATO or HP"}},"description":"Phenotype of human or other host. Use terms from the phenotypic quality ontology (pato) or the Human Phenotype Ontology (HP)","title":"host phenotype","examples":[{"value":"elongated [PATO:0001154]"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000874","alias":"host_phenotype","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_body_temp":{"name":"host_body_temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Core body temperature of the host when sample was collected","title":"host body temperature","examples":[{"value":"15 degree Celsius"}],"from_schema":"https://w3id.org/mixs","keywords":["body","host","host.","temperature"],"slot_uri":"MIXS:0000274","alias":"host_body_temp","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_dry_mass":{"name":"host_dry_mass","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"kilogram, gram"}},"description":"Measurement of dry mass","title":"host dry mass","examples":[{"value":"500 gram"}],"from_schema":"https://w3id.org/mixs","keywords":["dry","host","host.","mass"],"slot_uri":"MIXS:0000257","alias":"host_dry_mass","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"blood_press_diast":{"name":"blood_press_diast","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"millimeter mercury"}},"description":"Resting diastolic blood pressure, measured as mm mercury","title":"host blood pressure diastolic","from_schema":"https://w3id.org/mixs","keywords":["host","host.","pressure"],"slot_uri":"MIXS:0000258","alias":"blood_press_diast","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"blood_press_syst":{"name":"blood_press_syst","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"millimeter mercury"}},"description":"Resting systolic blood pressure, measured as mm mercury","title":"host blood pressure systolic","from_schema":"https://w3id.org/mixs","keywords":["host","host.","pressure"],"slot_uri":"MIXS:0000259","alias":"blood_press_syst","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_color":{"name":"host_color","description":"The color of host","title":"host color","from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000260","alias":"host_color","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_shape":{"name":"host_shape","description":"Morphological shape of host","title":"host shape","examples":[{"value":"round"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000261","alias":"host_shape","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"gravidity":{"name":"gravidity","description":"Whether or not subject is gravid, and if yes date due or date post-conception, specifying which is used","title":"gravidity","examples":[{"value":"yes;due date:2018-05-11"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{boolean};{timestamp}","slot_uri":"MIXS:0000875","alias":"gravidity","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"perturbation":{"name":"perturbation","annotations":{"Expected_value":{"tag":"Expected_value","value":"perturbation type name;perturbation interval and duration"}},"description":"Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types","title":"perturbation","examples":[{"value":"antibiotic addition;R2/2018-05-11T14:30Z/2018-05-11T19:30Z/P1H30M"}],"from_schema":"https://w3id.org/mixs","keywords":["perturbation"],"string_serialization":"{text};{Rn/start_time/end_time/duration}","slot_uri":"MIXS:0000754","alias":"perturbation","owner":"HostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"salinity":{"name":"salinity","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"practical salinity unit, percentage"}},"description":"The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater","title":"salinity","examples":[{"value":"25 practical salinity unit"}],"from_schema":"https://w3id.org/mixs","keywords":["salinity"],"slot_uri":"MIXS:0000183","alias":"salinity","owner":"HostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"oxy_stat_samp":{"name":"oxy_stat_samp","description":"Oxygenation status of sample","title":"oxygenation status of sample","examples":[{"value":"aerobic"}],"from_schema":"https://w3id.org/mixs","keywords":["oxygen","sample","status"],"slot_uri":"MIXS:0000753","alias":"oxy_stat_samp","owner":"HostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"OXY_STAT_SAMP_ENUM"},"temp":{"name":"temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Temperature of the sample at the time of sampling","title":"temperature","examples":[{"value":"25 degree Celsius"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["temperature"],"slot_uri":"MIXS:0000113","alias":"temp","owner":"HostAssociated","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"organism_count":{"name":"organism_count","annotations":{"Expected_value":{"tag":"Expected_value","value":"organism name;measurement value;enumeration"}},"description":"Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)","title":"organism count","examples":[{"value":"total prokaryotes;3.5e7 cells per milliliter;qPCR"}],"from_schema":"https://w3id.org/mixs","keywords":["count","organism"],"string_serialization":"{text};{float} {unit};[ATP|MPN|qPCR|other]","slot_uri":"MIXS:0000103","alias":"organism_count","owner":"HostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"samp_vol_we_dna_ext":{"name":"samp_vol_we_dna_ext","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milliliter, gram, milligram, square centimeter"}},"description":"Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001)","title":"sample volume or weight for DNA extraction","examples":[{"value":"1500 milliliter"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["dna","sample","volume","weight"],"slot_uri":"MIXS:0000111","alias":"samp_vol_we_dna_ext","owner":"HostAssociated","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_store_temp":{"name":"samp_store_temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Temperature at which sample was stored, e.g. -80 degree Celsius","title":"sample storage temperature","examples":[{"value":"-80 degree Celsius"}],"from_schema":"https://w3id.org/mixs","keywords":["sample","storage","temperature"],"slot_uri":"MIXS:0000110","alias":"samp_store_temp","owner":"HostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_store_dur":{"name":"samp_store_dur","description":"Duration for which the sample was stored. Indicate the duration for which the sample was stored written in ISO 8601 format","title":"sample storage duration","examples":[{"value":"P1Y6M"}],"from_schema":"https://w3id.org/mixs","keywords":["duration","period","sample","storage"],"slot_uri":"MIXS:0000116","alias":"samp_store_dur","owner":"HostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^P(?:(?:\\\\d+D|\\\\d+M(?:\\\\d+D)?|\\\\d+Y(?:\\\\d+M(?:\\\\d+D)?)?)(?:T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S))?|T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S)|\\\\d+W)$","structured_pattern":{"syntax":"^{duration}$","interpolated":true,"partial_match":true}},"samp_store_loc":{"name":"samp_store_loc","annotations":{"Expected_value":{"tag":"Expected_value","value":"location name"}},"description":"Location at which sample was stored, usually name of a specific freezer/room","title":"sample storage location","examples":[{"value":"Freezer no:5"}],"from_schema":"https://w3id.org/mixs","keywords":["location","sample","storage"],"string_serialization":"{text}","slot_uri":"MIXS:0000755","alias":"samp_store_loc","owner":"HostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"host_symbiont":{"name":"host_symbiont","annotations":{"Expected_value":{"tag":"Expected_value","value":"species name or common name"}},"description":"The taxonomic name of the organism(s) found living in mutualistic, commensalistic, or parasitic symbiosis with the specific host. The sampled symbiont can have its own symbionts. For example, parasites may have hyperparasites (=parasites of the parasite)","title":"observed host symbionts","examples":[{"value":"flukeworms"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","observed","symbiosis"],"string_serialization":"{text}","slot_uri":"MIXS:0001298","alias":"host_symbiont","owner":"HostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"misc_param":{"name":"misc_param","annotations":{"Expected_value":{"tag":"Expected_value","value":"parameter name;measurement value"}},"description":"Any other measurement performed or parameter collected, that is not listed here","title":"miscellaneous parameter","examples":[{"value":"Bicarbonate ion concentration;2075 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","keywords":["parameter"],"string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000752","alias":"misc_param","owner":"HostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true}},"class_uri":"MIXS:0016002"},"Sediment":{"name":"Sediment","description":"sediment extension","title":"sediment","from_schema":"https://w3id.org/mixs","is_a":"Extension","slot_usage":{"alkyl_diethers":{"name":"alkyl_diethers"},"ammonium":{"name":"ammonium"},"bacteria_carb_prod":{"name":"bacteria_carb_prod"},"chem_administration":{"name":"chem_administration"},"depth":{"name":"depth","examples":[{"value":"10 meter"}],"required":true},"diss_org_nitro":{"name":"diss_org_nitro"},"elev":{"name":"elev","recommended":true},"methane":{"name":"methane","examples":[{"value":"0.15 micromole per liter"}]},"n_alkanes":{"name":"n_alkanes"},"nitrate":{"name":"nitrate"},"nitrite":{"name":"nitrite"},"org_carb":{"name":"org_carb"},"org_nitro":{"name":"org_nitro"},"organism_count":{"name":"organism_count","examples":[{"value":"total prokaryotes;3.5e7 cells per milliliter;qPCR"}]},"oxy_stat_samp":{"name":"oxy_stat_samp"},"part_org_carb":{"name":"part_org_carb"},"perturbation":{"name":"perturbation"},"ph":{"name":"ph"},"phosplipid_fatt_acid":{"name":"phosplipid_fatt_acid"},"samp_store_dur":{"name":"samp_store_dur"},"samp_store_loc":{"name":"samp_store_loc"},"samp_vol_we_dna_ext":{"name":"samp_vol_we_dna_ext"},"sulfate":{"name":"sulfate"},"sulfide":{"name":"sulfide"},"temp":{"name":"temp"},"tot_carb":{"name":"tot_carb"},"tot_nitro_content":{"name":"tot_nitro_content"},"tot_org_carb":{"name":"tot_org_carb"},"turbidity":{"name":"turbidity"},"water_content":{"name":"water_content","string_serialization":"{float} {unit}"}},"attributes":{"samp_name":{"name":"samp_name","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":""}},"description":"A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name","title":"sample name","examples":[{"value":"ISDsoil1"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["sample"],"slot_uri":"MIXS:0001107","alias":"samp_name","owner":"Sediment","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"project_name":{"name":"project_name","description":"Name of the project within which the sequencing was organized","title":"project name","examples":[{"value":"Forest soil metagenome"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["project"],"slot_uri":"MIXS:0000092","alias":"project_name","owner":"Sediment","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"depth":{"name":"depth","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"The vertical distance below local surface. For sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples","title":"depth","examples":[{"value":"10 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["depth"],"slot_uri":"MIXS:0000018","alias":"depth","owner":"Sediment","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"elev":{"name":"elev","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth\'s surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit","title":"elevation","examples":[{"value":"100 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["elevation"],"slot_uri":"MIXS:0000093","alias":"elev","owner":"Sediment","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"alkalinity":{"name":"alkalinity","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milliequivalent per liter, milligram per liter"}},"description":"Alkalinity, the ability of a solution to neutralize acids to the equivalence point of carbonate or bicarbonate","title":"alkalinity","examples":[{"value":"50 milligram per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["alkalinity"],"slot_uri":"MIXS:0000421","alias":"alkalinity","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"alkyl_diethers":{"name":"alkyl_diethers","description":"Concentration of alkyl diethers","title":"alkyl diethers","examples":[{"value":"0.005 mole per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000490","alias":"alkyl_diethers","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"aminopept_act":{"name":"aminopept_act","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"mole per liter per hour"}},"description":"Measurement of aminopeptidase activity","title":"aminopeptidase activity","examples":[{"value":"0.269 mole per liter per hour"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000172","alias":"aminopept_act","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"ammonium":{"name":"ammonium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of ammonium in the sample","title":"ammonium","examples":[{"value":"1.5 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000427","alias":"ammonium","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"bacteria_carb_prod":{"name":"bacteria_carb_prod","description":"Measurement of bacterial carbon production","title":"bacterial carbon production","examples":[{"value":"2.53 microgram per liter per hour"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","production"],"slot_uri":"MIXS:0000173","alias":"bacteria_carb_prod","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"biomass":{"name":"biomass","annotations":{"Expected_value":{"tag":"Expected_value","value":"biomass type;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"ton, kilogram, gram"}},"description":"Amount of biomass; should include the name for the part of biomass measured, e.g. Microbial, total. Can include multiple measurements","title":"biomass","examples":[{"value":"total;20 gram"}],"from_schema":"https://w3id.org/mixs","keywords":["biomass"],"string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000174","alias":"biomass","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"bishomohopanol":{"name":"bishomohopanol","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"microgram per liter, microgram per gram"}},"description":"Concentration of bishomohopanol","title":"bishomohopanol","examples":[{"value":"14 microgram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000175","alias":"bishomohopanol","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"bromide":{"name":"bromide","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"parts per million"}},"description":"Concentration of bromide","title":"bromide","examples":[{"value":"0.05 parts per million"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000176","alias":"bromide","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"calcium":{"name":"calcium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, micromole per liter, parts per million"}},"description":"Concentration of calcium in the sample","title":"calcium","examples":[{"value":"0.2 micromole per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000432","alias":"calcium","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"carb_nitro_ratio":{"name":"carb_nitro_ratio","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement value"}},"description":"Ratio of amount or concentrations of carbon to nitrogen. Used for quality control value in proteinaceous samples for radiocarbon dating.","title":"carbon/nitrogen ratio","examples":[{"value":"0.417361111"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","nitrogen","ratio"],"string_serialization":"{float}:{float}","slot_uri":"MIXS:0000310","alias":"carb_nitro_ratio","owner":"Sediment","domain_of":["Sediment","RadiocarbonDating","MimsSediment"],"range":"float","required":false,"recommended":true,"multivalued":false},"chem_administration":{"name":"chem_administration","annotations":{"Expected_value":{"tag":"Expected_value","value":"CHEBI;timestamp"}},"description":"List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi","title":"chemical administration","examples":[{"value":"agar [CHEBI:2509];2018-05-11T20:00Z"}],"from_schema":"https://w3id.org/mixs","keywords":["administration"],"string_serialization":"{termLabel} [{termID}];{timestamp}","slot_uri":"MIXS:0000751","alias":"chem_administration","owner":"Sediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"chloride":{"name":"chloride","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, parts per million"}},"description":"Concentration of chloride in the sample","title":"chloride","examples":[{"value":"5000 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000429","alias":"chloride","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"chlorophyll":{"name":"chlorophyll","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per cubic meter, microgram per liter"}},"description":"Concentration of chlorophyll","title":"chlorophyll","examples":[{"value":"5 milligram per cubic meter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000177","alias":"chlorophyll","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"density":{"name":"density","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"gram per cubic meter, gram per cubic centimeter"}},"description":"Density of the sample, which is its mass per unit volume (aka volumetric mass density)","title":"density","examples":[{"value":"1000 kilogram per cubic meter"}],"from_schema":"https://w3id.org/mixs","keywords":["density"],"slot_uri":"MIXS:0000435","alias":"density","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diether_lipids":{"name":"diether_lipids","annotations":{"Expected_value":{"tag":"Expected_value","value":"diether lipid name;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"nanogram per liter"}},"description":"Concentration of diether lipids; can include multiple types of diether lipids","title":"diether lipids","examples":[{"value":"0.2 nanogram per liter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000178","alias":"diether_lipids","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"diss_carb_dioxide":{"name":"diss_carb_dioxide","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter"}},"description":"Concentration of dissolved carbon dioxide in the sample or liquid portion of the sample","title":"dissolved carbon dioxide","examples":[{"value":"5 milligram per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","dissolved"],"slot_uri":"MIXS:0000436","alias":"diss_carb_dioxide","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_hydrogen":{"name":"diss_hydrogen","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of dissolved hydrogen","title":"dissolved hydrogen","examples":[{"value":"0.3 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["dissolved"],"slot_uri":"MIXS:0000179","alias":"diss_hydrogen","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_inorg_carb":{"name":"diss_inorg_carb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"microgram per liter, milligram per liter, parts per million"}},"description":"Dissolved inorganic carbon concentration in the sample, typically measured after filtering the sample using a 0.45 micrometer filter","title":"dissolved inorganic carbon","examples":[{"value":"2059 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","dissolved","inorganic"],"slot_uri":"MIXS:0000434","alias":"diss_inorg_carb","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_org_carb":{"name":"diss_org_carb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter"}},"description":"Concentration of dissolved organic carbon in the sample, liquid portion of the sample, or aqueous phase of the fluid","title":"dissolved organic carbon","examples":[{"value":"197 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","dissolved","organic"],"slot_uri":"MIXS:0000433","alias":"diss_org_carb","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_org_nitro":{"name":"diss_org_nitro","description":"Dissolved organic nitrogen concentration measured as; total dissolved nitrogen - NH4 - NO3 - NO2","title":"dissolved organic nitrogen","examples":[{"value":"0.05 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["dissolved","nitrogen","organic"],"slot_uri":"MIXS:0000162","alias":"diss_org_nitro","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_oxygen":{"name":"diss_oxygen","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per kilogram, milligram per liter"}},"description":"Concentration of dissolved oxygen","title":"dissolved oxygen","examples":[{"value":"175 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","keywords":["dissolved","oxygen"],"slot_uri":"MIXS:0000119","alias":"diss_oxygen","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"glucosidase_act":{"name":"glucosidase_act","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"mol per liter per hour"}},"description":"Measurement of glucosidase activity","title":"glucosidase activity","examples":[{"value":"5 mol per liter per hour"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000137","alias":"glucosidase_act","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"magnesium":{"name":"magnesium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"mole per liter, milligram per liter, parts per million, micromole per kilogram"}},"description":"Concentration of magnesium in the sample","title":"magnesium","examples":[{"value":"52.8 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000431","alias":"magnesium","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"mean_frict_vel":{"name":"mean_frict_vel","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter per second"}},"description":"Measurement of mean friction velocity","title":"mean friction velocity","examples":[{"value":"0.5 meter per second"}],"from_schema":"https://w3id.org/mixs","keywords":["mean","velocity"],"slot_uri":"MIXS:0000498","alias":"mean_frict_vel","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"mean_peak_frict_vel":{"name":"mean_peak_frict_vel","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter per second"}},"description":"Measurement of mean peak friction velocity","title":"mean peak friction velocity","examples":[{"value":"1 meter per second"}],"from_schema":"https://w3id.org/mixs","keywords":["mean","peak","velocity"],"slot_uri":"MIXS:0000502","alias":"mean_peak_frict_vel","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"methane":{"name":"methane","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, parts per billion, parts per million"}},"description":"Methane (gas) amount or concentration at the time of sampling","title":"methane","examples":[{"value":"0.15 micromole per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000101","alias":"methane","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"misc_param":{"name":"misc_param","annotations":{"Expected_value":{"tag":"Expected_value","value":"parameter name;measurement value"}},"description":"Any other measurement performed or parameter collected, that is not listed here","title":"miscellaneous parameter","examples":[{"value":"Bicarbonate ion concentration;2075 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","keywords":["parameter"],"string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000752","alias":"misc_param","owner":"Sediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"n_alkanes":{"name":"n_alkanes","annotations":{"Expected_value":{"tag":"Expected_value","value":"n-alkane name;measurement value"}},"description":"Concentration of n-alkanes; can include multiple n-alkanes","title":"n-alkanes","examples":[{"value":"n-hexadecane;100 milligram per liter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000503","alias":"n_alkanes","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"nitrate":{"name":"nitrate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of nitrate in the sample","title":"nitrate","examples":[{"value":"65 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrate"],"slot_uri":"MIXS:0000425","alias":"nitrate","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"nitrite":{"name":"nitrite","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of nitrite in the sample","title":"nitrite","examples":[{"value":"0.5 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrite"],"slot_uri":"MIXS:0000426","alias":"nitrite","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"nitro":{"name":"nitro","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of nitrogen (total)","title":"nitrogen","examples":[{"value":"4.2 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrogen"],"slot_uri":"MIXS:0000504","alias":"nitro","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"org_carb":{"name":"org_carb","description":"Concentration of organic carbon","title":"organic carbon","examples":[{"value":"1.5 microgram per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","organic"],"slot_uri":"MIXS:0000508","alias":"org_carb","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"org_matter":{"name":"org_matter","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"microgram per liter"}},"description":"Concentration of organic matter","title":"organic matter","examples":[{"value":"1.75 milligram per cubic meter"}],"from_schema":"https://w3id.org/mixs","keywords":["organic"],"slot_uri":"MIXS:0000204","alias":"org_matter","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"org_nitro":{"name":"org_nitro","description":"Concentration of organic nitrogen","title":"organic nitrogen","examples":[{"value":"4 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrogen","organic"],"slot_uri":"MIXS:0000205","alias":"org_nitro","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"organism_count":{"name":"organism_count","annotations":{"Expected_value":{"tag":"Expected_value","value":"organism name;measurement value;enumeration"}},"description":"Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)","title":"organism count","examples":[{"value":"total prokaryotes;3.5e7 cells per milliliter;qPCR"}],"from_schema":"https://w3id.org/mixs","keywords":["count","organism"],"string_serialization":"{text};{float} {unit};[ATP|MPN|qPCR|other]","slot_uri":"MIXS:0000103","alias":"organism_count","owner":"Sediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"oxy_stat_samp":{"name":"oxy_stat_samp","description":"Oxygenation status of sample","title":"oxygenation status of sample","examples":[{"value":"aerobic"}],"from_schema":"https://w3id.org/mixs","keywords":["oxygen","sample","status"],"slot_uri":"MIXS:0000753","alias":"oxy_stat_samp","owner":"Sediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"OXY_STAT_SAMP_ENUM"},"ph":{"name":"ph","description":"Ph measurement of the sample, or liquid portion of sample, or aqueous phase of the fluid","title":"pH","examples":[{"value":"7.2"}],"from_schema":"https://w3id.org/mixs","keywords":["ph"],"slot_uri":"MIXS:0001001","alias":"ph","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"float"},"particle_class":{"name":"particle_class","annotations":{"Expected_value":{"tag":"Expected_value","value":"particle name;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"micrometer"}},"description":"Particles are classified, based on their size, into six general categories:clay, silt, sand, gravel, cobbles, and boulders; should include amount of particle preceded by the name of the particle type; can include multiple values","title":"particle classification","from_schema":"https://w3id.org/mixs","keywords":["classification","particle"],"string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000206","alias":"particle_class","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"part_org_carb":{"name":"part_org_carb","description":"Concentration of particulate organic carbon","title":"particulate organic carbon","examples":[{"value":"1.92 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","organic","particle","particulate"],"slot_uri":"MIXS:0000515","alias":"part_org_carb","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"perturbation":{"name":"perturbation","annotations":{"Expected_value":{"tag":"Expected_value","value":"perturbation type name;perturbation interval and duration"}},"description":"Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types","title":"perturbation","examples":[{"value":"antibiotic addition;R2/2018-05-11T14:30Z/2018-05-11T19:30Z/P1H30M"}],"from_schema":"https://w3id.org/mixs","keywords":["perturbation"],"string_serialization":"{text};{Rn/start_time/end_time/duration}","slot_uri":"MIXS:0000754","alias":"perturbation","owner":"Sediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"petroleum_hydrocarb":{"name":"petroleum_hydrocarb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of petroleum hydrocarbon","title":"petroleum hydrocarbon","examples":[{"value":"0.05 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["hydrocarbon","petroleum"],"slot_uri":"MIXS:0000516","alias":"petroleum_hydrocarb","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"phaeopigments":{"name":"phaeopigments","annotations":{"Expected_value":{"tag":"Expected_value","value":"phaeopigment name;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per cubic meter"}},"description":"Concentration of phaeopigments; can include multiple phaeopigments","title":"phaeopigments","examples":[{"value":"2.5 milligram per cubic meter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000180","alias":"phaeopigments","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"phosphate":{"name":"phosphate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of phosphate","title":"phosphate","examples":[{"value":"0.7 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["phosphate"],"slot_uri":"MIXS:0000505","alias":"phosphate","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"phosplipid_fatt_acid":{"name":"phosplipid_fatt_acid","annotations":{"Expected_value":{"tag":"Expected_value","value":"phospholipid fatty acid name;measurement value"}},"description":"Concentration of phospholipid fatty acids; can include multiple values","title":"phospholipid fatty acid","examples":[{"value":"2.98 milligram per liter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000181","alias":"phosplipid_fatt_acid","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"porosity":{"name":"porosity","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement value or range"},"Preferred_unit":{"tag":"Preferred_unit","value":"percentage"}},"description":"Porosity of deposited sediment is volume of voids divided by the total volume of sample","title":"porosity","from_schema":"https://w3id.org/mixs","keywords":["porosity"],"string_serialization":"{float} - {float} {unit}","slot_uri":"MIXS:0000211","alias":"porosity","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string"},"potassium":{"name":"potassium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, parts per million"}},"description":"Concentration of potassium in the sample","title":"potassium","examples":[{"value":"463 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000430","alias":"potassium","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"pressure":{"name":"pressure","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"atmosphere"}},"description":"Pressure to which the sample is subject to, in atmospheres","title":"pressure","examples":[{"value":"50 atmosphere"}],"from_schema":"https://w3id.org/mixs","keywords":["pressure"],"slot_uri":"MIXS:0000412","alias":"pressure","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"redox_potential":{"name":"redox_potential","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"millivolt"}},"description":"Redox potential, measured relative to a hydrogen cell, indicating oxidation or reduction potential","title":"redox potential","examples":[{"value":"300 millivolt"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000182","alias":"redox_potential","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"salinity":{"name":"salinity","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"practical salinity unit, percentage"}},"description":"The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater","title":"salinity","examples":[{"value":"25 practical salinity unit"}],"from_schema":"https://w3id.org/mixs","keywords":["salinity"],"slot_uri":"MIXS:0000183","alias":"salinity","owner":"Sediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_store_dur":{"name":"samp_store_dur","description":"Duration for which the sample was stored. Indicate the duration for which the sample was stored written in ISO 8601 format","title":"sample storage duration","examples":[{"value":"P1Y6M"}],"from_schema":"https://w3id.org/mixs","keywords":["duration","period","sample","storage"],"slot_uri":"MIXS:0000116","alias":"samp_store_dur","owner":"Sediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^P(?:(?:\\\\d+D|\\\\d+M(?:\\\\d+D)?|\\\\d+Y(?:\\\\d+M(?:\\\\d+D)?)?)(?:T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S))?|T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S)|\\\\d+W)$","structured_pattern":{"syntax":"^{duration}$","interpolated":true,"partial_match":true}},"samp_store_loc":{"name":"samp_store_loc","annotations":{"Expected_value":{"tag":"Expected_value","value":"location name"}},"description":"Location at which sample was stored, usually name of a specific freezer/room","title":"sample storage location","examples":[{"value":"Freezer no:5"}],"from_schema":"https://w3id.org/mixs","keywords":["location","sample","storage"],"string_serialization":"{text}","slot_uri":"MIXS:0000755","alias":"samp_store_loc","owner":"Sediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"samp_store_temp":{"name":"samp_store_temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Temperature at which sample was stored, e.g. -80 degree Celsius","title":"sample storage temperature","examples":[{"value":"-80 degree Celsius"}],"from_schema":"https://w3id.org/mixs","keywords":["sample","storage","temperature"],"slot_uri":"MIXS:0000110","alias":"samp_store_temp","owner":"Sediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_vol_we_dna_ext":{"name":"samp_vol_we_dna_ext","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milliliter, gram, milligram, square centimeter"}},"description":"Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001)","title":"sample volume or weight for DNA extraction","examples":[{"value":"1500 milliliter"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["dna","sample","volume","weight"],"slot_uri":"MIXS:0000111","alias":"samp_vol_we_dna_ext","owner":"Sediment","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sediment_type":{"name":"sediment_type","description":"Information about the sediment type based on major constituents","title":"sediment type","examples":[{"value":"biogenous"}],"from_schema":"https://w3id.org/mixs","keywords":["sediment","type"],"slot_uri":"MIXS:0001078","alias":"sediment_type","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"SEDIMENT_TYPE_ENUM"},"silicate":{"name":"silicate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of silicate","title":"silicate","examples":[{"value":"0.05 micromole per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000184","alias":"silicate","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sodium":{"name":"sodium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, parts per million"}},"description":"Sodium concentration in the sample","title":"sodium","examples":[{"value":"10.5 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000428","alias":"sodium","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sulfate":{"name":"sulfate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of sulfate in the sample","title":"sulfate","examples":[{"value":"5 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["sulfate"],"slot_uri":"MIXS:0000423","alias":"sulfate","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sulfide":{"name":"sulfide","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of sulfide in the sample","title":"sulfide","examples":[{"value":"2 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["sulfide"],"slot_uri":"MIXS:0000424","alias":"sulfide","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"temp":{"name":"temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Temperature of the sample at the time of sampling","title":"temperature","examples":[{"value":"25 degree Celsius"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["temperature"],"slot_uri":"MIXS:0000113","alias":"temp","owner":"Sediment","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tidal_stage":{"name":"tidal_stage","description":"Stage of tide","title":"tidal stage","examples":[{"value":"high tide"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000750","alias":"tidal_stage","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"TIDAL_STAGE_ENUM"},"tot_carb":{"name":"tot_carb","description":"Total carbon content","title":"total carbon","from_schema":"https://w3id.org/mixs","keywords":["carbon","total"],"slot_uri":"MIXS:0000525","alias":"tot_carb","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tot_depth_water_col":{"name":"tot_depth_water_col","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Measurement of total depth of water column","title":"total depth of water column","examples":[{"value":"500 meter"}],"from_schema":"https://w3id.org/mixs","keywords":["depth","total","water"],"slot_uri":"MIXS:0000634","alias":"tot_depth_water_col","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tot_nitro_content":{"name":"tot_nitro_content","description":"Total nitrogen content of the sample","title":"total nitrogen content","examples":[{"value":"35 milligrams Nitrogen per kilogram of soil"}],"from_schema":"https://w3id.org/mixs","keywords":["content","nitrogen","total"],"slot_uri":"MIXS:0000530","alias":"tot_nitro_content","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tot_org_carb":{"name":"tot_org_carb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"gram Carbon per kilogram sample material"}},"description":"Total organic carbon content","title":"total organic carbon","examples":[{"value":"0.02"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","organic","total"],"slot_uri":"MIXS:0000533","alias":"tot_org_carb","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"turbidity":{"name":"turbidity","description":"Measure of the amount of cloudiness or haziness in water caused by individual particles","title":"turbidity","examples":[{"value":"0.3 nephelometric turbidity units"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000191","alias":"turbidity","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"water_content":{"name":"water_content","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"gram per gram or cubic centimeter per cubic centimeter"}},"description":"Water content measurement","title":"water content","from_schema":"https://w3id.org/mixs","keywords":["content","water"],"string_serialization":"{float} {unit}","slot_uri":"MIXS:0000185","alias":"water_content","owner":"Sediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}}},"class_uri":"MIXS:0016011"},"Checklist":{"name":"Checklist","description":"A collection of metadata terms (slots) to minimally describe the sampling and sequencing method of a specimen used to generate a nucleotide sequence.","from_schema":"https://w3id.org/mixs"},"Extension":{"name":"Extension","description":"A collection of recommended metadata terms (slots) developed by community experts, describing the specific context under which a sample was collected.","from_schema":"https://w3id.org/mixs","aliases":["EnvironmentalPackage"]},"MixsCompliantData":{"name":"MixsCompliantData","description":"A collection of Data that comply with some combination of a MIxS checklist and environmental extension","title":"MIxS compliant data","from_schema":"https://w3id.org/mixs","attributes":{"mims_host_associated_ancient_data":{"name":"mims_host_associated_ancient_data","description":"Data that comply with Ancient combined with MimsHostAssociated","title":"MimsHostAssociatedAncient data","from_schema":"https://w3id.org/mixs","domain":"MixsCompliantData","slot_uri":"MIXS:mims_hostassociated_ancient_data","alias":"mims_host_associated_ancient_data","owner":"MixsCompliantData","domain_of":["MixsCompliantData"],"range":"MimsHostAssociatedAncient","multivalued":true,"inlined":true,"inlined_as_list":true},"mims_sediment_ancient_data":{"name":"mims_sediment_ancient_data","description":"Data that comply with Ancient combined with MimsSediment","title":"MimsSedimentAncient data","from_schema":"https://w3id.org/mixs","domain":"MixsCompliantData","slot_uri":"MIXS:mims_sediment_ancient_data","alias":"mims_sediment_ancient_data","owner":"MixsCompliantData","domain_of":["MixsCompliantData"],"range":"MimsSedimentAncient","multivalued":true,"inlined":true,"inlined_as_list":true}},"tree_root":true},"MimsHostAssociated":{"name":"MimsHostAssociated","description":"MIxS data that complies with the Mims checklist and the HostAssociated Extension","title":"Mims combined with HostAssociated","in_subset":["combination_classes"],"from_schema":"https://w3id.org/mixs","is_a":"HostAssociated","mixins":["Mims"],"attributes":{"samp_name":{"name":"samp_name","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":""}},"description":"A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name","title":"sample name","examples":[{"value":"ISDsoil1"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["sample"],"slot_uri":"MIXS:0001107","alias":"samp_name","owner":"MimsHostAssociated","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"size_frac":{"name":"size_frac","annotations":{"Expected_value":{"tag":"Expected_value","value":"filter size value range"}},"description":"Filtering pore size used in sample preparation","title":"size fraction selected","examples":[{"value":"0-0.22 micrometer"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["fraction","size"],"string_serialization":"{float}-{float} {unit}","slot_uri":"MIXS:0000017","alias":"size_frac","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"lib_screen":{"name":"lib_screen","annotations":{"Expected_value":{"tag":"Expected_value","value":"screening strategy name"}},"description":"Specific enrichment or screening methods applied before and/or after creating libraries","title":"library screening strategy","examples":[{"value":"enriched, screened, normalized"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"string_serialization":"{text}","slot_uri":"MIXS:0000043","alias":"lib_screen","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"ref_db":{"name":"ref_db","annotations":{"Expected_value":{"tag":"Expected_value","value":"names, versions, and references of databases"}},"description":"List of database(s) used for ORF annotation, along with version number and reference to website or publication","title":"reference database(s)","examples":[{"value":"pVOGs;5;http://dmk-brain.ecn.uiowa.edu/pVOGs/ Grazziotin et al. 2017 doi:10.1093/nar/gkw975"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["database"],"string_serialization":"{database};{version};{reference}","slot_uri":"MIXS:0000062","alias":"ref_db","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"nucl_acid_amp":{"name":"nucl_acid_amp","description":"A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids","title":"nucleic acid amplification","examples":[{"value":"https://phylogenomics.me/protocols/16s-pcr-protocol/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000038","alias":"nucl_acid_amp","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_size":{"name":"lib_size","description":"Total number of clones in the library prepared for the project","title":"library size","examples":[{"value":"50"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","size"],"slot_uri":"MIXS:0000039","alias":"lib_size","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer","recommended":true},"mid":{"name":"mid","description":"Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to specifically tag unique samples in a sequencing run. Sequence should be reported in uppercase letters","title":"multiplex identifiers","examples":[{"value":"GTGAATAT"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["identifier"],"slot_uri":"MIXS:0000047","alias":"mid","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[ACGTRKSYMWBHDVN]+$","structured_pattern":{"syntax":"^{ambiguous_nucleotides}$","interpolated":true,"partial_match":true}},"assembly_name":{"name":"assembly_name","annotations":{"Expected_value":{"tag":"Expected_value","value":"name and version of assembly"}},"description":"Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community","title":"assembly name","examples":[{"value":"HuRef, JCVI_ISG_i3_1.0"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text} {text}","slot_uri":"MIXS:0000057","alias":"assembly_name","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"temp":{"name":"temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Temperature of the sample at the time of sampling","title":"temperature","examples":[{"value":"25 degree Celsius"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["temperature"],"slot_uri":"MIXS:0000113","alias":"temp","owner":"MimsHostAssociated","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"nucl_acid_ext":{"name":"nucl_acid_ext","description":"A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample","title":"nucleic acid extraction","examples":[{"value":"https://mobio.com/media/wysiwyg/pdfs/protocols/12888.pdf"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000037","alias":"nucl_acid_ext","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"samp_size":{"name":"samp_size","description":"The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected","title":"amount or size of sample collected","examples":[{"value":"5 liter"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["sample","size"],"slot_uri":"MIXS:0000001","alias":"samp_size","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"alt":{"name":"alt","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth\'s surface. In this context, the altitude measurement is the vertical distance between the earth\'s surface above sea level and the sampled position in the air","title":"altitude","examples":[{"value":"100 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000094","alias":"alt","owner":"MimsHostAssociated","domain_of":["Mims","HostAssociated","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"source_mat_id":{"name":"source_mat_id","annotations":{"Expected_value":{"tag":"Expected_value","value":"for cultures of microorganisms: identifiers for two culture collections; for other material a unique arbitrary identifer"}},"description":"A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain \'UAM:Herps:14\' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2)","title":"source material identifiers","examples":[{"value":"MPI012345"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["identifier","material","source"],"string_serialization":"{text}","slot_uri":"MIXS:0000026","alias":"source_mat_id","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true},"samp_vol_we_dna_ext":{"name":"samp_vol_we_dna_ext","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milliliter, gram, milligram, square centimeter"}},"description":"Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001)","title":"sample volume or weight for DNA extraction","examples":[{"value":"1500 milliliter"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["dna","sample","volume","weight"],"slot_uri":"MIXS:0000111","alias":"samp_vol_we_dna_ext","owner":"MimsHostAssociated","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"lib_reads_seqd":{"name":"lib_reads_seqd","description":"Total number of clones sequenced from the library","title":"library reads sequenced","examples":[{"value":"20"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"slot_uri":"MIXS:0000040","alias":"lib_reads_seqd","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer","recommended":true},"rel_to_oxygen":{"name":"rel_to_oxygen","description":"Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments","title":"relationship to oxygen","examples":[{"value":"aerobe"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["oxygen","relationship"],"slot_uri":"MIXS:0000015","alias":"rel_to_oxygen","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"REL_TO_OXYGEN_ENUM"},"samp_collect_device":{"name":"samp_collect_device","annotations":{"Expected_value":{"tag":"Expected_value","value":"device name"}},"description":"The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094)","title":"sample collection device","examples":[{"value":"swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713]"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["device","sample"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000002","alias":"samp_collect_device","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"number_contig":{"name":"number_contig","description":"Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG","title":"number of contigs","examples":[{"value":"40"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["number"],"slot_uri":"MIXS:0000060","alias":"number_contig","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer","recommended":true},"lib_layout":{"name":"lib_layout","description":"Specify whether to expect single, paired, or other configuration of reads","title":"library layout","examples":[{"value":"paired"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"slot_uri":"MIXS:0000041","alias":"lib_layout","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"LIB_LAYOUT_ENUM","recommended":true},"assembly_qual":{"name":"assembly_qual","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling   90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling < 90% of the expected genome or replicon sequence, or for which no genome size could be estimated","title":"assembly quality","examples":[{"value":"High-quality draft genome"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["quality"],"string_serialization":"[Finished genome|High-quality draft genome|Medium-quality draft genome|Low-quality draft genome|Genome fragment(s)]","slot_uri":"MIXS:0000056","alias":"assembly_qual","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"ref_biomaterial":{"name":"ref_biomaterial","description":"Primary publication if isolated before genome publication; otherwise, primary genome report","title":"reference for biomaterial","examples":[{"value":"doi:10.1016/j.syapm.2018.01.009"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000025","alias":"ref_biomaterial","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"project_name":{"name":"project_name","description":"Name of the project within which the sequencing was organized","title":"project name","examples":[{"value":"Forest soil metagenome"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["project"],"slot_uri":"MIXS:0000092","alias":"project_name","owner":"MimsHostAssociated","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"lib_vector":{"name":"lib_vector","annotations":{"Expected_value":{"tag":"Expected_value","value":"vector"}},"description":"Cloning vector type(s) used in construction of libraries","title":"library vector","examples":[{"value":"Bacteriophage P1"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"string_serialization":"{text}","slot_uri":"MIXS:0000042","alias":"lib_vector","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"adapters":{"name":"adapters","annotations":{"Expected_value":{"tag":"Expected_value","value":"adapter A and B sequence"}},"description":"Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters","title":"adapters","examples":[{"value":"AATGATACGGCGACCACCGAGATCTACACGCT;CAAGCAGAAGACGGCATACGAGAT"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{dna};{dna}","slot_uri":"MIXS:0000048","alias":"adapters","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"neg_cont_type":{"name":"neg_cont_type","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration or text"}},"description":"The substance or equipment used as a negative control in an investigation","title":"negative control type","in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["type"],"slot_uri":"MIXS:0001321","alias":"neg_cont_type","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"NEG_CONT_TYPE_ENUM","recommended":true},"assembly_software":{"name":"assembly_software","description":"Tool(s) used for assembly, including version number and parameters","title":"assembly software","examples":[{"value":"metaSPAdes;3.11.0;kmer set 21,33,55,77,99,121, default parameters otherwise"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["software"],"slot_uri":"MIXS:0000058","alias":"assembly_software","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"annot":{"name":"annot","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of tool or pipeline used, or annotation source description"}},"description":"Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter","title":"annotation","examples":[{"value":"prokka"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:0000059","alias":"annot","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"pos_cont_type":{"name":"pos_cont_type","description":"The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive","title":"positive control type","in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["type"],"string_serialization":"{term} or {text}","slot_uri":"MIXS:0001322","alias":"pos_cont_type","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"feat_pred":{"name":"feat_pred","description":"Method used to predict UViGs features such as ORFs, integration site, etc","title":"feature prediction","examples":[{"value":"Prodigal;2.6.3;default parameters"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["feature","predict"],"slot_uri":"MIXS:0000061","alias":"feat_pred","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"env_local_scale":{"name":"env_local_scale","annotations":{"Expected_value":{"tag":"Expected_value","value":"Environmental entities having causal influences upon the entity at time of sampling"}},"description":"Report the entity or entities which are in the sample or specimen s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS","title":"local environmental context","examples":[{"value":"hillside [ENVO:01000333]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["context","environmental"],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000013","alias":"env_local_scale","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"samp_mat_process":{"name":"samp_mat_process","description":"A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed","title":"sample material processing","examples":[{"value":"filtering of seawater, storing samples in ethanol"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["material","process","sample"],"slot_uri":"MIXS:0000016","alias":"samp_mat_process","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"sim_search_meth":{"name":"sim_search_meth","description":"Tool used to compare ORFs with database, along with version and cutoffs used","title":"similarity search method","examples":[{"value":"HMMER3;3.1b2;hmmsearch, cutoff of 50 on score"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["method"],"slot_uri":"MIXS:0000063","alias":"sim_search_meth","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"depth":{"name":"depth","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"The vertical distance below local surface. For sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples","title":"depth","examples":[{"value":"10 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["depth"],"slot_uri":"MIXS:0000018","alias":"depth","owner":"MimsHostAssociated","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_collect_method":{"name":"samp_collect_method","description":"The method employed for collecting the sample","title":"sample collection method","examples":[{"value":"swabbing"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["method","sample"],"slot_uri":"MIXS:0001225","alias":"samp_collect_method","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$|([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}|{text}$","interpolated":true,"partial_match":true}},"env_medium":{"name":"env_medium","description":"Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of \'environmental material\' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top)","title":"environmental medium","examples":[{"value":"bluegrass field soil [ENVO:00005789]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["environmental"],"slot_uri":"MIXS:0000014","alias":"env_medium","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\]$","structured_pattern":{"syntax":"^{termLabel} \\\\[{termID}\\\\]$","interpolated":true,"partial_match":true}},"samp_taxon_id":{"name":"samp_taxon_id","description":"NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use \'synthetic metagenome  for mock community/positive controls, or \'blank sample\' for negative controls","title":"taxonomy ID of DNA sample","examples":[{"value":"Gut Metagenome [NCBITaxon:749906]"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dna","identifier","sample","taxon"],"slot_uri":"MIXS:0001320","alias":"samp_taxon_id","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[NCBITaxon:\\\\d+\\\\]$","structured_pattern":{"syntax":"^{text} \\\\[{NCBItaxon_id}\\\\]$","interpolated":true,"partial_match":true}},"geo_loc_name":{"name":"geo_loc_name","description":"The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)","title":"geographic location (country and/or sea,region)","examples":[{"value":"USA: Maryland, Bethesda"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["geographic","location"],"slot_uri":"MIXS:0000010","alias":"geo_loc_name","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+): ([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+), ([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{text}: {text}, {text}$","interpolated":true,"partial_match":true}},"collection_date":{"name":"collection_date","description":"The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant","title":"collection date","examples":[{"value":"2013-03-25T12:42:31+01:00"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["date"],"slot_uri":"MIXS:0000011","alias":"collection_date","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"datetime","required":true},"seq_meth":{"name":"seq_meth","description":"Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103)","title":"sequencing method","examples":[{"value":"454 Genome Sequencer FLX [OBI:0000702]"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["method"],"slot_uri":"MIXS:0000050","alias":"seq_meth","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)|(([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\])$","structured_pattern":{"syntax":"^{text}|({termLabel} \\\\[{termID}\\\\])$","interpolated":true,"partial_match":true}},"lat_lon":{"name":"lat_lon","annotations":{"Expected_value":{"tag":"Expected_value","value":"decimal degrees,  limit to 8 decimal points"}},"description":"The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system","title":"geographic location (latitude and longitude)","examples":[{"value":"50.586825 6.408977"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["geographic","location"],"string_serialization":"{float} {float}","slot_uri":"MIXS:0000009","alias":"lat_lon","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^(-?((?:[0-8]?[0-9](?:\\\\.\\\\d{0,8})?)|90)) -?[0-9]+(?:\\\\.[0-9]{0,8})?$|^-?(1[0-7]{1,2})$","structured_pattern":{"syntax":"^{lat} {lon}$","interpolated":true,"partial_match":true}},"elev":{"name":"elev","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth\'s surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit","title":"elevation","examples":[{"value":"100 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["elevation"],"slot_uri":"MIXS:0000093","alias":"elev","owner":"MimsHostAssociated","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"env_broad_scale":{"name":"env_broad_scale","description":"Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS","title":"broad-scale environmental context","examples":[{"value":"rangeland biome [ENVO:01000247]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["context","environmental"],"slot_uri":"MIXS:0000012","alias":"env_broad_scale","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\]$","structured_pattern":{"syntax":"^{termLabel} \\\\[{termID}\\\\]$","interpolated":true,"partial_match":true}},"tax_class":{"name":"tax_class","description":"Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes","title":"taxonomic classification","examples":[{"value":"vConTACT vContact2 (references from NCBI RefSeq v83, genus rank classification, default parameters)"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["classification","taxon"],"slot_uri":"MIXS:0000064","alias":"tax_class","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"experimental_factor":{"name":"experimental_factor","annotations":{"Expected_value":{"tag":"Expected_value","value":"text or EFO and/or OBI"}},"description":"Variable aspects of an experiment design that can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI)","title":"experimental factor","examples":[{"value":"time series design [EFO:0001779]"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["experimental","factor"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000008","alias":"experimental_factor","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true,"pattern":"^\\\\S+.*\\\\S+ \\\\[[a-zA-Z]{2,}:\\\\d+\\\\]$"},"associated_resource":{"name":"associated_resource","annotations":{"Expected_value":{"tag":"Expected_value","value":"reference to resource"}},"description":"A related resource that is referenced, cited, or otherwise associated to the sequence","title":"relevant electronic resources","examples":[{"value":"http://www.earthmicrobiome.org/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["resource"],"string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000091","alias":"associated_resource","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true},"sop":{"name":"sop","annotations":{"Expected_value":{"tag":"Expected_value","value":"reference to SOP"}},"description":"Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences","title":"relevant standard operating procedures","examples":[{"value":"http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/its/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["procedures"],"string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000090","alias":"sop","owner":"MimsHostAssociated","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true},"ances_data":{"name":"ances_data","description":"Information about either pedigree or other ancestral information description (e.g. parental variety in case of mutant or selection), e.g. A/3*B (meaning [(A x B) x B] x B)","title":"ancestral data","examples":[{"value":"A/3*B"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000247","alias":"ances_data","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"biol_stat":{"name":"biol_stat","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"The level of genome modification","title":"biological status","examples":[{"value":"natural"}],"from_schema":"https://w3id.org/mixs","keywords":["status"],"string_serialization":"[wild|natural|semi-natural|inbred line|breeder\'s line|hybrid|clonal selection|mutant]","slot_uri":"MIXS:0000858","alias":"biol_stat","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"genetic_mod":{"name":"genetic_mod","annotations":{"Expected_value":{"tag":"Expected_value","value":"PMID, DOI, URL or text"}},"description":"Genetic modifications of the genome of an organism, which may occur naturally by spontaneous mutation, or be introduced by some experimental means, e.g. specification of a transgene or the gene knocked-out or details of transient transfection","title":"genetic modification","examples":[{"value":"aox1A transgenic"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000859","alias":"genetic_mod","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_common_name":{"name":"host_common_name","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":""}},"description":"Common name of the host","title":"host common name","examples":[{"value":"human"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000248","alias":"host_common_name","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"samp_capt_status":{"name":"samp_capt_status","description":"Reason for the sample","title":"sample capture status","examples":[{"value":"farm sample"}],"from_schema":"https://w3id.org/mixs","keywords":["sample","status"],"slot_uri":"MIXS:0000860","alias":"samp_capt_status","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"SAMP_CAPT_STATUS_ENUM"},"samp_dis_stage":{"name":"samp_dis_stage","description":"Stage of the disease at the time of sample collection, e.g. inoculation, penetration, infection, growth and reproduction, dissemination of pathogen","title":"sample disease stage","examples":[{"value":"infection"}],"from_schema":"https://w3id.org/mixs","keywords":["disease","sample"],"slot_uri":"MIXS:0000249","alias":"samp_dis_stage","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"SAMP_DIS_STAGE_ENUM"},"host_taxid":{"name":"host_taxid","annotations":{"Expected_value":{"tag":"Expected_value","value":"NCBI taxon identifier"}},"description":"NCBI taxon id of the host, e.g. 9606","title":"host taxid","examples":[{"value":"7955"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","taxon"],"string_serialization":"{NCBI taxid}","slot_uri":"MIXS:0000250","alias":"host_taxid","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_subject_id":{"name":"host_subject_id","description":"A unique identifier by which each subject can be referred to, de-identified","title":"host subject id","examples":[{"value":"MPI123"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","identifier"],"slot_uri":"MIXS:0000861","alias":"host_subject_id","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_age":{"name":"host_age","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"year, day, hour"}},"description":"Age of host at the time of sampling; relevant scale depends on species and study, e.g. Could be seconds for amoebae or centuries for trees","title":"host age","examples":[{"value":"10 days"}],"from_schema":"https://w3id.org/mixs","keywords":["age","host","host."],"slot_uri":"MIXS:0000255","alias":"host_age","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_life_stage":{"name":"host_life_stage","annotations":{"Expected_value":{"tag":"Expected_value","value":"stage"}},"description":"Description of life stage of host","title":"host life stage","examples":[{"value":"adult"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","life"],"string_serialization":"{text}","slot_uri":"MIXS:0000251","alias":"host_life_stage","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_sex":{"name":"host_sex","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Gender or physical sex of the host","title":"host sex","comments":["example of non-binary from Excel sheets does not match any of the enumerated values"],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"string_serialization":"[female|hermaphrodite|non-binary|male|transgender|transgender (female to male)|transgender (male to female)\\n|undeclared]","slot_uri":"MIXS:0000811","alias":"host_sex","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_disease_stat":{"name":"host_disease_stat","annotations":{"Expected_value":{"tag":"Expected_value","value":"disease name or Disease Ontology term"}},"description":"List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text","title":"host disease status","examples":[{"value":"rabies [DOID:11260]"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["disease","host","host.","status"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000031","alias":"host_disease_stat","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"chem_administration":{"name":"chem_administration","annotations":{"Expected_value":{"tag":"Expected_value","value":"CHEBI;timestamp"}},"description":"List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi","title":"chemical administration","examples":[{"value":"agar [CHEBI:2509];2018-05-11T20:00Z"}],"from_schema":"https://w3id.org/mixs","keywords":["administration"],"string_serialization":"{termLabel} [{termID}];{timestamp}","slot_uri":"MIXS:0000751","alias":"chem_administration","owner":"MimsHostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"host_body_habitat":{"name":"host_body_habitat","description":"Original body habitat where the sample was obtained from","title":"host body habitat","examples":[{"value":"nasopharynx"}],"from_schema":"https://w3id.org/mixs","keywords":["body","habitat","host","host."],"slot_uri":"MIXS:0000866","alias":"host_body_habitat","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_body_site":{"name":"host_body_site","annotations":{"Expected_value":{"tag":"Expected_value","value":"FMA or UBERON"}},"description":"Name of body site where the sample was obtained from, such as a specific organ or tissue (tongue, lung etc...). Use terms from the foundational model of anatomy ontology (fma) or the Uber-anatomy ontology (UBERON)","title":"host body site","examples":[{"value":"gill [UBERON:0002535]"}],"from_schema":"https://w3id.org/mixs","keywords":["body","host","host.","site"],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000867","alias":"host_body_site","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_body_product":{"name":"host_body_product","annotations":{"Expected_value":{"tag":"Expected_value","value":"FMA or UBERON"}},"description":"Substance produced by the body, e.g. Stool, mucus, where the sample was obtained from. Use terms from the foundational model of anatomy ontology (fma) or Uber-anatomy ontology (UBERON)","title":"host body product","examples":[{"value":"mucus [FMA:66938]"}],"from_schema":"https://w3id.org/mixs","keywords":["body","host","host.","product"],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000888","alias":"host_body_product","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_tot_mass":{"name":"host_tot_mass","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"kilogram, gram"}},"description":"Total mass of the host at collection, the unit depends on host","title":"host total mass","examples":[{"value":"2500 gram"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","mass","total"],"slot_uri":"MIXS:0000263","alias":"host_tot_mass","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_height":{"name":"host_height","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"centimeter, millimeter, meter"}},"description":"The height of subject","title":"host height","examples":[{"value":"0.1 meter"}],"from_schema":"https://w3id.org/mixs","keywords":["height","host","host."],"slot_uri":"MIXS:0000264","alias":"host_height","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_length":{"name":"host_length","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"centimeter, millimeter, meter"}},"description":"The length of subject","title":"host length","examples":[{"value":"1 meter"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","length"],"slot_uri":"MIXS:0000256","alias":"host_length","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_diet":{"name":"host_diet","description":"Type of diet depending on the host, for animals omnivore, herbivore etc., for humans high-fat, meditteranean etc.; can include multiple diet types","title":"host diet","examples":[{"value":"herbivore"}],"from_schema":"https://w3id.org/mixs","keywords":["diet","host","host."],"slot_uri":"MIXS:0000869","alias":"host_diet","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"host_last_meal":{"name":"host_last_meal","annotations":{"Expected_value":{"tag":"Expected_value","value":"content;duration"}},"description":"Content of last meal and time since feeding; can include multiple values","title":"host last meal","examples":[{"value":"corn feed;P2H"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"string_serialization":"{text};{duration}","slot_uri":"MIXS:0000870","alias":"host_last_meal","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"host_growth_cond":{"name":"host_growth_cond","description":"Literature reference giving growth conditions of the host","title":"host growth conditions","examples":[{"value":"https://academic.oup.com/icesjms/article/68/2/349/617247"}],"from_schema":"https://w3id.org/mixs","keywords":["condition","growth","host","host."],"slot_uri":"MIXS:0000871","alias":"host_growth_cond","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$|([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}|{text}$","interpolated":true,"partial_match":true}},"host_substrate":{"name":"host_substrate","description":"The growth substrate of the host","title":"host substrate","examples":[{"value":"rock"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000252","alias":"host_substrate","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_fam_rel":{"name":"host_fam_rel","annotations":{"Expected_value":{"tag":"Expected_value","value":"relationship type;arbitrary identifier"}},"description":"Relationships to other hosts in the same study; can include multiple relationships","title":"host family relationship","examples":[{"value":"offspring;Mussel25"}],"from_schema":"https://w3id.org/mixs","keywords":["family","host","host.","relationship"],"string_serialization":"{text};{text}","slot_uri":"MIXS:0000872","alias":"host_fam_rel","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"host_subspecf_genlin":{"name":"host_subspecf_genlin","annotations":{"Expected_value":{"tag":"Expected_value","value":"Genetic lineage below lowest rank of NCBI taxonomy, which is subspecies, e.g. serovar, biotype, ecotype, variety, cultivar"}},"description":"Information about the genetic distinctness of the host organism below the subspecies level e.g., serovar, serotype, biotype, ecotype, variety, cultivar, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123","title":"host subspecific genetic lineage","examples":[{"value":"serovar:Newport, variety:glabrum, cultivar: Red Delicious"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","lineage"],"string_serialization":"{rank name}:{text}","slot_uri":"MIXS:0001318","alias":"host_subspecf_genlin","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"host_genotype":{"name":"host_genotype","description":"Observed genotype","title":"host genotype","examples":[{"value":"C57BL/6"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000365","alias":"host_genotype","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_phenotype":{"name":"host_phenotype","annotations":{"Expected_value":{"tag":"Expected_value","value":"PATO or HP"}},"description":"Phenotype of human or other host. Use terms from the phenotypic quality ontology (pato) or the Human Phenotype Ontology (HP)","title":"host phenotype","examples":[{"value":"elongated [PATO:0001154]"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000874","alias":"host_phenotype","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_body_temp":{"name":"host_body_temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Core body temperature of the host when sample was collected","title":"host body temperature","examples":[{"value":"15 degree Celsius"}],"from_schema":"https://w3id.org/mixs","keywords":["body","host","host.","temperature"],"slot_uri":"MIXS:0000274","alias":"host_body_temp","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_dry_mass":{"name":"host_dry_mass","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"kilogram, gram"}},"description":"Measurement of dry mass","title":"host dry mass","examples":[{"value":"500 gram"}],"from_schema":"https://w3id.org/mixs","keywords":["dry","host","host.","mass"],"slot_uri":"MIXS:0000257","alias":"host_dry_mass","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"blood_press_diast":{"name":"blood_press_diast","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"millimeter mercury"}},"description":"Resting diastolic blood pressure, measured as mm mercury","title":"host blood pressure diastolic","from_schema":"https://w3id.org/mixs","keywords":["host","host.","pressure"],"slot_uri":"MIXS:0000258","alias":"blood_press_diast","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"blood_press_syst":{"name":"blood_press_syst","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"millimeter mercury"}},"description":"Resting systolic blood pressure, measured as mm mercury","title":"host blood pressure systolic","from_schema":"https://w3id.org/mixs","keywords":["host","host.","pressure"],"slot_uri":"MIXS:0000259","alias":"blood_press_syst","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_color":{"name":"host_color","description":"The color of host","title":"host color","from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000260","alias":"host_color","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_shape":{"name":"host_shape","description":"Morphological shape of host","title":"host shape","examples":[{"value":"round"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000261","alias":"host_shape","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"gravidity":{"name":"gravidity","description":"Whether or not subject is gravid, and if yes date due or date post-conception, specifying which is used","title":"gravidity","examples":[{"value":"yes;due date:2018-05-11"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{boolean};{timestamp}","slot_uri":"MIXS:0000875","alias":"gravidity","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"perturbation":{"name":"perturbation","annotations":{"Expected_value":{"tag":"Expected_value","value":"perturbation type name;perturbation interval and duration"}},"description":"Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types","title":"perturbation","examples":[{"value":"antibiotic addition;R2/2018-05-11T14:30Z/2018-05-11T19:30Z/P1H30M"}],"from_schema":"https://w3id.org/mixs","keywords":["perturbation"],"string_serialization":"{text};{Rn/start_time/end_time/duration}","slot_uri":"MIXS:0000754","alias":"perturbation","owner":"MimsHostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"salinity":{"name":"salinity","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"practical salinity unit, percentage"}},"description":"The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater","title":"salinity","examples":[{"value":"25 practical salinity unit"}],"from_schema":"https://w3id.org/mixs","keywords":["salinity"],"slot_uri":"MIXS:0000183","alias":"salinity","owner":"MimsHostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"oxy_stat_samp":{"name":"oxy_stat_samp","description":"Oxygenation status of sample","title":"oxygenation status of sample","examples":[{"value":"aerobic"}],"from_schema":"https://w3id.org/mixs","keywords":["oxygen","sample","status"],"slot_uri":"MIXS:0000753","alias":"oxy_stat_samp","owner":"MimsHostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"OXY_STAT_SAMP_ENUM"},"organism_count":{"name":"organism_count","annotations":{"Expected_value":{"tag":"Expected_value","value":"organism name;measurement value;enumeration"}},"description":"Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)","title":"organism count","examples":[{"value":"total prokaryotes;3.5e7 cells per milliliter;qPCR"}],"from_schema":"https://w3id.org/mixs","keywords":["count","organism"],"string_serialization":"{text};{float} {unit};[ATP|MPN|qPCR|other]","slot_uri":"MIXS:0000103","alias":"organism_count","owner":"MimsHostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"samp_store_temp":{"name":"samp_store_temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Temperature at which sample was stored, e.g. -80 degree Celsius","title":"sample storage temperature","examples":[{"value":"-80 degree Celsius"}],"from_schema":"https://w3id.org/mixs","keywords":["sample","storage","temperature"],"slot_uri":"MIXS:0000110","alias":"samp_store_temp","owner":"MimsHostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_store_dur":{"name":"samp_store_dur","description":"Duration for which the sample was stored. Indicate the duration for which the sample was stored written in ISO 8601 format","title":"sample storage duration","examples":[{"value":"P1Y6M"}],"from_schema":"https://w3id.org/mixs","keywords":["duration","period","sample","storage"],"slot_uri":"MIXS:0000116","alias":"samp_store_dur","owner":"MimsHostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^P(?:(?:\\\\d+D|\\\\d+M(?:\\\\d+D)?|\\\\d+Y(?:\\\\d+M(?:\\\\d+D)?)?)(?:T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S))?|T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S)|\\\\d+W)$","structured_pattern":{"syntax":"^{duration}$","interpolated":true,"partial_match":true}},"samp_store_loc":{"name":"samp_store_loc","annotations":{"Expected_value":{"tag":"Expected_value","value":"location name"}},"description":"Location at which sample was stored, usually name of a specific freezer/room","title":"sample storage location","examples":[{"value":"Freezer no:5"}],"from_schema":"https://w3id.org/mixs","keywords":["location","sample","storage"],"string_serialization":"{text}","slot_uri":"MIXS:0000755","alias":"samp_store_loc","owner":"MimsHostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"host_symbiont":{"name":"host_symbiont","annotations":{"Expected_value":{"tag":"Expected_value","value":"species name or common name"}},"description":"The taxonomic name of the organism(s) found living in mutualistic, commensalistic, or parasitic symbiosis with the specific host. The sampled symbiont can have its own symbionts. For example, parasites may have hyperparasites (=parasites of the parasite)","title":"observed host symbionts","examples":[{"value":"flukeworms"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","observed","symbiosis"],"string_serialization":"{text}","slot_uri":"MIXS:0001298","alias":"host_symbiont","owner":"MimsHostAssociated","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"misc_param":{"name":"misc_param","annotations":{"Expected_value":{"tag":"Expected_value","value":"parameter name;measurement value"}},"description":"Any other measurement performed or parameter collected, that is not listed here","title":"miscellaneous parameter","examples":[{"value":"Bicarbonate ion concentration;2075 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","keywords":["parameter"],"string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000752","alias":"misc_param","owner":"MimsHostAssociated","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true}},"class_uri":"MIXS:0010007_0016002"},"MimsSediment":{"name":"MimsSediment","description":"MIxS data that complies with the Mims checklist and the Sediment Extension","title":"Mims combined with Sediment","in_subset":["combination_classes"],"from_schema":"https://w3id.org/mixs","is_a":"Sediment","mixins":["Mims"],"attributes":{"samp_name":{"name":"samp_name","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":""}},"description":"A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name","title":"sample name","examples":[{"value":"ISDsoil1"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["sample"],"slot_uri":"MIXS:0001107","alias":"samp_name","owner":"MimsSediment","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"size_frac":{"name":"size_frac","annotations":{"Expected_value":{"tag":"Expected_value","value":"filter size value range"}},"description":"Filtering pore size used in sample preparation","title":"size fraction selected","examples":[{"value":"0-0.22 micrometer"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["fraction","size"],"string_serialization":"{float}-{float} {unit}","slot_uri":"MIXS:0000017","alias":"size_frac","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"lib_screen":{"name":"lib_screen","annotations":{"Expected_value":{"tag":"Expected_value","value":"screening strategy name"}},"description":"Specific enrichment or screening methods applied before and/or after creating libraries","title":"library screening strategy","examples":[{"value":"enriched, screened, normalized"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"string_serialization":"{text}","slot_uri":"MIXS:0000043","alias":"lib_screen","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"ref_db":{"name":"ref_db","annotations":{"Expected_value":{"tag":"Expected_value","value":"names, versions, and references of databases"}},"description":"List of database(s) used for ORF annotation, along with version number and reference to website or publication","title":"reference database(s)","examples":[{"value":"pVOGs;5;http://dmk-brain.ecn.uiowa.edu/pVOGs/ Grazziotin et al. 2017 doi:10.1093/nar/gkw975"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["database"],"string_serialization":"{database};{version};{reference}","slot_uri":"MIXS:0000062","alias":"ref_db","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"nucl_acid_amp":{"name":"nucl_acid_amp","description":"A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids","title":"nucleic acid amplification","examples":[{"value":"https://phylogenomics.me/protocols/16s-pcr-protocol/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000038","alias":"nucl_acid_amp","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_size":{"name":"lib_size","description":"Total number of clones in the library prepared for the project","title":"library size","examples":[{"value":"50"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","size"],"slot_uri":"MIXS:0000039","alias":"lib_size","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer","recommended":true},"mid":{"name":"mid","description":"Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to specifically tag unique samples in a sequencing run. Sequence should be reported in uppercase letters","title":"multiplex identifiers","examples":[{"value":"GTGAATAT"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["identifier"],"slot_uri":"MIXS:0000047","alias":"mid","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[ACGTRKSYMWBHDVN]+$","structured_pattern":{"syntax":"^{ambiguous_nucleotides}$","interpolated":true,"partial_match":true}},"assembly_name":{"name":"assembly_name","annotations":{"Expected_value":{"tag":"Expected_value","value":"name and version of assembly"}},"description":"Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community","title":"assembly name","examples":[{"value":"HuRef, JCVI_ISG_i3_1.0"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text} {text}","slot_uri":"MIXS:0000057","alias":"assembly_name","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"temp":{"name":"temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Temperature of the sample at the time of sampling","title":"temperature","examples":[{"value":"25 degree Celsius"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["temperature"],"slot_uri":"MIXS:0000113","alias":"temp","owner":"MimsSediment","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"nucl_acid_ext":{"name":"nucl_acid_ext","description":"A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample","title":"nucleic acid extraction","examples":[{"value":"https://mobio.com/media/wysiwyg/pdfs/protocols/12888.pdf"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000037","alias":"nucl_acid_ext","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"samp_size":{"name":"samp_size","description":"The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected","title":"amount or size of sample collected","examples":[{"value":"5 liter"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["sample","size"],"slot_uri":"MIXS:0000001","alias":"samp_size","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"alt":{"name":"alt","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth\'s surface. In this context, the altitude measurement is the vertical distance between the earth\'s surface above sea level and the sampled position in the air","title":"altitude","examples":[{"value":"100 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000094","alias":"alt","owner":"MimsSediment","domain_of":["Mims","HostAssociated","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"source_mat_id":{"name":"source_mat_id","annotations":{"Expected_value":{"tag":"Expected_value","value":"for cultures of microorganisms: identifiers for two culture collections; for other material a unique arbitrary identifer"}},"description":"A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain \'UAM:Herps:14\' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2)","title":"source material identifiers","examples":[{"value":"MPI012345"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["identifier","material","source"],"string_serialization":"{text}","slot_uri":"MIXS:0000026","alias":"source_mat_id","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true},"samp_vol_we_dna_ext":{"name":"samp_vol_we_dna_ext","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milliliter, gram, milligram, square centimeter"}},"description":"Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001)","title":"sample volume or weight for DNA extraction","examples":[{"value":"1500 milliliter"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["dna","sample","volume","weight"],"slot_uri":"MIXS:0000111","alias":"samp_vol_we_dna_ext","owner":"MimsSediment","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"lib_reads_seqd":{"name":"lib_reads_seqd","description":"Total number of clones sequenced from the library","title":"library reads sequenced","examples":[{"value":"20"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"slot_uri":"MIXS:0000040","alias":"lib_reads_seqd","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer","recommended":true},"rel_to_oxygen":{"name":"rel_to_oxygen","description":"Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments","title":"relationship to oxygen","examples":[{"value":"aerobe"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["oxygen","relationship"],"slot_uri":"MIXS:0000015","alias":"rel_to_oxygen","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"REL_TO_OXYGEN_ENUM"},"samp_collect_device":{"name":"samp_collect_device","annotations":{"Expected_value":{"tag":"Expected_value","value":"device name"}},"description":"The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094)","title":"sample collection device","examples":[{"value":"swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713]"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["device","sample"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000002","alias":"samp_collect_device","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"number_contig":{"name":"number_contig","description":"Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG","title":"number of contigs","examples":[{"value":"40"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["number"],"slot_uri":"MIXS:0000060","alias":"number_contig","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer","recommended":true},"lib_layout":{"name":"lib_layout","description":"Specify whether to expect single, paired, or other configuration of reads","title":"library layout","examples":[{"value":"paired"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"slot_uri":"MIXS:0000041","alias":"lib_layout","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"LIB_LAYOUT_ENUM","recommended":true},"assembly_qual":{"name":"assembly_qual","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling   90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling < 90% of the expected genome or replicon sequence, or for which no genome size could be estimated","title":"assembly quality","examples":[{"value":"High-quality draft genome"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["quality"],"string_serialization":"[Finished genome|High-quality draft genome|Medium-quality draft genome|Low-quality draft genome|Genome fragment(s)]","slot_uri":"MIXS:0000056","alias":"assembly_qual","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"ref_biomaterial":{"name":"ref_biomaterial","description":"Primary publication if isolated before genome publication; otherwise, primary genome report","title":"reference for biomaterial","examples":[{"value":"doi:10.1016/j.syapm.2018.01.009"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000025","alias":"ref_biomaterial","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"project_name":{"name":"project_name","description":"Name of the project within which the sequencing was organized","title":"project name","examples":[{"value":"Forest soil metagenome"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["project"],"slot_uri":"MIXS:0000092","alias":"project_name","owner":"MimsSediment","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"lib_vector":{"name":"lib_vector","annotations":{"Expected_value":{"tag":"Expected_value","value":"vector"}},"description":"Cloning vector type(s) used in construction of libraries","title":"library vector","examples":[{"value":"Bacteriophage P1"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"string_serialization":"{text}","slot_uri":"MIXS:0000042","alias":"lib_vector","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"adapters":{"name":"adapters","annotations":{"Expected_value":{"tag":"Expected_value","value":"adapter A and B sequence"}},"description":"Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters","title":"adapters","examples":[{"value":"AATGATACGGCGACCACCGAGATCTACACGCT;CAAGCAGAAGACGGCATACGAGAT"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{dna};{dna}","slot_uri":"MIXS:0000048","alias":"adapters","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"neg_cont_type":{"name":"neg_cont_type","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration or text"}},"description":"The substance or equipment used as a negative control in an investigation","title":"negative control type","in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["type"],"slot_uri":"MIXS:0001321","alias":"neg_cont_type","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"NEG_CONT_TYPE_ENUM","recommended":true},"assembly_software":{"name":"assembly_software","description":"Tool(s) used for assembly, including version number and parameters","title":"assembly software","examples":[{"value":"metaSPAdes;3.11.0;kmer set 21,33,55,77,99,121, default parameters otherwise"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["software"],"slot_uri":"MIXS:0000058","alias":"assembly_software","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"annot":{"name":"annot","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of tool or pipeline used, or annotation source description"}},"description":"Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter","title":"annotation","examples":[{"value":"prokka"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:0000059","alias":"annot","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"pos_cont_type":{"name":"pos_cont_type","description":"The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive","title":"positive control type","in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["type"],"string_serialization":"{term} or {text}","slot_uri":"MIXS:0001322","alias":"pos_cont_type","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"feat_pred":{"name":"feat_pred","description":"Method used to predict UViGs features such as ORFs, integration site, etc","title":"feature prediction","examples":[{"value":"Prodigal;2.6.3;default parameters"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["feature","predict"],"slot_uri":"MIXS:0000061","alias":"feat_pred","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"env_local_scale":{"name":"env_local_scale","annotations":{"Expected_value":{"tag":"Expected_value","value":"Environmental entities having causal influences upon the entity at time of sampling"}},"description":"Report the entity or entities which are in the sample or specimen s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS","title":"local environmental context","examples":[{"value":"hillside [ENVO:01000333]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["context","environmental"],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000013","alias":"env_local_scale","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"samp_mat_process":{"name":"samp_mat_process","description":"A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed","title":"sample material processing","examples":[{"value":"filtering of seawater, storing samples in ethanol"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["material","process","sample"],"slot_uri":"MIXS:0000016","alias":"samp_mat_process","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"sim_search_meth":{"name":"sim_search_meth","description":"Tool used to compare ORFs with database, along with version and cutoffs used","title":"similarity search method","examples":[{"value":"HMMER3;3.1b2;hmmsearch, cutoff of 50 on score"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["method"],"slot_uri":"MIXS:0000063","alias":"sim_search_meth","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"depth":{"name":"depth","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"The vertical distance below local surface. For sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples","title":"depth","examples":[{"value":"10 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["depth"],"slot_uri":"MIXS:0000018","alias":"depth","owner":"MimsSediment","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_collect_method":{"name":"samp_collect_method","description":"The method employed for collecting the sample","title":"sample collection method","examples":[{"value":"swabbing"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["method","sample"],"slot_uri":"MIXS:0001225","alias":"samp_collect_method","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$|([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}|{text}$","interpolated":true,"partial_match":true}},"env_medium":{"name":"env_medium","description":"Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of \'environmental material\' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top)","title":"environmental medium","examples":[{"value":"bluegrass field soil [ENVO:00005789]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["environmental"],"slot_uri":"MIXS:0000014","alias":"env_medium","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\]$","structured_pattern":{"syntax":"^{termLabel} \\\\[{termID}\\\\]$","interpolated":true,"partial_match":true}},"samp_taxon_id":{"name":"samp_taxon_id","description":"NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use \'synthetic metagenome  for mock community/positive controls, or \'blank sample\' for negative controls","title":"taxonomy ID of DNA sample","examples":[{"value":"Gut Metagenome [NCBITaxon:749906]"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dna","identifier","sample","taxon"],"slot_uri":"MIXS:0001320","alias":"samp_taxon_id","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[NCBITaxon:\\\\d+\\\\]$","structured_pattern":{"syntax":"^{text} \\\\[{NCBItaxon_id}\\\\]$","interpolated":true,"partial_match":true}},"geo_loc_name":{"name":"geo_loc_name","description":"The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)","title":"geographic location (country and/or sea,region)","examples":[{"value":"USA: Maryland, Bethesda"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["geographic","location"],"slot_uri":"MIXS:0000010","alias":"geo_loc_name","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+): ([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+), ([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{text}: {text}, {text}$","interpolated":true,"partial_match":true}},"collection_date":{"name":"collection_date","description":"The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant","title":"collection date","examples":[{"value":"2013-03-25T12:42:31+01:00"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["date"],"slot_uri":"MIXS:0000011","alias":"collection_date","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"datetime","required":true},"seq_meth":{"name":"seq_meth","description":"Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103)","title":"sequencing method","examples":[{"value":"454 Genome Sequencer FLX [OBI:0000702]"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["method"],"slot_uri":"MIXS:0000050","alias":"seq_meth","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)|(([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\])$","structured_pattern":{"syntax":"^{text}|({termLabel} \\\\[{termID}\\\\])$","interpolated":true,"partial_match":true}},"lat_lon":{"name":"lat_lon","annotations":{"Expected_value":{"tag":"Expected_value","value":"decimal degrees,  limit to 8 decimal points"}},"description":"The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system","title":"geographic location (latitude and longitude)","examples":[{"value":"50.586825 6.408977"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["geographic","location"],"string_serialization":"{float} {float}","slot_uri":"MIXS:0000009","alias":"lat_lon","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^(-?((?:[0-8]?[0-9](?:\\\\.\\\\d{0,8})?)|90)) -?[0-9]+(?:\\\\.[0-9]{0,8})?$|^-?(1[0-7]{1,2})$","structured_pattern":{"syntax":"^{lat} {lon}$","interpolated":true,"partial_match":true}},"elev":{"name":"elev","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth\'s surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit","title":"elevation","examples":[{"value":"100 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["elevation"],"slot_uri":"MIXS:0000093","alias":"elev","owner":"MimsSediment","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"env_broad_scale":{"name":"env_broad_scale","description":"Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS","title":"broad-scale environmental context","examples":[{"value":"rangeland biome [ENVO:01000247]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["context","environmental"],"slot_uri":"MIXS:0000012","alias":"env_broad_scale","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\]$","structured_pattern":{"syntax":"^{termLabel} \\\\[{termID}\\\\]$","interpolated":true,"partial_match":true}},"tax_class":{"name":"tax_class","description":"Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes","title":"taxonomic classification","examples":[{"value":"vConTACT vContact2 (references from NCBI RefSeq v83, genus rank classification, default parameters)"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["classification","taxon"],"slot_uri":"MIXS:0000064","alias":"tax_class","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"experimental_factor":{"name":"experimental_factor","annotations":{"Expected_value":{"tag":"Expected_value","value":"text or EFO and/or OBI"}},"description":"Variable aspects of an experiment design that can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI)","title":"experimental factor","examples":[{"value":"time series design [EFO:0001779]"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["experimental","factor"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000008","alias":"experimental_factor","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true,"pattern":"^\\\\S+.*\\\\S+ \\\\[[a-zA-Z]{2,}:\\\\d+\\\\]$"},"associated_resource":{"name":"associated_resource","annotations":{"Expected_value":{"tag":"Expected_value","value":"reference to resource"}},"description":"A related resource that is referenced, cited, or otherwise associated to the sequence","title":"relevant electronic resources","examples":[{"value":"http://www.earthmicrobiome.org/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["resource"],"string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000091","alias":"associated_resource","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true},"sop":{"name":"sop","annotations":{"Expected_value":{"tag":"Expected_value","value":"reference to SOP"}},"description":"Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences","title":"relevant standard operating procedures","examples":[{"value":"http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/its/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["procedures"],"string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000090","alias":"sop","owner":"MimsSediment","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true},"alkalinity":{"name":"alkalinity","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milliequivalent per liter, milligram per liter"}},"description":"Alkalinity, the ability of a solution to neutralize acids to the equivalence point of carbonate or bicarbonate","title":"alkalinity","examples":[{"value":"50 milligram per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["alkalinity"],"slot_uri":"MIXS:0000421","alias":"alkalinity","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"alkyl_diethers":{"name":"alkyl_diethers","description":"Concentration of alkyl diethers","title":"alkyl diethers","examples":[{"value":"0.005 mole per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000490","alias":"alkyl_diethers","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"aminopept_act":{"name":"aminopept_act","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"mole per liter per hour"}},"description":"Measurement of aminopeptidase activity","title":"aminopeptidase activity","examples":[{"value":"0.269 mole per liter per hour"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000172","alias":"aminopept_act","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"ammonium":{"name":"ammonium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of ammonium in the sample","title":"ammonium","examples":[{"value":"1.5 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000427","alias":"ammonium","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"bacteria_carb_prod":{"name":"bacteria_carb_prod","description":"Measurement of bacterial carbon production","title":"bacterial carbon production","examples":[{"value":"2.53 microgram per liter per hour"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","production"],"slot_uri":"MIXS:0000173","alias":"bacteria_carb_prod","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"biomass":{"name":"biomass","annotations":{"Expected_value":{"tag":"Expected_value","value":"biomass type;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"ton, kilogram, gram"}},"description":"Amount of biomass; should include the name for the part of biomass measured, e.g. Microbial, total. Can include multiple measurements","title":"biomass","examples":[{"value":"total;20 gram"}],"from_schema":"https://w3id.org/mixs","keywords":["biomass"],"string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000174","alias":"biomass","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"bishomohopanol":{"name":"bishomohopanol","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"microgram per liter, microgram per gram"}},"description":"Concentration of bishomohopanol","title":"bishomohopanol","examples":[{"value":"14 microgram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000175","alias":"bishomohopanol","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"bromide":{"name":"bromide","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"parts per million"}},"description":"Concentration of bromide","title":"bromide","examples":[{"value":"0.05 parts per million"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000176","alias":"bromide","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"calcium":{"name":"calcium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, micromole per liter, parts per million"}},"description":"Concentration of calcium in the sample","title":"calcium","examples":[{"value":"0.2 micromole per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000432","alias":"calcium","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"carb_nitro_ratio":{"name":"carb_nitro_ratio","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement value"}},"description":"Ratio of amount or concentrations of carbon to nitrogen. Used for quality control value in proteinaceous samples for radiocarbon dating.","title":"carbon/nitrogen ratio","examples":[{"value":"0.417361111"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","nitrogen","ratio"],"string_serialization":"{float}:{float}","slot_uri":"MIXS:0000310","alias":"carb_nitro_ratio","owner":"MimsSediment","domain_of":["Sediment","RadiocarbonDating","MimsSediment"],"range":"float","required":false,"recommended":true,"multivalued":false},"chem_administration":{"name":"chem_administration","annotations":{"Expected_value":{"tag":"Expected_value","value":"CHEBI;timestamp"}},"description":"List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi","title":"chemical administration","examples":[{"value":"agar [CHEBI:2509];2018-05-11T20:00Z"}],"from_schema":"https://w3id.org/mixs","keywords":["administration"],"string_serialization":"{termLabel} [{termID}];{timestamp}","slot_uri":"MIXS:0000751","alias":"chem_administration","owner":"MimsSediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"chloride":{"name":"chloride","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, parts per million"}},"description":"Concentration of chloride in the sample","title":"chloride","examples":[{"value":"5000 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000429","alias":"chloride","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"chlorophyll":{"name":"chlorophyll","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per cubic meter, microgram per liter"}},"description":"Concentration of chlorophyll","title":"chlorophyll","examples":[{"value":"5 milligram per cubic meter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000177","alias":"chlorophyll","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"density":{"name":"density","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"gram per cubic meter, gram per cubic centimeter"}},"description":"Density of the sample, which is its mass per unit volume (aka volumetric mass density)","title":"density","examples":[{"value":"1000 kilogram per cubic meter"}],"from_schema":"https://w3id.org/mixs","keywords":["density"],"slot_uri":"MIXS:0000435","alias":"density","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diether_lipids":{"name":"diether_lipids","annotations":{"Expected_value":{"tag":"Expected_value","value":"diether lipid name;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"nanogram per liter"}},"description":"Concentration of diether lipids; can include multiple types of diether lipids","title":"diether lipids","examples":[{"value":"0.2 nanogram per liter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000178","alias":"diether_lipids","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"diss_carb_dioxide":{"name":"diss_carb_dioxide","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter"}},"description":"Concentration of dissolved carbon dioxide in the sample or liquid portion of the sample","title":"dissolved carbon dioxide","examples":[{"value":"5 milligram per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","dissolved"],"slot_uri":"MIXS:0000436","alias":"diss_carb_dioxide","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_hydrogen":{"name":"diss_hydrogen","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of dissolved hydrogen","title":"dissolved hydrogen","examples":[{"value":"0.3 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["dissolved"],"slot_uri":"MIXS:0000179","alias":"diss_hydrogen","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_inorg_carb":{"name":"diss_inorg_carb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"microgram per liter, milligram per liter, parts per million"}},"description":"Dissolved inorganic carbon concentration in the sample, typically measured after filtering the sample using a 0.45 micrometer filter","title":"dissolved inorganic carbon","examples":[{"value":"2059 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","dissolved","inorganic"],"slot_uri":"MIXS:0000434","alias":"diss_inorg_carb","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_org_carb":{"name":"diss_org_carb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter"}},"description":"Concentration of dissolved organic carbon in the sample, liquid portion of the sample, or aqueous phase of the fluid","title":"dissolved organic carbon","examples":[{"value":"197 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","dissolved","organic"],"slot_uri":"MIXS:0000433","alias":"diss_org_carb","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_org_nitro":{"name":"diss_org_nitro","description":"Dissolved organic nitrogen concentration measured as; total dissolved nitrogen - NH4 - NO3 - NO2","title":"dissolved organic nitrogen","examples":[{"value":"0.05 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["dissolved","nitrogen","organic"],"slot_uri":"MIXS:0000162","alias":"diss_org_nitro","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_oxygen":{"name":"diss_oxygen","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per kilogram, milligram per liter"}},"description":"Concentration of dissolved oxygen","title":"dissolved oxygen","examples":[{"value":"175 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","keywords":["dissolved","oxygen"],"slot_uri":"MIXS:0000119","alias":"diss_oxygen","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"glucosidase_act":{"name":"glucosidase_act","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"mol per liter per hour"}},"description":"Measurement of glucosidase activity","title":"glucosidase activity","examples":[{"value":"5 mol per liter per hour"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000137","alias":"glucosidase_act","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"magnesium":{"name":"magnesium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"mole per liter, milligram per liter, parts per million, micromole per kilogram"}},"description":"Concentration of magnesium in the sample","title":"magnesium","examples":[{"value":"52.8 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000431","alias":"magnesium","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"mean_frict_vel":{"name":"mean_frict_vel","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter per second"}},"description":"Measurement of mean friction velocity","title":"mean friction velocity","examples":[{"value":"0.5 meter per second"}],"from_schema":"https://w3id.org/mixs","keywords":["mean","velocity"],"slot_uri":"MIXS:0000498","alias":"mean_frict_vel","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"mean_peak_frict_vel":{"name":"mean_peak_frict_vel","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter per second"}},"description":"Measurement of mean peak friction velocity","title":"mean peak friction velocity","examples":[{"value":"1 meter per second"}],"from_schema":"https://w3id.org/mixs","keywords":["mean","peak","velocity"],"slot_uri":"MIXS:0000502","alias":"mean_peak_frict_vel","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"methane":{"name":"methane","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, parts per billion, parts per million"}},"description":"Methane (gas) amount or concentration at the time of sampling","title":"methane","examples":[{"value":"0.15 micromole per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000101","alias":"methane","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"misc_param":{"name":"misc_param","annotations":{"Expected_value":{"tag":"Expected_value","value":"parameter name;measurement value"}},"description":"Any other measurement performed or parameter collected, that is not listed here","title":"miscellaneous parameter","examples":[{"value":"Bicarbonate ion concentration;2075 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","keywords":["parameter"],"string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000752","alias":"misc_param","owner":"MimsSediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"n_alkanes":{"name":"n_alkanes","annotations":{"Expected_value":{"tag":"Expected_value","value":"n-alkane name;measurement value"}},"description":"Concentration of n-alkanes; can include multiple n-alkanes","title":"n-alkanes","examples":[{"value":"n-hexadecane;100 milligram per liter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000503","alias":"n_alkanes","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"nitrate":{"name":"nitrate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of nitrate in the sample","title":"nitrate","examples":[{"value":"65 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrate"],"slot_uri":"MIXS:0000425","alias":"nitrate","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"nitrite":{"name":"nitrite","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of nitrite in the sample","title":"nitrite","examples":[{"value":"0.5 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrite"],"slot_uri":"MIXS:0000426","alias":"nitrite","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"nitro":{"name":"nitro","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of nitrogen (total)","title":"nitrogen","examples":[{"value":"4.2 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrogen"],"slot_uri":"MIXS:0000504","alias":"nitro","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"org_carb":{"name":"org_carb","description":"Concentration of organic carbon","title":"organic carbon","examples":[{"value":"1.5 microgram per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","organic"],"slot_uri":"MIXS:0000508","alias":"org_carb","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"org_matter":{"name":"org_matter","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"microgram per liter"}},"description":"Concentration of organic matter","title":"organic matter","examples":[{"value":"1.75 milligram per cubic meter"}],"from_schema":"https://w3id.org/mixs","keywords":["organic"],"slot_uri":"MIXS:0000204","alias":"org_matter","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"org_nitro":{"name":"org_nitro","description":"Concentration of organic nitrogen","title":"organic nitrogen","examples":[{"value":"4 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrogen","organic"],"slot_uri":"MIXS:0000205","alias":"org_nitro","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"organism_count":{"name":"organism_count","annotations":{"Expected_value":{"tag":"Expected_value","value":"organism name;measurement value;enumeration"}},"description":"Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)","title":"organism count","examples":[{"value":"total prokaryotes;3.5e7 cells per milliliter;qPCR"}],"from_schema":"https://w3id.org/mixs","keywords":["count","organism"],"string_serialization":"{text};{float} {unit};[ATP|MPN|qPCR|other]","slot_uri":"MIXS:0000103","alias":"organism_count","owner":"MimsSediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"oxy_stat_samp":{"name":"oxy_stat_samp","description":"Oxygenation status of sample","title":"oxygenation status of sample","examples":[{"value":"aerobic"}],"from_schema":"https://w3id.org/mixs","keywords":["oxygen","sample","status"],"slot_uri":"MIXS:0000753","alias":"oxy_stat_samp","owner":"MimsSediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"OXY_STAT_SAMP_ENUM"},"ph":{"name":"ph","description":"Ph measurement of the sample, or liquid portion of sample, or aqueous phase of the fluid","title":"pH","examples":[{"value":"7.2"}],"from_schema":"https://w3id.org/mixs","keywords":["ph"],"slot_uri":"MIXS:0001001","alias":"ph","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"float"},"particle_class":{"name":"particle_class","annotations":{"Expected_value":{"tag":"Expected_value","value":"particle name;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"micrometer"}},"description":"Particles are classified, based on their size, into six general categories:clay, silt, sand, gravel, cobbles, and boulders; should include amount of particle preceded by the name of the particle type; can include multiple values","title":"particle classification","from_schema":"https://w3id.org/mixs","keywords":["classification","particle"],"string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000206","alias":"particle_class","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"part_org_carb":{"name":"part_org_carb","description":"Concentration of particulate organic carbon","title":"particulate organic carbon","examples":[{"value":"1.92 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","organic","particle","particulate"],"slot_uri":"MIXS:0000515","alias":"part_org_carb","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"perturbation":{"name":"perturbation","annotations":{"Expected_value":{"tag":"Expected_value","value":"perturbation type name;perturbation interval and duration"}},"description":"Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types","title":"perturbation","examples":[{"value":"antibiotic addition;R2/2018-05-11T14:30Z/2018-05-11T19:30Z/P1H30M"}],"from_schema":"https://w3id.org/mixs","keywords":["perturbation"],"string_serialization":"{text};{Rn/start_time/end_time/duration}","slot_uri":"MIXS:0000754","alias":"perturbation","owner":"MimsSediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"petroleum_hydrocarb":{"name":"petroleum_hydrocarb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of petroleum hydrocarbon","title":"petroleum hydrocarbon","examples":[{"value":"0.05 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["hydrocarbon","petroleum"],"slot_uri":"MIXS:0000516","alias":"petroleum_hydrocarb","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"phaeopigments":{"name":"phaeopigments","annotations":{"Expected_value":{"tag":"Expected_value","value":"phaeopigment name;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per cubic meter"}},"description":"Concentration of phaeopigments; can include multiple phaeopigments","title":"phaeopigments","examples":[{"value":"2.5 milligram per cubic meter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000180","alias":"phaeopigments","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"phosphate":{"name":"phosphate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of phosphate","title":"phosphate","examples":[{"value":"0.7 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["phosphate"],"slot_uri":"MIXS:0000505","alias":"phosphate","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"phosplipid_fatt_acid":{"name":"phosplipid_fatt_acid","annotations":{"Expected_value":{"tag":"Expected_value","value":"phospholipid fatty acid name;measurement value"}},"description":"Concentration of phospholipid fatty acids; can include multiple values","title":"phospholipid fatty acid","examples":[{"value":"2.98 milligram per liter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000181","alias":"phosplipid_fatt_acid","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"porosity":{"name":"porosity","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement value or range"},"Preferred_unit":{"tag":"Preferred_unit","value":"percentage"}},"description":"Porosity of deposited sediment is volume of voids divided by the total volume of sample","title":"porosity","from_schema":"https://w3id.org/mixs","keywords":["porosity"],"string_serialization":"{float} - {float} {unit}","slot_uri":"MIXS:0000211","alias":"porosity","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string"},"potassium":{"name":"potassium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, parts per million"}},"description":"Concentration of potassium in the sample","title":"potassium","examples":[{"value":"463 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000430","alias":"potassium","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"pressure":{"name":"pressure","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"atmosphere"}},"description":"Pressure to which the sample is subject to, in atmospheres","title":"pressure","examples":[{"value":"50 atmosphere"}],"from_schema":"https://w3id.org/mixs","keywords":["pressure"],"slot_uri":"MIXS:0000412","alias":"pressure","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"redox_potential":{"name":"redox_potential","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"millivolt"}},"description":"Redox potential, measured relative to a hydrogen cell, indicating oxidation or reduction potential","title":"redox potential","examples":[{"value":"300 millivolt"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000182","alias":"redox_potential","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"salinity":{"name":"salinity","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"practical salinity unit, percentage"}},"description":"The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater","title":"salinity","examples":[{"value":"25 practical salinity unit"}],"from_schema":"https://w3id.org/mixs","keywords":["salinity"],"slot_uri":"MIXS:0000183","alias":"salinity","owner":"MimsSediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_store_dur":{"name":"samp_store_dur","description":"Duration for which the sample was stored. Indicate the duration for which the sample was stored written in ISO 8601 format","title":"sample storage duration","examples":[{"value":"P1Y6M"}],"from_schema":"https://w3id.org/mixs","keywords":["duration","period","sample","storage"],"slot_uri":"MIXS:0000116","alias":"samp_store_dur","owner":"MimsSediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^P(?:(?:\\\\d+D|\\\\d+M(?:\\\\d+D)?|\\\\d+Y(?:\\\\d+M(?:\\\\d+D)?)?)(?:T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S))?|T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S)|\\\\d+W)$","structured_pattern":{"syntax":"^{duration}$","interpolated":true,"partial_match":true}},"samp_store_loc":{"name":"samp_store_loc","annotations":{"Expected_value":{"tag":"Expected_value","value":"location name"}},"description":"Location at which sample was stored, usually name of a specific freezer/room","title":"sample storage location","examples":[{"value":"Freezer no:5"}],"from_schema":"https://w3id.org/mixs","keywords":["location","sample","storage"],"string_serialization":"{text}","slot_uri":"MIXS:0000755","alias":"samp_store_loc","owner":"MimsSediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"samp_store_temp":{"name":"samp_store_temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Temperature at which sample was stored, e.g. -80 degree Celsius","title":"sample storage temperature","examples":[{"value":"-80 degree Celsius"}],"from_schema":"https://w3id.org/mixs","keywords":["sample","storage","temperature"],"slot_uri":"MIXS:0000110","alias":"samp_store_temp","owner":"MimsSediment","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sediment_type":{"name":"sediment_type","description":"Information about the sediment type based on major constituents","title":"sediment type","examples":[{"value":"biogenous"}],"from_schema":"https://w3id.org/mixs","keywords":["sediment","type"],"slot_uri":"MIXS:0001078","alias":"sediment_type","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"SEDIMENT_TYPE_ENUM"},"silicate":{"name":"silicate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of silicate","title":"silicate","examples":[{"value":"0.05 micromole per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000184","alias":"silicate","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sodium":{"name":"sodium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, parts per million"}},"description":"Sodium concentration in the sample","title":"sodium","examples":[{"value":"10.5 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000428","alias":"sodium","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sulfate":{"name":"sulfate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of sulfate in the sample","title":"sulfate","examples":[{"value":"5 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["sulfate"],"slot_uri":"MIXS:0000423","alias":"sulfate","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sulfide":{"name":"sulfide","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of sulfide in the sample","title":"sulfide","examples":[{"value":"2 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["sulfide"],"slot_uri":"MIXS:0000424","alias":"sulfide","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tidal_stage":{"name":"tidal_stage","description":"Stage of tide","title":"tidal stage","examples":[{"value":"high tide"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000750","alias":"tidal_stage","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"TIDAL_STAGE_ENUM"},"tot_carb":{"name":"tot_carb","description":"Total carbon content","title":"total carbon","from_schema":"https://w3id.org/mixs","keywords":["carbon","total"],"slot_uri":"MIXS:0000525","alias":"tot_carb","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tot_depth_water_col":{"name":"tot_depth_water_col","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Measurement of total depth of water column","title":"total depth of water column","examples":[{"value":"500 meter"}],"from_schema":"https://w3id.org/mixs","keywords":["depth","total","water"],"slot_uri":"MIXS:0000634","alias":"tot_depth_water_col","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tot_nitro_content":{"name":"tot_nitro_content","description":"Total nitrogen content of the sample","title":"total nitrogen content","examples":[{"value":"35 milligrams Nitrogen per kilogram of soil"}],"from_schema":"https://w3id.org/mixs","keywords":["content","nitrogen","total"],"slot_uri":"MIXS:0000530","alias":"tot_nitro_content","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tot_org_carb":{"name":"tot_org_carb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"gram Carbon per kilogram sample material"}},"description":"Total organic carbon content","title":"total organic carbon","examples":[{"value":"0.02"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","organic","total"],"slot_uri":"MIXS:0000533","alias":"tot_org_carb","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"turbidity":{"name":"turbidity","description":"Measure of the amount of cloudiness or haziness in water caused by individual particles","title":"turbidity","examples":[{"value":"0.3 nephelometric turbidity units"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000191","alias":"turbidity","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"water_content":{"name":"water_content","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"gram per gram or cubic centimeter per cubic centimeter"}},"description":"Water content measurement","title":"water content","from_schema":"https://w3id.org/mixs","keywords":["content","water"],"string_serialization":"{float} {unit}","slot_uri":"MIXS:0000185","alias":"water_content","owner":"MimsSediment","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}}},"class_uri":"MIXS:0010007_0016011"},"Ancient":{"name":"Ancient","annotations":{"use_cases":{"tag":"use_cases","value":"ancient dna samples"}},"description":"A collection of terms appropriate when collecting samples and sequencing samples  of data containing ancient DNA, i.e., degraded molecules not from living organisms.","title":"ancient","comments":["This extension that is intended to be used in very specific cases but almost always in conjunction with other MIxS checklists and extensions. It does not therefore (currently) include additional common terms from checklists."],"from_schema":"https://w3id.org/mixs","is_a":"Extension","slot_usage":{"samp_age_range_youngest_limit":{"name":"samp_age_range_youngest_limit","recommended":true},"samp_age_range_oldest_limit":{"name":"samp_age_range_oldest_limit","recommended":true}},"attributes":{"collection_context_name":{"name":"collection_context_name","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of institution"}},"description":"Name of where sample originated and is typically stored. Typically will be \'owning institution\'.","title":"context name where location was collected","examples":[{"value":"Natural History Museum London"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["storage"],"slot_uri":"MIXS:999999901","alias":"collection_context_name","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"cultural_era":{"name":"cultural_era","annotations":{"Expected_value":{"tag":"Expected_value","value":"chronotology or PeriodO term; text"}},"description":"The cultural era approximating the period in which the individual lived from https://chronontology.dainst.org/ or PeriodO if possible. If not available, follow INSDC standard for reporting missing data (https://www.insdc.org/submitting-standards/missing-value-reporting/).","title":"cultural era or period","examples":[{"value":"Copper Age [Chronotology: NW6hofAScJSE]"},{"value":"Upper Middle Palaeolthic"},{"value":"Not collected"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient","age"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:999999902","alias":"cultural_era","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":true,"recommended":true,"multivalued":false},"damage_treatment":{"name":"damage_treatment","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Indication of whether characteristic ancient DNA damage has been  altered or removed from a DNA extract in a laboratory","title":"Damage treatment type","examples":[{"value":"none"},{"value":"partial-udg"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"slot_uri":"MIXS:999999903","alias":"damage_treatment","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"DamageTreatmentEnum","required":true,"multivalued":true},"dna_extraction_date":{"name":"dna_extraction_date","description":"The date when the nucleic acid extraction was started from the sample material. In case no exact time is available, the date can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant","title":"date of extraction of nucleic acids from sample","examples":[{"value":"2023-12-01"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999904","alias":"dna_extraction_date","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"datetime","required":false,"recommended":true},"ethical_authority":{"name":"ethical_authority","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of authority providing ethical approval"}},"description":"Name of the authorit(ies) or institution(s) that awarded sampling and  analysis (e.g. human remains) and/or export permission  (e.g. animal remains)","title":"ethical authority","examples":[{"value":"University of Copenhagen"},{"value":"Federal Foreign Office (Germany)"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["ethics","location"],"slot_uri":"MIXS:999999905","alias":"ethical_authority","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"ethical_date":{"name":"ethical_date","description":"Date of award of ethical/export permission. The date can be right truncated  i.e. all of these are valid times: 2008-01-23; 2008-01; 2008;  Except: 2008-01; 2008 all are ISO8601 compliant","title":"date of ethical approval","examples":[{"value":"2023-12-01"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999906","alias":"ethical_date","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"datetime","required":false,"recommended":true},"ethical_id":{"name":"ethical_id","description":"The permissions code, ID, document, or similar provided by any authority  (socal, legal, academic etc.) associated with approval of the analysis of  this particular sample.","title":"ethical permit or approval ID","examples":[{"value":"DE-123-JK"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"slot_uri":"MIXS:999999906","alias":"ethical_id","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"experimental_procedures":{"name":"experimental_procedures","description":"Provide a DOI or URL to refer to the paper where the field report, nucleic acid extraction,  library construction, and other procedures are explained in more detail, e.g. the paper reporting the data.","title":"experimental procedures","examples":[{"value":"doi:10.1093/nar/gkr771"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999907","alias":"experimental_procedures","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_concentration":{"name":"lib_concentration","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement value in number of ng per l"}},"description":"Concentration of library in ng per l, as inferred by qPCR, digital PCR, Qubit etc.","title":"library concentration","examples":[{"value":"2"},{"value":"0.5"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{float}","slot_uri":"MIXS:999999908","alias":"lib_concentration","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"lib_index_polymerase":{"name":"lib_index_polymerase","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of polymerase used during library indexing"}},"description":"The DNA polymerase enzyme used for indexing DNA libraries","title":"library index polymerase","examples":[{"value":"Agilent PfuTurbo Cx HotStart"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["polymerase","library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999909","alias":"lib_index_polymerase","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true},"lib_preparation_protocol":{"name":"lib_preparation_protocol","description":"Citation(s) for the DNA library preparation protocol","title":"library preparation protocols","examples":[{"value":"doi:10.1093/nar/gkr771"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999907","alias":"lib_preparation_protocol","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_reamplification_polymerase":{"name":"lib_reamplification_polymerase","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of polymerase used during library indexing"}},"description":"The name of the DNA polymerase enzyme used for reamplifying DNA libraries post-indexing.","title":"library reamplification polymerase","examples":[{"value":"KAPA HiFi HotStart Uracil+"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["polymerase","library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999909","alias":"lib_reamplification_polymerase","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true},"lib_type":{"name":"lib_type","description":"The type of library created, i.e., amplicon, enriched, or shotgun.  Amplicon based is a library that has been amplified to target a specific single region of a genome (e.g. a specific gene).  An enriched library is a library has had a particular genome or genome regions \'captured\' or enriched typically via baits or probes. A shotgun library has had no type of targeted enrichment for a particular gene or genome.","title":"library type","examples":[{"value":"amplicon"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"slot_uri":"MIXS:999999910","alias":"lib_type","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"LibTypeEnum","required":false},"neg_cont_status":{"name":"neg_cont_status","description":"Specify whether the sample is a negative control or not.","title":"negative control status","examples":[{"value":"true"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["control"],"slot_uri":"MIXS:999999911","alias":"neg_cont_status","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"boolean"},"num_capture_amp_cycles":{"name":"num_capture_amp_cycles","description":"Number of amplification cycles after capture enrichment","title":"number of reamplification cycles after capture","examples":[{"value":"12"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999912","alias":"num_capture_amp_cycles","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer","multivalued":true},"num_reamp_cycles":{"name":"num_reamp_cycles","description":"Number of amplification cycles after library indexing PCR. If capture data, this refers to amplifications prior to the capture experiments.","title":"number of reamplification cycles","examples":[{"value":"12"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999913","alias":"num_reamp_cycles","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer"},"preservational_treatment":{"name":"preservational_treatment","description":"Description of any treatment applied to samples for the purpose of  maximising collection preservation that may influence downstream  DNA recovery or library construction, such as storage fluid or  reconstructive glue","title":"preservational treatment","examples":[{"value":"stored in formalin"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999914","alias":"preservational_treatment","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"prev_pubs":{"name":"prev_pubs","description":"Any publications that report data from the same body/skeleton/individual","title":"previous publications","examples":[{"value":"doi:10.1016/j.jas.2015.02.0181"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999915","alias":"prev_pubs","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"recovery_date":{"name":"recovery_date","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"year"}},"description":"Date of excavation or retrieval from burial or depositional context, if known. In case no exact time is available, the date can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant. If excavations were done during a longer period, report its midpoint at a month or year level.","title":"date of retrieval from depositional context","examples":[{"value":"2023"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999916","alias":"recovery_date","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"datetime","required":false,"recommended":true},"samp_decontam_pretreat":{"name":"samp_decontam_pretreat","description":"Method(s) employed for surface decontamination of samples of external  modern DNA; Treatment used on the samples. Depends on the sample type.  More relevant for bones than environmental samples. E.g. buffers, EDTA, etc.","title":"sample decontamination pretreatment","examples":[{"value":"doi:10.1016/j.jas.2015.02.0181"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999917","alias":"samp_decontam_pretreat","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"samp_age_range_youngest_limit":{"name":"samp_age_range_youngest_limit","annotations":{"Expected_value":{"tag":"Expected_value","value":"age value corresponding to unit"}},"description":"The youngest possible age of the sample. Youngest refers to the time closest to present day. If a sample has only a single age value, the value from samp_age_range_oldest_limit can be repeated. Sample age refers to a period of time in the past, such as when an organism was living and then died, not the biological age.","title":"sample age range youngest limit","examples":[{"value":"100000"},{"value":"1700"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999918","alias":"samp_age_range_youngest_limit","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer","required":true,"recommended":true},"samp_age_range_youngest_unit":{"name":"samp_age_range_youngest_unit","description":"The unit of the youngest age in the sample age range.","title":"sample age range youngest unit","examples":[{"value":"cal BP"},{"value":"CE"},{"value":"Ma"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999919","alias":"samp_age_range_youngest_unit","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"SampAgeUnit","required":true,"recommended":true,"multivalued":true},"samp_age_range_oldest_limit":{"name":"samp_age_range_oldest_limit","annotations":{"Expected_value":{"tag":"Expected_value","value":"age value corresponding to unit"}},"description":"The oldest possible age of the sample. Oldest refers to the time furthest from present day. If a sample has only a single age value, the value from samp_age_range_youngest_limit can be repeated. Sample age refers to a period of time in the past, such as when an organism was living and then died, not the biological age.","title":"sample age range oldest limit","examples":[{"value":"120000"},{"value":"1900"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999920","alias":"samp_age_range_oldest_limit","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer","required":true,"recommended":true},"samp_age_range_oldest_unit":{"name":"samp_age_range_oldest_unit","description":"The unit of the oldest age in sample age range.","title":"sample age range oldest unit","examples":[{"value":"cal BP"},{"value":"CE"},{"value":"Ma"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999921","alias":"samp_age_range_oldest_unit","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"SampAgeUnit","required":true,"recommended":true,"multivalued":true},"samp_age_range_inference_methods":{"name":"samp_age_range_inference_methods","description":"The method used to infer the sample age. This corresponds to a fixed list of most commonly used dating methods, such as 14C, OSL, stratigraphic inference, historical records, other etc. If the method is not in the list, please provide a description in samp_age_range_inference_description.","title":"sample age inference methods","examples":[{"value":"radiocarbon dating, calibrated with OxCal v4.3"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999922","alias":"samp_age_range_inference_methods","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"SampAgeInferMethod","required":true,"recommended":true,"multivalued":true},"samp_age_range_inference_description":{"name":"samp_age_range_inference_description","description":"Additional general information about the inference method used to infer the sample age that can be useful for  understanding how the sample age was estimated. This can be things such as whether radiocarbon age estimated on AMS or conventional radiocarbon dating, or which calibration confidence interval was used.  It can also be used to describe what artefacts were used to age the stratigraphic layer of historical contexts.","title":"sample age inference description","examples":[{"value":"radiocarbon dating, calibrated with OxCal v4.3 with 95% confidence interval"},{"value":"based on proxy dating from other bone samples of stratigraphic layer"},{"value":"a coin found in the burial was from the 3rd century"},{"value":"age taken from previous publication Doe et al. 2019"},{"value":"radiocarbon age ID: OxA-12345"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999923","alias":"samp_age_range_inference_description","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":false},"samp_alt_lab_ids":{"name":"samp_alt_lab_ids","description":"An alternative sample or material ID related to the sample not already covered by terms samp_name and source_mat_id, including from associated other non-genetic analyses of the same sample (e.g. museum IDs, internal lab sample ID from sampling such a bone drilling). Can be specified multiple times for different ID contexts.","title":"alternative sample IDs","examples":[{"value":"ABC_24"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:999999924","alias":"samp_alt_lab_ids","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"storage_conditions":{"name":"storage_conditions","description":"General conditions in which the material (before sampling for DNA analysis) was stored in long-term collection storage, that may influenced DNA recovery or library construction. For example, specify temperature, humidity, presence of microbial overgrowth etc..","title":"storage conditions","examples":[{"value":"climate-controlled"},{"value":"Stored in the museum from recovery_date to collection_date"},{"value":"Stored at -20oC until dna_extraction_date"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:999999925","alias":"storage_conditions","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string"},"palaeopath_status":{"name":"palaeopath_status","annotations":{"Expected_value":{"tag":"Expected_value","value":"description of health related observation on ancient remains"}},"description":"Describe briefly any relevant palaeopathological or health-related observations of the remains of the individual under study.","title":"palaeopathology status.","examples":[{"value":"Osteoporosis."},{"value":"Parasites found in pelvic area."},{"value":"Caries on right upper molar."}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["palaeopathology","host health","ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999926","alias":"palaeopath_status","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":false},"biocultural_label":{"name":"biocultural_label","annotations":{"Expected_value":{"tag":"Expected_value","value":"Relevant biocultural label from https://localcontexts.org/labels/biocultural-labels/"}},"description":"Relevant biocultural labels that apply to the sample under study, based on https://localcontexts.org/label/bc-provenance/. These help indicate how indigenous peoples and similar communities view in which contexts this data can be reused.","title":"biocultural label","examples":[{"value":"BC R"},{"value":"BC MC"}],"in_subset":["ethics"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"slot_uri":"MIXS:999999927","alias":"biocultural_label","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"BioCulturalLabel","required":false,"recommended":false,"multivalued":true},"ethics_perm_scope":{"name":"ethics_perm_scope","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the scope of ethical permissions for data use"}},"description":"Description of the original scope and permissions of the research on the genetic material, as was approved by the ethical authority (e.g. bacteria only, DNA only, bacteria and human, no host read analysis allowed).","title":"ethical Permissions & Scope","examples":[{"value":"Defined scope only includes the study of bacterial sequences and any human sequence is not covered under the agreement."}],"in_subset":["ethics"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"string_serialization":"{text}","slot_uri":"MIXS:999999928","alias":"ethics_perm_scope","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"host_body_preservation_state":{"name":"host_body_preservation_state","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the preservation of the sampled (ancient) organism/host"}},"description":"Description of the state of the sampled (ancient) organism/host as originally preserved in the burial environment, e.g. if the the sample was artificially or naturally (fully/partially) mummified at the time of death.","title":"ancient body preservation state","examples":[{"value":"complete artificial mummification."},{"value":"natural partial mummification."},{"value":"fully skeletonised."}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999929","alias":"host_body_preservation_state","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":false},"lib_strandedness":{"name":"lib_strandedness","annotations":{"Expected_value":{"tag":"Expected_value","value":"DNA library strandedness"}},"description":"The strandedness of the original template DNA molecules used for constructing the sequencing library","title":"DNA strandedness in library creation","examples":[{"value":"single"},{"value":"double"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"slot_uri":"MIXS:999999930","alias":"lib_strandedness","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"LibStrand","required":false,"recommended":true,"multivalued":true},"enrichment_probe_set":{"name":"enrichment_probe_set","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of probe set used for target enrichment/library selection"}},"description":"Description of target enrichment probe designs used (e.g., species included, sequences, type, company). This can include custom kits (please provide a general description and DOI if available) or commercially available kit (provide ID and company).","title":"enrichment probe design description","examples":[{"value":"Custom probe design (70 bp biotinylated RNA probes) covering 500 full E. Coli genomes; company TE-Science"},{"value":"Commercial kit ABC001-EC from company TE-Science, purchased 2020"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","enrichment"],"string_serialization":"{text}","slot_uri":"MIXS:999999931","alias":"enrichment_probe_set","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":true},"mid_config":{"name":"mid_config","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the indexing configuration of the library"}},"description":"Index/barcode/primer configuration used during library building for sequencing. This includes information such as the number, type and location of indexes, the index/primer kit/list, or if \'inline\' barcodes or UMIs were ligated directly onto the template molecules.","title":"multiplex identifiers or indexing configuration of libraries","examples":[{"value":"dual index with single internal barcode."},{"value":"UDI index sequences."}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999932","alias":"mid_config","owner":"Ancient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true}},"class_uri":"MIXS:9999903"},"RadiocarbonDating":{"name":"RadiocarbonDating","annotations":{"use_cases":{"tag":"use_cases","value":"RadiocarbonDating"}},"description":"A collection of terms describing radiocarbon dating measurements and calibration, including the sample material, pretreatment method, and calibration curve used. These date values and metadata are typically associated with a the sample sample material used for nucleic acis extraction","title":"Radiocarbon dating","comments":["Typically will be used with ancient DNA samples"],"from_schema":"https://w3id.org/mixs","is_a":"Extension","attributes":{"c14_date":{"name":"c14_date","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"years before present (yr BP)"}},"description":"The uncalibrated date from the laboratory measurement. Should be a conventional 14C date (i.e., 14C yr BP) NOT in AD/BC format.  This is typically the \'raw\' date reported by the radiocarbon lab, in Before Present (BP) notation.","title":"Radiocarbon date","examples":[{"value":"1453"},{"value":"12560"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999971","alias":"c14_date","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"integer","required":true,"recommended":true,"multivalued":false},"c14_date_sd":{"name":"c14_date_sd","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"years before present (yr BP)"}},"description":"The one standard deviation around the \'raw\' C14 Date measurement, normally indicated as a  after the main date. Must be in the same format (i.e., y BP).","title":"Radiocarbon date standard deviation","examples":[{"value":"120"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999972","alias":"c14_date_sd","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"integer","required":true,"recommended":true,"multivalued":false},"c14_date_lab_code":{"name":"c14_date_lab_code","annotations":{"Expected_value":{"tag":"Expected_value","value":"unique identifier"}},"description":"Unique laboratory identification number which is a combination of a lab identifier prefix, and a number. The prefix should be derived from: https://radiocarbon.webhost.uits.arizona.edu/laboratories.","title":"Laboratory code for the date determination","examples":[{"value":"OxA-1539"},{"value":"KN-1369"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{text}-{integer}","slot_uri":"MIXS:99999973","alias":"c14_date_lab_code","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"string","required":true,"recommended":true,"multivalued":false},"sample_material":{"name":"sample_material","annotations":{"Expected_value":{"tag":"Expected_value","value":"text"}},"description":"Material of the sample used to extract carbon used for radiocarbon dating measurements. Use ontology terms where possible, e.g. from UBERON for anatomical parts, or ENVO for other organic samples.","title":"Radiocarbon dating sample material","examples":[{"value":"charcoal [ENVO_01000560]"},{"value":"molar tooth [UBERON_0003655]"},{"value":"wood fragment"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:99999974","alias":"sample_material","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"string","required":false,"recommended":true,"multivalued":false},"delta_13_c":{"name":"delta_13_c","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"per mil ()"}},"description":"The delta carbon-13 value of the sample (13C), which is the ratio of the stable isotope 13C to 12C, expressed in per mil () notation. Used as a quality control measurement.","title":"Delta 13C value","examples":[{"value":"-25.3"},{"value":"-20.1"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{float}","slot_uri":"MIXS:99999976","alias":"delta_13_c","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"string","required":true,"recommended":true,"multivalued":false},"delta_13_c_method":{"name":"delta_13_c_method","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Which spectrophotometry method was used to measure the delta carbon-13 value, either with  Isotope Ratio Mass Spectrometer (IRMS) or Accelerated Mass Spectrometer (AMS).","title":"Delta 13C measurement method","examples":[{"value":"ams"},{"value":"imrs"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{text}","slot_uri":"MIXS:99999977","alias":"delta_13_c_method","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"MsMethod","required":false,"recommended":true},"carbon_perc":{"name":"carbon_perc","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"percent (%)"}},"description":"The percentage of carbon in a non-proteinaceous sample used for dating (such as charcoal), expressed as a percentage (%). Used as a quality control measurement.","title":"Carbon percentage","examples":[{"value":"50.3"},{"value":"45.1"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{float} {unit}","slot_uri":"MIXS:99999978","alias":"carbon_perc","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"float","required":true,"recommended":true,"multivalued":false},"carb_nitro_ratio":{"name":"carb_nitro_ratio","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement value"}},"description":"Ratio of amount or concentrations of carbon to nitrogen. Used for quality control value in proteinaceous samples for radiocarbon dating.","title":"carbon/nitrogen ratio","examples":[{"value":"0.417361111"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","nitrogen","ratio"],"string_serialization":"{float}:{float}","slot_uri":"MIXS:0000310","alias":"carb_nitro_ratio","owner":"RadiocarbonDating","domain_of":["Sediment","RadiocarbonDating","MimsSediment"],"range":"float","required":false,"recommended":true,"multivalued":false},"calib_age_youngest":{"name":"calib_age_youngest","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"calibrated years before present (cal BP)"}},"description":"Upper (youngest) date in the range of a calibrated age. Calendar timescale to be specified: AD/BC, CE/BCE, cal BP. The abbreviation BP is to be used for uncalibrated 14C determinations, cal BP must be used here.","title":"Radiocarbon calibration age upper limit","examples":[{"value":"1449"},{"value":"12560"},{"value":"1500"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999979","alias":"calib_age_youngest","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"integer","required":false,"recommended":true,"multivalued":true},"calib_age_median":{"name":"calib_age_median","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"calibrated years before present (cal BP)"}},"description":"Median date in the range of a calibrated age. Calendar timescale to be specified: AD/BC, CE/BCE, cal BP. The abbreviation BP is to be used for uncalibrated 14C determinations, cal BP must be used here.","title":"Radiocarbon calibration age upper limit","examples":[{"value":"1450"},{"value":"15700"},{"value":"1473"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999979","alias":"calib_age_median","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"integer","required":false,"recommended":true,"multivalued":true},"calib_age_oldest":{"name":"calib_age_oldest","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"calibrated years before present (cal BP)"}},"description":"Lower (oldest) date in the range of a calibrated age. Calendar timescale to be specified: AD/BC, CE/BCE, cal BP. Note: the abbreviation \'BP\' is to be used for uncalibrated 14C determinations, cal BP must be used here.","title":"Radiocarbon calibration age lower limit","examples":[{"value":"1451"},{"value":"18560"},{"value":"1453"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999980","alias":"calib_age_oldest","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"integer","required":false,"recommended":true,"multivalued":true},"calib_age_sigma":{"name":"calib_age_sigma","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Confidence level of the reported calibrated age range, typically in 2 (95.4%) or 1 (68.2%) levels of significance (sigma).","title":"Radiocarbon calibration age confidence level","examples":[{"value":"1"},{"value":"2"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999982","alias":"calib_age_sigma","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"ConfidenceSigma","required":false,"recommended":true,"multivalued":true},"calib_age_unit":{"name":"calib_age_unit","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Unit of the calibrated median date and calibrated range, i.e., whether in  cal AD/BC, cal CE/BCE, cal BP. Note the abbreviation BP is to be used for uncalibrated 14C determinations only, therefore cal BP must be used here.","title":"Unit of radiocarbon date calibration","examples":[{"value":"cal BP"},{"value":"cal AD"},{"value":"cal BCE"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{text}","slot_uri":"MIXS:99999981","alias":"calib_age_unit","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"RadiocarbonCalibUnit","required":false,"recommended":true,"multivalued":true},"calib_curve":{"name":"calib_curve","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Name of calibration curve used during radiocarbon date calibration. If a Marine curve is selected, the localised reservoir offset should be specified.","title":"Radiocarbon calibration curve name","examples":[{"value":"IntCal20"},{"value":"SHCal20"},{"value":"Bomb21NH3"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{text}","slot_uri":"MIXS:99999983","alias":"calib_curve","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"CalibCurve","required":false,"recommended":true,"multivalued":true},"calib_software":{"name":"calib_software","annotations":{"Expected_value":{"tag":"Expected_value","value":"text"}},"description":"The name of the software used to generated the calibrated version of the laboratory measurement date.","title":"Radiocarbon calibration software","examples":[{"value":"OxCal"},{"value":"CALIB"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{text}","slot_uri":"MIXS:99999984","alias":"calib_software","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"string","required":false,"recommended":true,"multivalued":true},"calib_version":{"name":"calib_version","annotations":{"Expected_value":{"tag":"Expected_value","value":"version"}},"description":"Version of the calibration software used for calibration.","title":"Radiocarbon calibration software version","examples":[{"value":"1.0.3"},{"value":"4.4"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{text}","slot_uri":"MIXS:99999985","alias":"calib_version","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"string","required":false,"recommended":true,"multivalued":true},"calib_settings":{"name":"calib_settings","annotations":{"Expected_value":{"tag":"Expected_value","value":"text"}},"description":"Settings used with the calibration software, including citations to age models used (when applicable).","title":"Radiocarbon calibration software settings","examples":[{"value":"Bayesian model"},{"value":"-m 10"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{text}","slot_uri":"MIXS:99999986","alias":"calib_settings","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"string","required":false,"recommended":true,"multivalued":true},"calib_citation":{"name":"calib_citation","annotations":{"Expected_value":{"tag":"Expected_value","value":"text"}},"description":"The DOI, URL or citation information of the publication where the calibration software was originally described.","title":"Radiocarbon calibration software citation","examples":[{"value":"doi:10.1017/S0033822200033865"},{"value":"Ramsey 2009, Radiocarbon"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"slot_uri":"MIXS:99999987","alias":"calib_citation","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"string","required":false,"recommended":true,"multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$|([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}|{text}$"}},"localised_reservoir_offset_value":{"name":"localised_reservoir_offset_value","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"years before present (yr BP)"}},"description":"The Delta R (R) value used modify the calibration curve to account for regional/localised marine reservoir effect.","title":"Reservoir offset","examples":[{"value":"49 yr BP"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999988","alias":"localised_reservoir_offset_value","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"integer","required":false,"recommended":false,"multivalued":true},"localised_reservoir_offset_sd":{"name":"localised_reservoir_offset_sd","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement"},"Preferred_unit":{"tag":"Preferred_unit","value":"years before present (yr BP)"}},"description":"The standard deviation of averaged Delta R (R) values used modify the calibration curve to account for regional/localised marine reservoir effect.","title":"Reservoir offset standard deviation","examples":[{"value":"3 yr BP"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dating"],"string_serialization":"{integer}","slot_uri":"MIXS:99999989","alias":"localised_reservoir_offset_sd","owner":"RadiocarbonDating","domain_of":["RadiocarbonDating"],"range":"integer","required":false,"recommended":false,"multivalued":true}},"class_uri":"MIXS:9999999"},"MimsHostAssociatedAncient":{"name":"MimsHostAssociatedAncient","description":"MIxS Data that comply with the Mims checklist and Host Associated, Ancient","title":"MimsHostAssociated combined with Ancient","in_subset":["combination_classes"],"from_schema":"https://w3id.org/mixs","is_a":"Ancient","mixins":["MimsHostAssociated"],"attributes":{"samp_name":{"name":"samp_name","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":""}},"description":"A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name","title":"sample name","examples":[{"value":"ISDsoil1"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["sample"],"slot_uri":"MIXS:0001107","alias":"samp_name","owner":"MimsHostAssociatedAncient","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"size_frac":{"name":"size_frac","annotations":{"Expected_value":{"tag":"Expected_value","value":"filter size value range"}},"description":"Filtering pore size used in sample preparation","title":"size fraction selected","examples":[{"value":"0-0.22 micrometer"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["fraction","size"],"string_serialization":"{float}-{float} {unit}","slot_uri":"MIXS:0000017","alias":"size_frac","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"lib_screen":{"name":"lib_screen","annotations":{"Expected_value":{"tag":"Expected_value","value":"screening strategy name"}},"description":"Specific enrichment or screening methods applied before and/or after creating libraries","title":"library screening strategy","examples":[{"value":"enriched, screened, normalized"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"string_serialization":"{text}","slot_uri":"MIXS:0000043","alias":"lib_screen","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"ref_db":{"name":"ref_db","annotations":{"Expected_value":{"tag":"Expected_value","value":"names, versions, and references of databases"}},"description":"List of database(s) used for ORF annotation, along with version number and reference to website or publication","title":"reference database(s)","examples":[{"value":"pVOGs;5;http://dmk-brain.ecn.uiowa.edu/pVOGs/ Grazziotin et al. 2017 doi:10.1093/nar/gkw975"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["database"],"string_serialization":"{database};{version};{reference}","slot_uri":"MIXS:0000062","alias":"ref_db","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"nucl_acid_amp":{"name":"nucl_acid_amp","description":"A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids","title":"nucleic acid amplification","examples":[{"value":"https://phylogenomics.me/protocols/16s-pcr-protocol/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000038","alias":"nucl_acid_amp","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_size":{"name":"lib_size","description":"Total number of clones in the library prepared for the project","title":"library size","examples":[{"value":"50"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","size"],"slot_uri":"MIXS:0000039","alias":"lib_size","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer","recommended":true},"mid":{"name":"mid","description":"Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to specifically tag unique samples in a sequencing run. Sequence should be reported in uppercase letters","title":"multiplex identifiers","examples":[{"value":"GTGAATAT"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["identifier"],"slot_uri":"MIXS:0000047","alias":"mid","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[ACGTRKSYMWBHDVN]+$","structured_pattern":{"syntax":"^{ambiguous_nucleotides}$","interpolated":true,"partial_match":true}},"assembly_name":{"name":"assembly_name","annotations":{"Expected_value":{"tag":"Expected_value","value":"name and version of assembly"}},"description":"Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community","title":"assembly name","examples":[{"value":"HuRef, JCVI_ISG_i3_1.0"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text} {text}","slot_uri":"MIXS:0000057","alias":"assembly_name","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"temp":{"name":"temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Temperature of the sample at the time of sampling","title":"temperature","examples":[{"value":"25 degree Celsius"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["temperature"],"slot_uri":"MIXS:0000113","alias":"temp","owner":"MimsHostAssociatedAncient","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"nucl_acid_ext":{"name":"nucl_acid_ext","description":"A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample","title":"nucleic acid extraction","examples":[{"value":"https://mobio.com/media/wysiwyg/pdfs/protocols/12888.pdf"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000037","alias":"nucl_acid_ext","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"samp_size":{"name":"samp_size","description":"The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected","title":"amount or size of sample collected","examples":[{"value":"5 liter"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["sample","size"],"slot_uri":"MIXS:0000001","alias":"samp_size","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"alt":{"name":"alt","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth\'s surface. In this context, the altitude measurement is the vertical distance between the earth\'s surface above sea level and the sampled position in the air","title":"altitude","examples":[{"value":"100 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000094","alias":"alt","owner":"MimsHostAssociatedAncient","domain_of":["Mims","HostAssociated","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"source_mat_id":{"name":"source_mat_id","annotations":{"Expected_value":{"tag":"Expected_value","value":"for cultures of microorganisms: identifiers for two culture collections; for other material a unique arbitrary identifer"}},"description":"A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain \'UAM:Herps:14\' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2)","title":"source material identifiers","examples":[{"value":"MPI012345"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["identifier","material","source"],"string_serialization":"{text}","slot_uri":"MIXS:0000026","alias":"source_mat_id","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true},"samp_vol_we_dna_ext":{"name":"samp_vol_we_dna_ext","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milliliter, gram, milligram, square centimeter"}},"description":"Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001)","title":"sample volume or weight for DNA extraction","examples":[{"value":"1500 milliliter"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["dna","sample","volume","weight"],"slot_uri":"MIXS:0000111","alias":"samp_vol_we_dna_ext","owner":"MimsHostAssociatedAncient","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"lib_reads_seqd":{"name":"lib_reads_seqd","description":"Total number of clones sequenced from the library","title":"library reads sequenced","examples":[{"value":"20"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"slot_uri":"MIXS:0000040","alias":"lib_reads_seqd","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer","recommended":true},"rel_to_oxygen":{"name":"rel_to_oxygen","description":"Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments","title":"relationship to oxygen","examples":[{"value":"aerobe"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["oxygen","relationship"],"slot_uri":"MIXS:0000015","alias":"rel_to_oxygen","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"REL_TO_OXYGEN_ENUM"},"samp_collect_device":{"name":"samp_collect_device","annotations":{"Expected_value":{"tag":"Expected_value","value":"device name"}},"description":"The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094)","title":"sample collection device","examples":[{"value":"swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713]"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["device","sample"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000002","alias":"samp_collect_device","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"number_contig":{"name":"number_contig","description":"Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG","title":"number of contigs","examples":[{"value":"40"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["number"],"slot_uri":"MIXS:0000060","alias":"number_contig","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer","recommended":true},"lib_layout":{"name":"lib_layout","description":"Specify whether to expect single, paired, or other configuration of reads","title":"library layout","examples":[{"value":"paired"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"slot_uri":"MIXS:0000041","alias":"lib_layout","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"LIB_LAYOUT_ENUM","recommended":true},"assembly_qual":{"name":"assembly_qual","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling   90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling < 90% of the expected genome or replicon sequence, or for which no genome size could be estimated","title":"assembly quality","examples":[{"value":"High-quality draft genome"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["quality"],"string_serialization":"[Finished genome|High-quality draft genome|Medium-quality draft genome|Low-quality draft genome|Genome fragment(s)]","slot_uri":"MIXS:0000056","alias":"assembly_qual","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"ref_biomaterial":{"name":"ref_biomaterial","description":"Primary publication if isolated before genome publication; otherwise, primary genome report","title":"reference for biomaterial","examples":[{"value":"doi:10.1016/j.syapm.2018.01.009"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000025","alias":"ref_biomaterial","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"project_name":{"name":"project_name","description":"Name of the project within which the sequencing was organized","title":"project name","examples":[{"value":"Forest soil metagenome"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["project"],"slot_uri":"MIXS:0000092","alias":"project_name","owner":"MimsHostAssociatedAncient","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"lib_vector":{"name":"lib_vector","annotations":{"Expected_value":{"tag":"Expected_value","value":"vector"}},"description":"Cloning vector type(s) used in construction of libraries","title":"library vector","examples":[{"value":"Bacteriophage P1"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"string_serialization":"{text}","slot_uri":"MIXS:0000042","alias":"lib_vector","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"adapters":{"name":"adapters","annotations":{"Expected_value":{"tag":"Expected_value","value":"adapter A and B sequence"}},"description":"Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters","title":"adapters","examples":[{"value":"AATGATACGGCGACCACCGAGATCTACACGCT;CAAGCAGAAGACGGCATACGAGAT"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{dna};{dna}","slot_uri":"MIXS:0000048","alias":"adapters","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"neg_cont_type":{"name":"neg_cont_type","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration or text"}},"description":"The substance or equipment used as a negative control in an investigation","title":"negative control type","in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["type"],"slot_uri":"MIXS:0001321","alias":"neg_cont_type","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"NEG_CONT_TYPE_ENUM","recommended":true},"assembly_software":{"name":"assembly_software","description":"Tool(s) used for assembly, including version number and parameters","title":"assembly software","examples":[{"value":"metaSPAdes;3.11.0;kmer set 21,33,55,77,99,121, default parameters otherwise"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["software"],"slot_uri":"MIXS:0000058","alias":"assembly_software","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"annot":{"name":"annot","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of tool or pipeline used, or annotation source description"}},"description":"Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter","title":"annotation","examples":[{"value":"prokka"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:0000059","alias":"annot","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"pos_cont_type":{"name":"pos_cont_type","description":"The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive","title":"positive control type","in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["type"],"string_serialization":"{term} or {text}","slot_uri":"MIXS:0001322","alias":"pos_cont_type","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"feat_pred":{"name":"feat_pred","description":"Method used to predict UViGs features such as ORFs, integration site, etc","title":"feature prediction","examples":[{"value":"Prodigal;2.6.3;default parameters"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["feature","predict"],"slot_uri":"MIXS:0000061","alias":"feat_pred","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"env_local_scale":{"name":"env_local_scale","annotations":{"Expected_value":{"tag":"Expected_value","value":"Environmental entities having causal influences upon the entity at time of sampling"}},"description":"Report the entity or entities which are in the sample or specimen s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS","title":"local environmental context","examples":[{"value":"hillside [ENVO:01000333]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["context","environmental"],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000013","alias":"env_local_scale","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"samp_mat_process":{"name":"samp_mat_process","description":"A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed","title":"sample material processing","examples":[{"value":"filtering of seawater, storing samples in ethanol"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["material","process","sample"],"slot_uri":"MIXS:0000016","alias":"samp_mat_process","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"sim_search_meth":{"name":"sim_search_meth","description":"Tool used to compare ORFs with database, along with version and cutoffs used","title":"similarity search method","examples":[{"value":"HMMER3;3.1b2;hmmsearch, cutoff of 50 on score"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["method"],"slot_uri":"MIXS:0000063","alias":"sim_search_meth","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"depth":{"name":"depth","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"The vertical distance below local surface. For sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples","title":"depth","examples":[{"value":"10 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["depth"],"slot_uri":"MIXS:0000018","alias":"depth","owner":"MimsHostAssociatedAncient","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_collect_method":{"name":"samp_collect_method","description":"The method employed for collecting the sample","title":"sample collection method","examples":[{"value":"swabbing"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["method","sample"],"slot_uri":"MIXS:0001225","alias":"samp_collect_method","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$|([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}|{text}$","interpolated":true,"partial_match":true}},"env_medium":{"name":"env_medium","description":"Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of \'environmental material\' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top)","title":"environmental medium","examples":[{"value":"bluegrass field soil [ENVO:00005789]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["environmental"],"slot_uri":"MIXS:0000014","alias":"env_medium","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\]$","structured_pattern":{"syntax":"^{termLabel} \\\\[{termID}\\\\]$","interpolated":true,"partial_match":true}},"samp_taxon_id":{"name":"samp_taxon_id","description":"NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use \'synthetic metagenome  for mock community/positive controls, or \'blank sample\' for negative controls","title":"taxonomy ID of DNA sample","examples":[{"value":"Gut Metagenome [NCBITaxon:749906]"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dna","identifier","sample","taxon"],"slot_uri":"MIXS:0001320","alias":"samp_taxon_id","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[NCBITaxon:\\\\d+\\\\]$","structured_pattern":{"syntax":"^{text} \\\\[{NCBItaxon_id}\\\\]$","interpolated":true,"partial_match":true}},"geo_loc_name":{"name":"geo_loc_name","description":"The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)","title":"geographic location (country and/or sea,region)","examples":[{"value":"USA: Maryland, Bethesda"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["geographic","location"],"slot_uri":"MIXS:0000010","alias":"geo_loc_name","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+): ([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+), ([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{text}: {text}, {text}$","interpolated":true,"partial_match":true}},"collection_date":{"name":"collection_date","description":"The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant","title":"collection date","examples":[{"value":"2013-03-25T12:42:31+01:00"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["date"],"slot_uri":"MIXS:0000011","alias":"collection_date","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"datetime","required":true},"seq_meth":{"name":"seq_meth","description":"Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103)","title":"sequencing method","examples":[{"value":"454 Genome Sequencer FLX [OBI:0000702]"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["method"],"slot_uri":"MIXS:0000050","alias":"seq_meth","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)|(([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\])$","structured_pattern":{"syntax":"^{text}|({termLabel} \\\\[{termID}\\\\])$","interpolated":true,"partial_match":true}},"lat_lon":{"name":"lat_lon","annotations":{"Expected_value":{"tag":"Expected_value","value":"decimal degrees,  limit to 8 decimal points"}},"description":"The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system","title":"geographic location (latitude and longitude)","examples":[{"value":"50.586825 6.408977"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["geographic","location"],"string_serialization":"{float} {float}","slot_uri":"MIXS:0000009","alias":"lat_lon","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^(-?((?:[0-8]?[0-9](?:\\\\.\\\\d{0,8})?)|90)) -?[0-9]+(?:\\\\.[0-9]{0,8})?$|^-?(1[0-7]{1,2})$","structured_pattern":{"syntax":"^{lat} {lon}$","interpolated":true,"partial_match":true}},"elev":{"name":"elev","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth\'s surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit","title":"elevation","examples":[{"value":"100 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["elevation"],"slot_uri":"MIXS:0000093","alias":"elev","owner":"MimsHostAssociatedAncient","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"env_broad_scale":{"name":"env_broad_scale","description":"Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS","title":"broad-scale environmental context","examples":[{"value":"rangeland biome [ENVO:01000247]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["context","environmental"],"slot_uri":"MIXS:0000012","alias":"env_broad_scale","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\]$","structured_pattern":{"syntax":"^{termLabel} \\\\[{termID}\\\\]$","interpolated":true,"partial_match":true}},"tax_class":{"name":"tax_class","description":"Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes","title":"taxonomic classification","examples":[{"value":"vConTACT vContact2 (references from NCBI RefSeq v83, genus rank classification, default parameters)"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["classification","taxon"],"slot_uri":"MIXS:0000064","alias":"tax_class","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"experimental_factor":{"name":"experimental_factor","annotations":{"Expected_value":{"tag":"Expected_value","value":"text or EFO and/or OBI"}},"description":"Variable aspects of an experiment design that can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI)","title":"experimental factor","examples":[{"value":"time series design [EFO:0001779]"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["experimental","factor"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000008","alias":"experimental_factor","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true,"pattern":"^\\\\S+.*\\\\S+ \\\\[[a-zA-Z]{2,}:\\\\d+\\\\]$"},"associated_resource":{"name":"associated_resource","annotations":{"Expected_value":{"tag":"Expected_value","value":"reference to resource"}},"description":"A related resource that is referenced, cited, or otherwise associated to the sequence","title":"relevant electronic resources","examples":[{"value":"http://www.earthmicrobiome.org/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["resource"],"string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000091","alias":"associated_resource","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true},"sop":{"name":"sop","annotations":{"Expected_value":{"tag":"Expected_value","value":"reference to SOP"}},"description":"Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences","title":"relevant standard operating procedures","examples":[{"value":"http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/its/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["procedures"],"string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000090","alias":"sop","owner":"MimsHostAssociatedAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true},"ances_data":{"name":"ances_data","description":"Information about either pedigree or other ancestral information description (e.g. parental variety in case of mutant or selection), e.g. A/3*B (meaning [(A x B) x B] x B)","title":"ancestral data","examples":[{"value":"A/3*B"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000247","alias":"ances_data","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"biol_stat":{"name":"biol_stat","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"The level of genome modification","title":"biological status","examples":[{"value":"natural"}],"from_schema":"https://w3id.org/mixs","keywords":["status"],"string_serialization":"[wild|natural|semi-natural|inbred line|breeder\'s line|hybrid|clonal selection|mutant]","slot_uri":"MIXS:0000858","alias":"biol_stat","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"genetic_mod":{"name":"genetic_mod","annotations":{"Expected_value":{"tag":"Expected_value","value":"PMID, DOI, URL or text"}},"description":"Genetic modifications of the genome of an organism, which may occur naturally by spontaneous mutation, or be introduced by some experimental means, e.g. specification of a transgene or the gene knocked-out or details of transient transfection","title":"genetic modification","examples":[{"value":"aox1A transgenic"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000859","alias":"genetic_mod","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_common_name":{"name":"host_common_name","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":""}},"description":"Common name of the host","title":"host common name","examples":[{"value":"human"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000248","alias":"host_common_name","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"samp_capt_status":{"name":"samp_capt_status","description":"Reason for the sample","title":"sample capture status","examples":[{"value":"farm sample"}],"from_schema":"https://w3id.org/mixs","keywords":["sample","status"],"slot_uri":"MIXS:0000860","alias":"samp_capt_status","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"SAMP_CAPT_STATUS_ENUM"},"samp_dis_stage":{"name":"samp_dis_stage","description":"Stage of the disease at the time of sample collection, e.g. inoculation, penetration, infection, growth and reproduction, dissemination of pathogen","title":"sample disease stage","examples":[{"value":"infection"}],"from_schema":"https://w3id.org/mixs","keywords":["disease","sample"],"slot_uri":"MIXS:0000249","alias":"samp_dis_stage","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"SAMP_DIS_STAGE_ENUM"},"host_taxid":{"name":"host_taxid","annotations":{"Expected_value":{"tag":"Expected_value","value":"NCBI taxon identifier"}},"description":"NCBI taxon id of the host, e.g. 9606","title":"host taxid","examples":[{"value":"7955"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","taxon"],"string_serialization":"{NCBI taxid}","slot_uri":"MIXS:0000250","alias":"host_taxid","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_subject_id":{"name":"host_subject_id","description":"A unique identifier by which each subject can be referred to, de-identified","title":"host subject id","examples":[{"value":"MPI123"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","identifier"],"slot_uri":"MIXS:0000861","alias":"host_subject_id","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_age":{"name":"host_age","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"year, day, hour"}},"description":"Age of host at the time of sampling; relevant scale depends on species and study, e.g. Could be seconds for amoebae or centuries for trees","title":"host age","examples":[{"value":"10 days"}],"from_schema":"https://w3id.org/mixs","keywords":["age","host","host."],"slot_uri":"MIXS:0000255","alias":"host_age","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_life_stage":{"name":"host_life_stage","annotations":{"Expected_value":{"tag":"Expected_value","value":"stage"}},"description":"Description of life stage of host","title":"host life stage","examples":[{"value":"adult"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","life"],"string_serialization":"{text}","slot_uri":"MIXS:0000251","alias":"host_life_stage","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_sex":{"name":"host_sex","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Gender or physical sex of the host","title":"host sex","comments":["example of non-binary from Excel sheets does not match any of the enumerated values"],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"string_serialization":"[female|hermaphrodite|non-binary|male|transgender|transgender (female to male)|transgender (male to female)\\n|undeclared]","slot_uri":"MIXS:0000811","alias":"host_sex","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_disease_stat":{"name":"host_disease_stat","annotations":{"Expected_value":{"tag":"Expected_value","value":"disease name or Disease Ontology term"}},"description":"List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text","title":"host disease status","examples":[{"value":"rabies [DOID:11260]"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["disease","host","host.","status"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000031","alias":"host_disease_stat","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"chem_administration":{"name":"chem_administration","annotations":{"Expected_value":{"tag":"Expected_value","value":"CHEBI;timestamp"}},"description":"List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi","title":"chemical administration","examples":[{"value":"agar [CHEBI:2509];2018-05-11T20:00Z"}],"from_schema":"https://w3id.org/mixs","keywords":["administration"],"string_serialization":"{termLabel} [{termID}];{timestamp}","slot_uri":"MIXS:0000751","alias":"chem_administration","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"host_body_habitat":{"name":"host_body_habitat","description":"Original body habitat where the sample was obtained from","title":"host body habitat","examples":[{"value":"nasopharynx"}],"from_schema":"https://w3id.org/mixs","keywords":["body","habitat","host","host."],"slot_uri":"MIXS:0000866","alias":"host_body_habitat","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_body_site":{"name":"host_body_site","annotations":{"Expected_value":{"tag":"Expected_value","value":"FMA or UBERON"}},"description":"Name of body site where the sample was obtained from, such as a specific organ or tissue (tongue, lung etc...). Use terms from the foundational model of anatomy ontology (fma) or the Uber-anatomy ontology (UBERON)","title":"host body site","examples":[{"value":"gill [UBERON:0002535]"}],"from_schema":"https://w3id.org/mixs","keywords":["body","host","host.","site"],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000867","alias":"host_body_site","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_body_product":{"name":"host_body_product","annotations":{"Expected_value":{"tag":"Expected_value","value":"FMA or UBERON"}},"description":"Substance produced by the body, e.g. Stool, mucus, where the sample was obtained from. Use terms from the foundational model of anatomy ontology (fma) or Uber-anatomy ontology (UBERON)","title":"host body product","examples":[{"value":"mucus [FMA:66938]"}],"from_schema":"https://w3id.org/mixs","keywords":["body","host","host.","product"],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000888","alias":"host_body_product","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_tot_mass":{"name":"host_tot_mass","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"kilogram, gram"}},"description":"Total mass of the host at collection, the unit depends on host","title":"host total mass","examples":[{"value":"2500 gram"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","mass","total"],"slot_uri":"MIXS:0000263","alias":"host_tot_mass","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_height":{"name":"host_height","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"centimeter, millimeter, meter"}},"description":"The height of subject","title":"host height","examples":[{"value":"0.1 meter"}],"from_schema":"https://w3id.org/mixs","keywords":["height","host","host."],"slot_uri":"MIXS:0000264","alias":"host_height","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_length":{"name":"host_length","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"centimeter, millimeter, meter"}},"description":"The length of subject","title":"host length","examples":[{"value":"1 meter"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","length"],"slot_uri":"MIXS:0000256","alias":"host_length","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_diet":{"name":"host_diet","description":"Type of diet depending on the host, for animals omnivore, herbivore etc., for humans high-fat, meditteranean etc.; can include multiple diet types","title":"host diet","examples":[{"value":"herbivore"}],"from_schema":"https://w3id.org/mixs","keywords":["diet","host","host."],"slot_uri":"MIXS:0000869","alias":"host_diet","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"host_last_meal":{"name":"host_last_meal","annotations":{"Expected_value":{"tag":"Expected_value","value":"content;duration"}},"description":"Content of last meal and time since feeding; can include multiple values","title":"host last meal","examples":[{"value":"corn feed;P2H"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"string_serialization":"{text};{duration}","slot_uri":"MIXS:0000870","alias":"host_last_meal","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"host_growth_cond":{"name":"host_growth_cond","description":"Literature reference giving growth conditions of the host","title":"host growth conditions","examples":[{"value":"https://academic.oup.com/icesjms/article/68/2/349/617247"}],"from_schema":"https://w3id.org/mixs","keywords":["condition","growth","host","host."],"slot_uri":"MIXS:0000871","alias":"host_growth_cond","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$|([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}|{text}$","interpolated":true,"partial_match":true}},"host_substrate":{"name":"host_substrate","description":"The growth substrate of the host","title":"host substrate","examples":[{"value":"rock"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000252","alias":"host_substrate","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_fam_rel":{"name":"host_fam_rel","annotations":{"Expected_value":{"tag":"Expected_value","value":"relationship type;arbitrary identifier"}},"description":"Relationships to other hosts in the same study; can include multiple relationships","title":"host family relationship","examples":[{"value":"offspring;Mussel25"}],"from_schema":"https://w3id.org/mixs","keywords":["family","host","host.","relationship"],"string_serialization":"{text};{text}","slot_uri":"MIXS:0000872","alias":"host_fam_rel","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"host_subspecf_genlin":{"name":"host_subspecf_genlin","annotations":{"Expected_value":{"tag":"Expected_value","value":"Genetic lineage below lowest rank of NCBI taxonomy, which is subspecies, e.g. serovar, biotype, ecotype, variety, cultivar"}},"description":"Information about the genetic distinctness of the host organism below the subspecies level e.g., serovar, serotype, biotype, ecotype, variety, cultivar, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123","title":"host subspecific genetic lineage","examples":[{"value":"serovar:Newport, variety:glabrum, cultivar: Red Delicious"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","lineage"],"string_serialization":"{rank name}:{text}","slot_uri":"MIXS:0001318","alias":"host_subspecf_genlin","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"host_genotype":{"name":"host_genotype","description":"Observed genotype","title":"host genotype","examples":[{"value":"C57BL/6"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000365","alias":"host_genotype","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_phenotype":{"name":"host_phenotype","annotations":{"Expected_value":{"tag":"Expected_value","value":"PATO or HP"}},"description":"Phenotype of human or other host. Use terms from the phenotypic quality ontology (pato) or the Human Phenotype Ontology (HP)","title":"host phenotype","examples":[{"value":"elongated [PATO:0001154]"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000874","alias":"host_phenotype","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_body_temp":{"name":"host_body_temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Core body temperature of the host when sample was collected","title":"host body temperature","examples":[{"value":"15 degree Celsius"}],"from_schema":"https://w3id.org/mixs","keywords":["body","host","host.","temperature"],"slot_uri":"MIXS:0000274","alias":"host_body_temp","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_dry_mass":{"name":"host_dry_mass","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"kilogram, gram"}},"description":"Measurement of dry mass","title":"host dry mass","examples":[{"value":"500 gram"}],"from_schema":"https://w3id.org/mixs","keywords":["dry","host","host.","mass"],"slot_uri":"MIXS:0000257","alias":"host_dry_mass","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"blood_press_diast":{"name":"blood_press_diast","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"millimeter mercury"}},"description":"Resting diastolic blood pressure, measured as mm mercury","title":"host blood pressure diastolic","from_schema":"https://w3id.org/mixs","keywords":["host","host.","pressure"],"slot_uri":"MIXS:0000258","alias":"blood_press_diast","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"blood_press_syst":{"name":"blood_press_syst","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"millimeter mercury"}},"description":"Resting systolic blood pressure, measured as mm mercury","title":"host blood pressure systolic","from_schema":"https://w3id.org/mixs","keywords":["host","host.","pressure"],"slot_uri":"MIXS:0000259","alias":"blood_press_syst","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"host_color":{"name":"host_color","description":"The color of host","title":"host color","from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000260","alias":"host_color","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"host_shape":{"name":"host_shape","description":"Morphological shape of host","title":"host shape","examples":[{"value":"round"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host."],"slot_uri":"MIXS:0000261","alias":"host_shape","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"gravidity":{"name":"gravidity","description":"Whether or not subject is gravid, and if yes date due or date post-conception, specifying which is used","title":"gravidity","examples":[{"value":"yes;due date:2018-05-11"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{boolean};{timestamp}","slot_uri":"MIXS:0000875","alias":"gravidity","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string"},"perturbation":{"name":"perturbation","annotations":{"Expected_value":{"tag":"Expected_value","value":"perturbation type name;perturbation interval and duration"}},"description":"Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types","title":"perturbation","examples":[{"value":"antibiotic addition;R2/2018-05-11T14:30Z/2018-05-11T19:30Z/P1H30M"}],"from_schema":"https://w3id.org/mixs","keywords":["perturbation"],"string_serialization":"{text};{Rn/start_time/end_time/duration}","slot_uri":"MIXS:0000754","alias":"perturbation","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"salinity":{"name":"salinity","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"practical salinity unit, percentage"}},"description":"The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater","title":"salinity","examples":[{"value":"25 practical salinity unit"}],"from_schema":"https://w3id.org/mixs","keywords":["salinity"],"slot_uri":"MIXS:0000183","alias":"salinity","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"oxy_stat_samp":{"name":"oxy_stat_samp","description":"Oxygenation status of sample","title":"oxygenation status of sample","examples":[{"value":"aerobic"}],"from_schema":"https://w3id.org/mixs","keywords":["oxygen","sample","status"],"slot_uri":"MIXS:0000753","alias":"oxy_stat_samp","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"OXY_STAT_SAMP_ENUM"},"organism_count":{"name":"organism_count","annotations":{"Expected_value":{"tag":"Expected_value","value":"organism name;measurement value;enumeration"}},"description":"Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)","title":"organism count","examples":[{"value":"total prokaryotes;3.5e7 cells per milliliter;qPCR"}],"from_schema":"https://w3id.org/mixs","keywords":["count","organism"],"string_serialization":"{text};{float} {unit};[ATP|MPN|qPCR|other]","slot_uri":"MIXS:0000103","alias":"organism_count","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"samp_store_temp":{"name":"samp_store_temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Temperature at which sample was stored, e.g. -80 degree Celsius","title":"sample storage temperature","examples":[{"value":"-80 degree Celsius"}],"from_schema":"https://w3id.org/mixs","keywords":["sample","storage","temperature"],"slot_uri":"MIXS:0000110","alias":"samp_store_temp","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_store_dur":{"name":"samp_store_dur","description":"Duration for which the sample was stored. Indicate the duration for which the sample was stored written in ISO 8601 format","title":"sample storage duration","examples":[{"value":"P1Y6M"}],"from_schema":"https://w3id.org/mixs","keywords":["duration","period","sample","storage"],"slot_uri":"MIXS:0000116","alias":"samp_store_dur","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^P(?:(?:\\\\d+D|\\\\d+M(?:\\\\d+D)?|\\\\d+Y(?:\\\\d+M(?:\\\\d+D)?)?)(?:T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S))?|T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S)|\\\\d+W)$","structured_pattern":{"syntax":"^{duration}$","interpolated":true,"partial_match":true}},"samp_store_loc":{"name":"samp_store_loc","annotations":{"Expected_value":{"tag":"Expected_value","value":"location name"}},"description":"Location at which sample was stored, usually name of a specific freezer/room","title":"sample storage location","examples":[{"value":"Freezer no:5"}],"from_schema":"https://w3id.org/mixs","keywords":["location","sample","storage"],"string_serialization":"{text}","slot_uri":"MIXS:0000755","alias":"samp_store_loc","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"host_symbiont":{"name":"host_symbiont","annotations":{"Expected_value":{"tag":"Expected_value","value":"species name or common name"}},"description":"The taxonomic name of the organism(s) found living in mutualistic, commensalistic, or parasitic symbiosis with the specific host. The sampled symbiont can have its own symbionts. For example, parasites may have hyperparasites (=parasites of the parasite)","title":"observed host symbionts","examples":[{"value":"flukeworms"}],"from_schema":"https://w3id.org/mixs","keywords":["host","host.","observed","symbiosis"],"string_serialization":"{text}","slot_uri":"MIXS:0001298","alias":"host_symbiont","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","MimsHostAssociated"],"range":"string","multivalued":true},"misc_param":{"name":"misc_param","annotations":{"Expected_value":{"tag":"Expected_value","value":"parameter name;measurement value"}},"description":"Any other measurement performed or parameter collected, that is not listed here","title":"miscellaneous parameter","examples":[{"value":"Bicarbonate ion concentration;2075 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","keywords":["parameter"],"string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000752","alias":"misc_param","owner":"MimsHostAssociatedAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"collection_context_name":{"name":"collection_context_name","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of institution"}},"description":"Name of where sample originated and is typically stored. Typically will be \'owning institution\'.","title":"context name where location was collected","examples":[{"value":"Natural History Museum London"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["storage"],"slot_uri":"MIXS:999999901","alias":"collection_context_name","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"cultural_era":{"name":"cultural_era","annotations":{"Expected_value":{"tag":"Expected_value","value":"chronotology or PeriodO term; text"}},"description":"The cultural era approximating the period in which the individual lived from https://chronontology.dainst.org/ or PeriodO if possible. If not available, follow INSDC standard for reporting missing data (https://www.insdc.org/submitting-standards/missing-value-reporting/).","title":"cultural era or period","examples":[{"value":"Copper Age [Chronotology: NW6hofAScJSE]"},{"value":"Upper Middle Palaeolthic"},{"value":"Not collected"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient","age"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:999999902","alias":"cultural_era","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":true,"recommended":true,"multivalued":false},"damage_treatment":{"name":"damage_treatment","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Indication of whether characteristic ancient DNA damage has been  altered or removed from a DNA extract in a laboratory","title":"Damage treatment type","examples":[{"value":"none"},{"value":"partial-udg"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"slot_uri":"MIXS:999999903","alias":"damage_treatment","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"DamageTreatmentEnum","required":true,"multivalued":true},"dna_extraction_date":{"name":"dna_extraction_date","description":"The date when the nucleic acid extraction was started from the sample material. In case no exact time is available, the date can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant","title":"date of extraction of nucleic acids from sample","examples":[{"value":"2023-12-01"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999904","alias":"dna_extraction_date","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"datetime","required":false,"recommended":true},"ethical_authority":{"name":"ethical_authority","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of authority providing ethical approval"}},"description":"Name of the authorit(ies) or institution(s) that awarded sampling and  analysis (e.g. human remains) and/or export permission  (e.g. animal remains)","title":"ethical authority","examples":[{"value":"University of Copenhagen"},{"value":"Federal Foreign Office (Germany)"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["ethics","location"],"slot_uri":"MIXS:999999905","alias":"ethical_authority","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"ethical_date":{"name":"ethical_date","description":"Date of award of ethical/export permission. The date can be right truncated  i.e. all of these are valid times: 2008-01-23; 2008-01; 2008;  Except: 2008-01; 2008 all are ISO8601 compliant","title":"date of ethical approval","examples":[{"value":"2023-12-01"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999906","alias":"ethical_date","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"datetime","required":false,"recommended":true},"ethical_id":{"name":"ethical_id","description":"The permissions code, ID, document, or similar provided by any authority  (socal, legal, academic etc.) associated with approval of the analysis of  this particular sample.","title":"ethical permit or approval ID","examples":[{"value":"DE-123-JK"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"slot_uri":"MIXS:999999906","alias":"ethical_id","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"experimental_procedures":{"name":"experimental_procedures","description":"Provide a DOI or URL to refer to the paper where the field report, nucleic acid extraction,  library construction, and other procedures are explained in more detail, e.g. the paper reporting the data.","title":"experimental procedures","examples":[{"value":"doi:10.1093/nar/gkr771"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999907","alias":"experimental_procedures","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_concentration":{"name":"lib_concentration","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement value in number of ng per l"}},"description":"Concentration of library in ng per l, as inferred by qPCR, digital PCR, Qubit etc.","title":"library concentration","examples":[{"value":"2"},{"value":"0.5"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{float}","slot_uri":"MIXS:999999908","alias":"lib_concentration","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"lib_index_polymerase":{"name":"lib_index_polymerase","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of polymerase used during library indexing"}},"description":"The DNA polymerase enzyme used for indexing DNA libraries","title":"library index polymerase","examples":[{"value":"Agilent PfuTurbo Cx HotStart"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["polymerase","library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999909","alias":"lib_index_polymerase","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true},"lib_preparation_protocol":{"name":"lib_preparation_protocol","description":"Citation(s) for the DNA library preparation protocol","title":"library preparation protocols","examples":[{"value":"doi:10.1093/nar/gkr771"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999907","alias":"lib_preparation_protocol","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_reamplification_polymerase":{"name":"lib_reamplification_polymerase","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of polymerase used during library indexing"}},"description":"The name of the DNA polymerase enzyme used for reamplifying DNA libraries post-indexing.","title":"library reamplification polymerase","examples":[{"value":"KAPA HiFi HotStart Uracil+"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["polymerase","library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999909","alias":"lib_reamplification_polymerase","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true},"lib_type":{"name":"lib_type","description":"The type of library created, i.e., amplicon, enriched, or shotgun.  Amplicon based is a library that has been amplified to target a specific single region of a genome (e.g. a specific gene).  An enriched library is a library has had a particular genome or genome regions \'captured\' or enriched typically via baits or probes. A shotgun library has had no type of targeted enrichment for a particular gene or genome.","title":"library type","examples":[{"value":"amplicon"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"slot_uri":"MIXS:999999910","alias":"lib_type","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"LibTypeEnum","required":false},"neg_cont_status":{"name":"neg_cont_status","description":"Specify whether the sample is a negative control or not.","title":"negative control status","examples":[{"value":"true"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["control"],"slot_uri":"MIXS:999999911","alias":"neg_cont_status","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"boolean"},"num_capture_amp_cycles":{"name":"num_capture_amp_cycles","description":"Number of amplification cycles after capture enrichment","title":"number of reamplification cycles after capture","examples":[{"value":"12"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999912","alias":"num_capture_amp_cycles","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer","multivalued":true},"num_reamp_cycles":{"name":"num_reamp_cycles","description":"Number of amplification cycles after library indexing PCR. If capture data, this refers to amplifications prior to the capture experiments.","title":"number of reamplification cycles","examples":[{"value":"12"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999913","alias":"num_reamp_cycles","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer"},"preservational_treatment":{"name":"preservational_treatment","description":"Description of any treatment applied to samples for the purpose of  maximising collection preservation that may influence downstream  DNA recovery or library construction, such as storage fluid or  reconstructive glue","title":"preservational treatment","examples":[{"value":"stored in formalin"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999914","alias":"preservational_treatment","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"prev_pubs":{"name":"prev_pubs","description":"Any publications that report data from the same body/skeleton/individual","title":"previous publications","examples":[{"value":"doi:10.1016/j.jas.2015.02.0181"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999915","alias":"prev_pubs","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"recovery_date":{"name":"recovery_date","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"year"}},"description":"Date of excavation or retrieval from burial or depositional context, if known. In case no exact time is available, the date can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant. If excavations were done during a longer period, report its midpoint at a month or year level.","title":"date of retrieval from depositional context","examples":[{"value":"2023"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999916","alias":"recovery_date","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"datetime","required":false,"recommended":true},"samp_decontam_pretreat":{"name":"samp_decontam_pretreat","description":"Method(s) employed for surface decontamination of samples of external  modern DNA; Treatment used on the samples. Depends on the sample type.  More relevant for bones than environmental samples. E.g. buffers, EDTA, etc.","title":"sample decontamination pretreatment","examples":[{"value":"doi:10.1016/j.jas.2015.02.0181"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999917","alias":"samp_decontam_pretreat","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"samp_age_range_youngest_limit":{"name":"samp_age_range_youngest_limit","annotations":{"Expected_value":{"tag":"Expected_value","value":"age value corresponding to unit"}},"description":"The youngest possible age of the sample. Youngest refers to the time closest to present day. If a sample has only a single age value, the value from samp_age_range_oldest_limit can be repeated. Sample age refers to a period of time in the past, such as when an organism was living and then died, not the biological age.","title":"sample age range youngest limit","examples":[{"value":"100000"},{"value":"1700"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999918","alias":"samp_age_range_youngest_limit","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer","required":true,"recommended":true},"samp_age_range_youngest_unit":{"name":"samp_age_range_youngest_unit","description":"The unit of the youngest age in the sample age range.","title":"sample age range youngest unit","examples":[{"value":"cal BP"},{"value":"CE"},{"value":"Ma"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999919","alias":"samp_age_range_youngest_unit","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"SampAgeUnit","required":true,"recommended":true,"multivalued":true},"samp_age_range_oldest_limit":{"name":"samp_age_range_oldest_limit","annotations":{"Expected_value":{"tag":"Expected_value","value":"age value corresponding to unit"}},"description":"The oldest possible age of the sample. Oldest refers to the time furthest from present day. If a sample has only a single age value, the value from samp_age_range_youngest_limit can be repeated. Sample age refers to a period of time in the past, such as when an organism was living and then died, not the biological age.","title":"sample age range oldest limit","examples":[{"value":"120000"},{"value":"1900"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999920","alias":"samp_age_range_oldest_limit","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer","required":true,"recommended":true},"samp_age_range_oldest_unit":{"name":"samp_age_range_oldest_unit","description":"The unit of the oldest age in sample age range.","title":"sample age range oldest unit","examples":[{"value":"cal BP"},{"value":"CE"},{"value":"Ma"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999921","alias":"samp_age_range_oldest_unit","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"SampAgeUnit","required":true,"recommended":true,"multivalued":true},"samp_age_range_inference_methods":{"name":"samp_age_range_inference_methods","description":"The method used to infer the sample age. This corresponds to a fixed list of most commonly used dating methods, such as 14C, OSL, stratigraphic inference, historical records, other etc. If the method is not in the list, please provide a description in samp_age_range_inference_description.","title":"sample age inference methods","examples":[{"value":"radiocarbon dating, calibrated with OxCal v4.3"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999922","alias":"samp_age_range_inference_methods","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"SampAgeInferMethod","required":true,"recommended":true,"multivalued":true},"samp_age_range_inference_description":{"name":"samp_age_range_inference_description","description":"Additional general information about the inference method used to infer the sample age that can be useful for  understanding how the sample age was estimated. This can be things such as whether radiocarbon age estimated on AMS or conventional radiocarbon dating, or which calibration confidence interval was used.  It can also be used to describe what artefacts were used to age the stratigraphic layer of historical contexts.","title":"sample age inference description","examples":[{"value":"radiocarbon dating, calibrated with OxCal v4.3 with 95% confidence interval"},{"value":"based on proxy dating from other bone samples of stratigraphic layer"},{"value":"a coin found in the burial was from the 3rd century"},{"value":"age taken from previous publication Doe et al. 2019"},{"value":"radiocarbon age ID: OxA-12345"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999923","alias":"samp_age_range_inference_description","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":false},"samp_alt_lab_ids":{"name":"samp_alt_lab_ids","description":"An alternative sample or material ID related to the sample not already covered by terms samp_name and source_mat_id, including from associated other non-genetic analyses of the same sample (e.g. museum IDs, internal lab sample ID from sampling such a bone drilling). Can be specified multiple times for different ID contexts.","title":"alternative sample IDs","examples":[{"value":"ABC_24"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:999999924","alias":"samp_alt_lab_ids","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"storage_conditions":{"name":"storage_conditions","description":"General conditions in which the material (before sampling for DNA analysis) was stored in long-term collection storage, that may influenced DNA recovery or library construction. For example, specify temperature, humidity, presence of microbial overgrowth etc..","title":"storage conditions","examples":[{"value":"climate-controlled"},{"value":"Stored in the museum from recovery_date to collection_date"},{"value":"Stored at -20oC until dna_extraction_date"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:999999925","alias":"storage_conditions","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string"},"palaeopath_status":{"name":"palaeopath_status","annotations":{"Expected_value":{"tag":"Expected_value","value":"description of health related observation on ancient remains"}},"description":"Describe briefly any relevant palaeopathological or health-related observations of the remains of the individual under study.","title":"palaeopathology status.","examples":[{"value":"Osteoporosis."},{"value":"Parasites found in pelvic area."},{"value":"Caries on right upper molar."}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["palaeopathology","host health","ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999926","alias":"palaeopath_status","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":false},"biocultural_label":{"name":"biocultural_label","annotations":{"Expected_value":{"tag":"Expected_value","value":"Relevant biocultural label from https://localcontexts.org/labels/biocultural-labels/"}},"description":"Relevant biocultural labels that apply to the sample under study, based on https://localcontexts.org/label/bc-provenance/. These help indicate how indigenous peoples and similar communities view in which contexts this data can be reused.","title":"biocultural label","examples":[{"value":"BC R"},{"value":"BC MC"}],"in_subset":["ethics"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"slot_uri":"MIXS:999999927","alias":"biocultural_label","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"BioCulturalLabel","required":false,"recommended":false,"multivalued":true},"ethics_perm_scope":{"name":"ethics_perm_scope","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the scope of ethical permissions for data use"}},"description":"Description of the original scope and permissions of the research on the genetic material, as was approved by the ethical authority (e.g. bacteria only, DNA only, bacteria and human, no host read analysis allowed).","title":"ethical Permissions & Scope","examples":[{"value":"Defined scope only includes the study of bacterial sequences and any human sequence is not covered under the agreement."}],"in_subset":["ethics"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"string_serialization":"{text}","slot_uri":"MIXS:999999928","alias":"ethics_perm_scope","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"host_body_preservation_state":{"name":"host_body_preservation_state","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the preservation of the sampled (ancient) organism/host"}},"description":"Description of the state of the sampled (ancient) organism/host as originally preserved in the burial environment, e.g. if the the sample was artificially or naturally (fully/partially) mummified at the time of death.","title":"ancient body preservation state","examples":[{"value":"complete artificial mummification."},{"value":"natural partial mummification."},{"value":"fully skeletonised."}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999929","alias":"host_body_preservation_state","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":false},"lib_strandedness":{"name":"lib_strandedness","annotations":{"Expected_value":{"tag":"Expected_value","value":"DNA library strandedness"}},"description":"The strandedness of the original template DNA molecules used for constructing the sequencing library","title":"DNA strandedness in library creation","examples":[{"value":"single"},{"value":"double"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"slot_uri":"MIXS:999999930","alias":"lib_strandedness","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"LibStrand","required":false,"recommended":true,"multivalued":true},"enrichment_probe_set":{"name":"enrichment_probe_set","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of probe set used for target enrichment/library selection"}},"description":"Description of target enrichment probe designs used (e.g., species included, sequences, type, company). This can include custom kits (please provide a general description and DOI if available) or commercially available kit (provide ID and company).","title":"enrichment probe design description","examples":[{"value":"Custom probe design (70 bp biotinylated RNA probes) covering 500 full E. Coli genomes; company TE-Science"},{"value":"Commercial kit ABC001-EC from company TE-Science, purchased 2020"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","enrichment"],"string_serialization":"{text}","slot_uri":"MIXS:999999931","alias":"enrichment_probe_set","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":true},"mid_config":{"name":"mid_config","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the indexing configuration of the library"}},"description":"Index/barcode/primer configuration used during library building for sequencing. This includes information such as the number, type and location of indexes, the index/primer kit/list, or if \'inline\' barcodes or UMIs were ligated directly onto the template molecules.","title":"multiplex identifiers or indexing configuration of libraries","examples":[{"value":"dual index with single internal barcode."},{"value":"UDI index sequences."}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999932","alias":"mid_config","owner":"MimsHostAssociatedAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true}},"class_uri":"MIXS:0010007_0016002_9999903"},"MimsSedimentAncient":{"name":"MimsSedimentAncient","description":"MIxS Data that comply with the Mims checklist and Sediment","title":"MimsSediment combined with Ancient","in_subset":["combination_classes"],"from_schema":"https://w3id.org/mixs","is_a":"Ancient","mixins":["MimsSediment"],"attributes":{"samp_name":{"name":"samp_name","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":""}},"description":"A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name","title":"sample name","examples":[{"value":"ISDsoil1"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["sample"],"slot_uri":"MIXS:0001107","alias":"samp_name","owner":"MimsSedimentAncient","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"size_frac":{"name":"size_frac","annotations":{"Expected_value":{"tag":"Expected_value","value":"filter size value range"}},"description":"Filtering pore size used in sample preparation","title":"size fraction selected","examples":[{"value":"0-0.22 micrometer"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["fraction","size"],"string_serialization":"{float}-{float} {unit}","slot_uri":"MIXS:0000017","alias":"size_frac","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"lib_screen":{"name":"lib_screen","annotations":{"Expected_value":{"tag":"Expected_value","value":"screening strategy name"}},"description":"Specific enrichment or screening methods applied before and/or after creating libraries","title":"library screening strategy","examples":[{"value":"enriched, screened, normalized"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"string_serialization":"{text}","slot_uri":"MIXS:0000043","alias":"lib_screen","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"ref_db":{"name":"ref_db","annotations":{"Expected_value":{"tag":"Expected_value","value":"names, versions, and references of databases"}},"description":"List of database(s) used for ORF annotation, along with version number and reference to website or publication","title":"reference database(s)","examples":[{"value":"pVOGs;5;http://dmk-brain.ecn.uiowa.edu/pVOGs/ Grazziotin et al. 2017 doi:10.1093/nar/gkw975"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["database"],"string_serialization":"{database};{version};{reference}","slot_uri":"MIXS:0000062","alias":"ref_db","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"nucl_acid_amp":{"name":"nucl_acid_amp","description":"A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids","title":"nucleic acid amplification","examples":[{"value":"https://phylogenomics.me/protocols/16s-pcr-protocol/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000038","alias":"nucl_acid_amp","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_size":{"name":"lib_size","description":"Total number of clones in the library prepared for the project","title":"library size","examples":[{"value":"50"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","size"],"slot_uri":"MIXS:0000039","alias":"lib_size","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer","recommended":true},"mid":{"name":"mid","description":"Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to specifically tag unique samples in a sequencing run. Sequence should be reported in uppercase letters","title":"multiplex identifiers","examples":[{"value":"GTGAATAT"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["identifier"],"slot_uri":"MIXS:0000047","alias":"mid","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[ACGTRKSYMWBHDVN]+$","structured_pattern":{"syntax":"^{ambiguous_nucleotides}$","interpolated":true,"partial_match":true}},"assembly_name":{"name":"assembly_name","annotations":{"Expected_value":{"tag":"Expected_value","value":"name and version of assembly"}},"description":"Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community","title":"assembly name","examples":[{"value":"HuRef, JCVI_ISG_i3_1.0"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text} {text}","slot_uri":"MIXS:0000057","alias":"assembly_name","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"temp":{"name":"temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Temperature of the sample at the time of sampling","title":"temperature","examples":[{"value":"25 degree Celsius"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["temperature"],"slot_uri":"MIXS:0000113","alias":"temp","owner":"MimsSedimentAncient","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"nucl_acid_ext":{"name":"nucl_acid_ext","description":"A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample","title":"nucleic acid extraction","examples":[{"value":"https://mobio.com/media/wysiwyg/pdfs/protocols/12888.pdf"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000037","alias":"nucl_acid_ext","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"samp_size":{"name":"samp_size","description":"The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected","title":"amount or size of sample collected","examples":[{"value":"5 liter"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["sample","size"],"slot_uri":"MIXS:0000001","alias":"samp_size","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"alt":{"name":"alt","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth\'s surface. In this context, the altitude measurement is the vertical distance between the earth\'s surface above sea level and the sampled position in the air","title":"altitude","examples":[{"value":"100 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000094","alias":"alt","owner":"MimsSedimentAncient","domain_of":["Mims","HostAssociated","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"source_mat_id":{"name":"source_mat_id","annotations":{"Expected_value":{"tag":"Expected_value","value":"for cultures of microorganisms: identifiers for two culture collections; for other material a unique arbitrary identifer"}},"description":"A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain \'UAM:Herps:14\' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2)","title":"source material identifiers","examples":[{"value":"MPI012345"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["identifier","material","source"],"string_serialization":"{text}","slot_uri":"MIXS:0000026","alias":"source_mat_id","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true},"samp_vol_we_dna_ext":{"name":"samp_vol_we_dna_ext","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milliliter, gram, milligram, square centimeter"}},"description":"Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001)","title":"sample volume or weight for DNA extraction","examples":[{"value":"1500 milliliter"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["dna","sample","volume","weight"],"slot_uri":"MIXS:0000111","alias":"samp_vol_we_dna_ext","owner":"MimsSedimentAncient","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"lib_reads_seqd":{"name":"lib_reads_seqd","description":"Total number of clones sequenced from the library","title":"library reads sequenced","examples":[{"value":"20"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"slot_uri":"MIXS:0000040","alias":"lib_reads_seqd","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer","recommended":true},"rel_to_oxygen":{"name":"rel_to_oxygen","description":"Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments","title":"relationship to oxygen","examples":[{"value":"aerobe"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["oxygen","relationship"],"slot_uri":"MIXS:0000015","alias":"rel_to_oxygen","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"REL_TO_OXYGEN_ENUM"},"samp_collect_device":{"name":"samp_collect_device","annotations":{"Expected_value":{"tag":"Expected_value","value":"device name"}},"description":"The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094)","title":"sample collection device","examples":[{"value":"swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713]"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["device","sample"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000002","alias":"samp_collect_device","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"number_contig":{"name":"number_contig","description":"Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG","title":"number of contigs","examples":[{"value":"40"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["number"],"slot_uri":"MIXS:0000060","alias":"number_contig","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"integer","recommended":true},"lib_layout":{"name":"lib_layout","description":"Specify whether to expect single, paired, or other configuration of reads","title":"library layout","examples":[{"value":"paired"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"slot_uri":"MIXS:0000041","alias":"lib_layout","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"LIB_LAYOUT_ENUM","recommended":true},"assembly_qual":{"name":"assembly_qual","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling   90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling < 90% of the expected genome or replicon sequence, or for which no genome size could be estimated","title":"assembly quality","examples":[{"value":"High-quality draft genome"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["quality"],"string_serialization":"[Finished genome|High-quality draft genome|Medium-quality draft genome|Low-quality draft genome|Genome fragment(s)]","slot_uri":"MIXS:0000056","alias":"assembly_qual","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"ref_biomaterial":{"name":"ref_biomaterial","description":"Primary publication if isolated before genome publication; otherwise, primary genome report","title":"reference for biomaterial","examples":[{"value":"doi:10.1016/j.syapm.2018.01.009"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000025","alias":"ref_biomaterial","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"project_name":{"name":"project_name","description":"Name of the project within which the sequencing was organized","title":"project name","examples":[{"value":"Forest soil metagenome"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["project"],"slot_uri":"MIXS:0000092","alias":"project_name","owner":"MimsSedimentAncient","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"lib_vector":{"name":"lib_vector","annotations":{"Expected_value":{"tag":"Expected_value","value":"vector"}},"description":"Cloning vector type(s) used in construction of libraries","title":"library vector","examples":[{"value":"Bacteriophage P1"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library"],"string_serialization":"{text}","slot_uri":"MIXS:0000042","alias":"lib_vector","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"adapters":{"name":"adapters","annotations":{"Expected_value":{"tag":"Expected_value","value":"adapter A and B sequence"}},"description":"Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters","title":"adapters","examples":[{"value":"AATGATACGGCGACCACCGAGATCTACACGCT;CAAGCAGAAGACGGCATACGAGAT"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{dna};{dna}","slot_uri":"MIXS:0000048","alias":"adapters","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"neg_cont_type":{"name":"neg_cont_type","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration or text"}},"description":"The substance or equipment used as a negative control in an investigation","title":"negative control type","in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["type"],"slot_uri":"MIXS:0001321","alias":"neg_cont_type","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"NEG_CONT_TYPE_ENUM","recommended":true},"assembly_software":{"name":"assembly_software","description":"Tool(s) used for assembly, including version number and parameters","title":"assembly software","examples":[{"value":"metaSPAdes;3.11.0;kmer set 21,33,55,77,99,121, default parameters otherwise"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["software"],"slot_uri":"MIXS:0000058","alias":"assembly_software","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"annot":{"name":"annot","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of tool or pipeline used, or annotation source description"}},"description":"Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter","title":"annotation","examples":[{"value":"prokka"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:0000059","alias":"annot","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"pos_cont_type":{"name":"pos_cont_type","description":"The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive","title":"positive control type","in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["type"],"string_serialization":"{term} or {text}","slot_uri":"MIXS:0001322","alias":"pos_cont_type","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"feat_pred":{"name":"feat_pred","description":"Method used to predict UViGs features such as ORFs, integration site, etc","title":"feature prediction","examples":[{"value":"Prodigal;2.6.3;default parameters"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["feature","predict"],"slot_uri":"MIXS:0000061","alias":"feat_pred","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"env_local_scale":{"name":"env_local_scale","annotations":{"Expected_value":{"tag":"Expected_value","value":"Environmental entities having causal influences upon the entity at time of sampling"}},"description":"Report the entity or entities which are in the sample or specimen s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS","title":"local environmental context","examples":[{"value":"hillside [ENVO:01000333]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["context","environmental"],"string_serialization":"{termLabel} [{termID}]","slot_uri":"MIXS:0000013","alias":"env_local_scale","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true},"samp_mat_process":{"name":"samp_mat_process","description":"A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed","title":"sample material processing","examples":[{"value":"filtering of seawater, storing samples in ethanol"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["material","process","sample"],"slot_uri":"MIXS:0000016","alias":"samp_mat_process","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true},"sim_search_meth":{"name":"sim_search_meth","description":"Tool used to compare ORFs with database, along with version and cutoffs used","title":"similarity search method","examples":[{"value":"HMMER3;3.1b2;hmmsearch, cutoff of 50 on score"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["method"],"slot_uri":"MIXS:0000063","alias":"sim_search_meth","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+);([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{software};{version};{parameters}$","interpolated":true,"partial_match":true}},"depth":{"name":"depth","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"The vertical distance below local surface. For sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples","title":"depth","examples":[{"value":"10 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["depth"],"slot_uri":"MIXS:0000018","alias":"depth","owner":"MimsSedimentAncient","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_collect_method":{"name":"samp_collect_method","description":"The method employed for collecting the sample","title":"sample collection method","examples":[{"value":"swabbing"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["method","sample"],"slot_uri":"MIXS:0001225","alias":"samp_collect_method","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$|([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}|{text}$","interpolated":true,"partial_match":true}},"env_medium":{"name":"env_medium","description":"Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of \'environmental material\' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top)","title":"environmental medium","examples":[{"value":"bluegrass field soil [ENVO:00005789]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["environmental"],"slot_uri":"MIXS:0000014","alias":"env_medium","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\]$","structured_pattern":{"syntax":"^{termLabel} \\\\[{termID}\\\\]$","interpolated":true,"partial_match":true}},"samp_taxon_id":{"name":"samp_taxon_id","description":"NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use \'synthetic metagenome  for mock community/positive controls, or \'blank sample\' for negative controls","title":"taxonomy ID of DNA sample","examples":[{"value":"Gut Metagenome [NCBITaxon:749906]"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["dna","identifier","sample","taxon"],"slot_uri":"MIXS:0001320","alias":"samp_taxon_id","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[NCBITaxon:\\\\d+\\\\]$","structured_pattern":{"syntax":"^{text} \\\\[{NCBItaxon_id}\\\\]$","interpolated":true,"partial_match":true}},"geo_loc_name":{"name":"geo_loc_name","description":"The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)","title":"geographic location (country and/or sea,region)","examples":[{"value":"USA: Maryland, Bethesda"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["geographic","location"],"slot_uri":"MIXS:0000010","alias":"geo_loc_name","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+): ([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+), ([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{text}: {text}, {text}$","interpolated":true,"partial_match":true}},"collection_date":{"name":"collection_date","description":"The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant","title":"collection date","examples":[{"value":"2013-03-25T12:42:31+01:00"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["date"],"slot_uri":"MIXS:0000011","alias":"collection_date","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"datetime","required":true},"seq_meth":{"name":"seq_meth","description":"Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103)","title":"sequencing method","examples":[{"value":"454 Genome Sequencer FLX [OBI:0000702]"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["method"],"slot_uri":"MIXS:0000050","alias":"seq_meth","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)|(([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\])$","structured_pattern":{"syntax":"^{text}|({termLabel} \\\\[{termID}\\\\])$","interpolated":true,"partial_match":true}},"lat_lon":{"name":"lat_lon","annotations":{"Expected_value":{"tag":"Expected_value","value":"decimal degrees,  limit to 8 decimal points"}},"description":"The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system","title":"geographic location (latitude and longitude)","examples":[{"value":"50.586825 6.408977"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["geographic","location"],"string_serialization":"{float} {float}","slot_uri":"MIXS:0000009","alias":"lat_lon","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^(-?((?:[0-8]?[0-9](?:\\\\.\\\\d{0,8})?)|90)) -?[0-9]+(?:\\\\.[0-9]{0,8})?$|^-?(1[0-7]{1,2})$","structured_pattern":{"syntax":"^{lat} {lon}$","interpolated":true,"partial_match":true}},"elev":{"name":"elev","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth\'s surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit","title":"elevation","examples":[{"value":"100 meter"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["elevation"],"slot_uri":"MIXS:0000093","alias":"elev","owner":"MimsSedimentAncient","domain_of":["Mims","HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"env_broad_scale":{"name":"env_broad_scale","description":"Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS","title":"broad-scale environmental context","examples":[{"value":"rangeland biome [ENVO:01000247]"}],"in_subset":["environment"],"from_schema":"https://w3id.org/mixs","keywords":["context","environmental"],"slot_uri":"MIXS:0000012","alias":"env_broad_scale","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","required":true,"pattern":"^([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+) \\\\[[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+\\\\]$","structured_pattern":{"syntax":"^{termLabel} \\\\[{termID}\\\\]$","interpolated":true,"partial_match":true}},"tax_class":{"name":"tax_class","description":"Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes","title":"taxonomic classification","examples":[{"value":"vConTACT vContact2 (references from NCBI RefSeq v83, genus rank classification, default parameters)"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["classification","taxon"],"slot_uri":"MIXS:0000064","alias":"tax_class","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"experimental_factor":{"name":"experimental_factor","annotations":{"Expected_value":{"tag":"Expected_value","value":"text or EFO and/or OBI"}},"description":"Variable aspects of an experiment design that can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI)","title":"experimental factor","examples":[{"value":"time series design [EFO:0001779]"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["experimental","factor"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:0000008","alias":"experimental_factor","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true,"pattern":"^\\\\S+.*\\\\S+ \\\\[[a-zA-Z]{2,}:\\\\d+\\\\]$"},"associated_resource":{"name":"associated_resource","annotations":{"Expected_value":{"tag":"Expected_value","value":"reference to resource"}},"description":"A related resource that is referenced, cited, or otherwise associated to the sequence","title":"relevant electronic resources","examples":[{"value":"http://www.earthmicrobiome.org/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["resource"],"string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000091","alias":"associated_resource","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true},"sop":{"name":"sop","annotations":{"Expected_value":{"tag":"Expected_value","value":"reference to SOP"}},"description":"Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences","title":"relevant standard operating procedures","examples":[{"value":"http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/its/"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["procedures"],"string_serialization":"{PMID}|{DOI}|{URL}","slot_uri":"MIXS:0000090","alias":"sop","owner":"MimsSedimentAncient","domain_of":["Mims","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","recommended":true,"multivalued":true},"alkalinity":{"name":"alkalinity","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milliequivalent per liter, milligram per liter"}},"description":"Alkalinity, the ability of a solution to neutralize acids to the equivalence point of carbonate or bicarbonate","title":"alkalinity","examples":[{"value":"50 milligram per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["alkalinity"],"slot_uri":"MIXS:0000421","alias":"alkalinity","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"alkyl_diethers":{"name":"alkyl_diethers","description":"Concentration of alkyl diethers","title":"alkyl diethers","examples":[{"value":"0.005 mole per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000490","alias":"alkyl_diethers","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"aminopept_act":{"name":"aminopept_act","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"mole per liter per hour"}},"description":"Measurement of aminopeptidase activity","title":"aminopeptidase activity","examples":[{"value":"0.269 mole per liter per hour"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000172","alias":"aminopept_act","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"ammonium":{"name":"ammonium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of ammonium in the sample","title":"ammonium","examples":[{"value":"1.5 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000427","alias":"ammonium","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"bacteria_carb_prod":{"name":"bacteria_carb_prod","description":"Measurement of bacterial carbon production","title":"bacterial carbon production","examples":[{"value":"2.53 microgram per liter per hour"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","production"],"slot_uri":"MIXS:0000173","alias":"bacteria_carb_prod","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"biomass":{"name":"biomass","annotations":{"Expected_value":{"tag":"Expected_value","value":"biomass type;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"ton, kilogram, gram"}},"description":"Amount of biomass; should include the name for the part of biomass measured, e.g. Microbial, total. Can include multiple measurements","title":"biomass","examples":[{"value":"total;20 gram"}],"from_schema":"https://w3id.org/mixs","keywords":["biomass"],"string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000174","alias":"biomass","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"bishomohopanol":{"name":"bishomohopanol","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"microgram per liter, microgram per gram"}},"description":"Concentration of bishomohopanol","title":"bishomohopanol","examples":[{"value":"14 microgram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000175","alias":"bishomohopanol","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"bromide":{"name":"bromide","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"parts per million"}},"description":"Concentration of bromide","title":"bromide","examples":[{"value":"0.05 parts per million"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000176","alias":"bromide","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"calcium":{"name":"calcium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, micromole per liter, parts per million"}},"description":"Concentration of calcium in the sample","title":"calcium","examples":[{"value":"0.2 micromole per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000432","alias":"calcium","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"carb_nitro_ratio":{"name":"carb_nitro_ratio","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement value"}},"description":"Ratio of amount or concentrations of carbon to nitrogen. Used for quality control value in proteinaceous samples for radiocarbon dating.","title":"carbon/nitrogen ratio","examples":[{"value":"0.417361111"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","nitrogen","ratio"],"string_serialization":"{float}:{float}","slot_uri":"MIXS:0000310","alias":"carb_nitro_ratio","owner":"MimsSedimentAncient","domain_of":["Sediment","RadiocarbonDating","MimsSediment"],"range":"float","required":false,"recommended":true,"multivalued":false},"chem_administration":{"name":"chem_administration","annotations":{"Expected_value":{"tag":"Expected_value","value":"CHEBI;timestamp"}},"description":"List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi","title":"chemical administration","examples":[{"value":"agar [CHEBI:2509];2018-05-11T20:00Z"}],"from_schema":"https://w3id.org/mixs","keywords":["administration"],"string_serialization":"{termLabel} [{termID}];{timestamp}","slot_uri":"MIXS:0000751","alias":"chem_administration","owner":"MimsSedimentAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"chloride":{"name":"chloride","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, parts per million"}},"description":"Concentration of chloride in the sample","title":"chloride","examples":[{"value":"5000 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000429","alias":"chloride","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"chlorophyll":{"name":"chlorophyll","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per cubic meter, microgram per liter"}},"description":"Concentration of chlorophyll","title":"chlorophyll","examples":[{"value":"5 milligram per cubic meter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000177","alias":"chlorophyll","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"density":{"name":"density","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"gram per cubic meter, gram per cubic centimeter"}},"description":"Density of the sample, which is its mass per unit volume (aka volumetric mass density)","title":"density","examples":[{"value":"1000 kilogram per cubic meter"}],"from_schema":"https://w3id.org/mixs","keywords":["density"],"slot_uri":"MIXS:0000435","alias":"density","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diether_lipids":{"name":"diether_lipids","annotations":{"Expected_value":{"tag":"Expected_value","value":"diether lipid name;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"nanogram per liter"}},"description":"Concentration of diether lipids; can include multiple types of diether lipids","title":"diether lipids","examples":[{"value":"0.2 nanogram per liter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000178","alias":"diether_lipids","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"diss_carb_dioxide":{"name":"diss_carb_dioxide","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter"}},"description":"Concentration of dissolved carbon dioxide in the sample or liquid portion of the sample","title":"dissolved carbon dioxide","examples":[{"value":"5 milligram per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","dissolved"],"slot_uri":"MIXS:0000436","alias":"diss_carb_dioxide","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_hydrogen":{"name":"diss_hydrogen","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of dissolved hydrogen","title":"dissolved hydrogen","examples":[{"value":"0.3 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["dissolved"],"slot_uri":"MIXS:0000179","alias":"diss_hydrogen","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_inorg_carb":{"name":"diss_inorg_carb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"microgram per liter, milligram per liter, parts per million"}},"description":"Dissolved inorganic carbon concentration in the sample, typically measured after filtering the sample using a 0.45 micrometer filter","title":"dissolved inorganic carbon","examples":[{"value":"2059 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","dissolved","inorganic"],"slot_uri":"MIXS:0000434","alias":"diss_inorg_carb","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_org_carb":{"name":"diss_org_carb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter"}},"description":"Concentration of dissolved organic carbon in the sample, liquid portion of the sample, or aqueous phase of the fluid","title":"dissolved organic carbon","examples":[{"value":"197 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","dissolved","organic"],"slot_uri":"MIXS:0000433","alias":"diss_org_carb","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_org_nitro":{"name":"diss_org_nitro","description":"Dissolved organic nitrogen concentration measured as; total dissolved nitrogen - NH4 - NO3 - NO2","title":"dissolved organic nitrogen","examples":[{"value":"0.05 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["dissolved","nitrogen","organic"],"slot_uri":"MIXS:0000162","alias":"diss_org_nitro","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"diss_oxygen":{"name":"diss_oxygen","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per kilogram, milligram per liter"}},"description":"Concentration of dissolved oxygen","title":"dissolved oxygen","examples":[{"value":"175 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","keywords":["dissolved","oxygen"],"slot_uri":"MIXS:0000119","alias":"diss_oxygen","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"glucosidase_act":{"name":"glucosidase_act","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"mol per liter per hour"}},"description":"Measurement of glucosidase activity","title":"glucosidase activity","examples":[{"value":"5 mol per liter per hour"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000137","alias":"glucosidase_act","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"magnesium":{"name":"magnesium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"mole per liter, milligram per liter, parts per million, micromole per kilogram"}},"description":"Concentration of magnesium in the sample","title":"magnesium","examples":[{"value":"52.8 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000431","alias":"magnesium","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"mean_frict_vel":{"name":"mean_frict_vel","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter per second"}},"description":"Measurement of mean friction velocity","title":"mean friction velocity","examples":[{"value":"0.5 meter per second"}],"from_schema":"https://w3id.org/mixs","keywords":["mean","velocity"],"slot_uri":"MIXS:0000498","alias":"mean_frict_vel","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"mean_peak_frict_vel":{"name":"mean_peak_frict_vel","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter per second"}},"description":"Measurement of mean peak friction velocity","title":"mean peak friction velocity","examples":[{"value":"1 meter per second"}],"from_schema":"https://w3id.org/mixs","keywords":["mean","peak","velocity"],"slot_uri":"MIXS:0000502","alias":"mean_peak_frict_vel","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"methane":{"name":"methane","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, parts per billion, parts per million"}},"description":"Methane (gas) amount or concentration at the time of sampling","title":"methane","examples":[{"value":"0.15 micromole per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000101","alias":"methane","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"misc_param":{"name":"misc_param","annotations":{"Expected_value":{"tag":"Expected_value","value":"parameter name;measurement value"}},"description":"Any other measurement performed or parameter collected, that is not listed here","title":"miscellaneous parameter","examples":[{"value":"Bicarbonate ion concentration;2075 micromole per kilogram"}],"from_schema":"https://w3id.org/mixs","keywords":["parameter"],"string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000752","alias":"misc_param","owner":"MimsSedimentAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"n_alkanes":{"name":"n_alkanes","annotations":{"Expected_value":{"tag":"Expected_value","value":"n-alkane name;measurement value"}},"description":"Concentration of n-alkanes; can include multiple n-alkanes","title":"n-alkanes","examples":[{"value":"n-hexadecane;100 milligram per liter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000503","alias":"n_alkanes","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"nitrate":{"name":"nitrate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of nitrate in the sample","title":"nitrate","examples":[{"value":"65 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrate"],"slot_uri":"MIXS:0000425","alias":"nitrate","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"nitrite":{"name":"nitrite","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of nitrite in the sample","title":"nitrite","examples":[{"value":"0.5 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrite"],"slot_uri":"MIXS:0000426","alias":"nitrite","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"nitro":{"name":"nitro","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of nitrogen (total)","title":"nitrogen","examples":[{"value":"4.2 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrogen"],"slot_uri":"MIXS:0000504","alias":"nitro","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"org_carb":{"name":"org_carb","description":"Concentration of organic carbon","title":"organic carbon","examples":[{"value":"1.5 microgram per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","organic"],"slot_uri":"MIXS:0000508","alias":"org_carb","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"org_matter":{"name":"org_matter","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"microgram per liter"}},"description":"Concentration of organic matter","title":"organic matter","examples":[{"value":"1.75 milligram per cubic meter"}],"from_schema":"https://w3id.org/mixs","keywords":["organic"],"slot_uri":"MIXS:0000204","alias":"org_matter","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"org_nitro":{"name":"org_nitro","description":"Concentration of organic nitrogen","title":"organic nitrogen","examples":[{"value":"4 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["nitrogen","organic"],"slot_uri":"MIXS:0000205","alias":"org_nitro","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"organism_count":{"name":"organism_count","annotations":{"Expected_value":{"tag":"Expected_value","value":"organism name;measurement value;enumeration"}},"description":"Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)","title":"organism count","examples":[{"value":"total prokaryotes;3.5e7 cells per milliliter;qPCR"}],"from_schema":"https://w3id.org/mixs","keywords":["count","organism"],"string_serialization":"{text};{float} {unit};[ATP|MPN|qPCR|other]","slot_uri":"MIXS:0000103","alias":"organism_count","owner":"MimsSedimentAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"oxy_stat_samp":{"name":"oxy_stat_samp","description":"Oxygenation status of sample","title":"oxygenation status of sample","examples":[{"value":"aerobic"}],"from_schema":"https://w3id.org/mixs","keywords":["oxygen","sample","status"],"slot_uri":"MIXS:0000753","alias":"oxy_stat_samp","owner":"MimsSedimentAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"OXY_STAT_SAMP_ENUM"},"ph":{"name":"ph","description":"Ph measurement of the sample, or liquid portion of sample, or aqueous phase of the fluid","title":"pH","examples":[{"value":"7.2"}],"from_schema":"https://w3id.org/mixs","keywords":["ph"],"slot_uri":"MIXS:0001001","alias":"ph","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"float"},"particle_class":{"name":"particle_class","annotations":{"Expected_value":{"tag":"Expected_value","value":"particle name;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"micrometer"}},"description":"Particles are classified, based on their size, into six general categories:clay, silt, sand, gravel, cobbles, and boulders; should include amount of particle preceded by the name of the particle type; can include multiple values","title":"particle classification","from_schema":"https://w3id.org/mixs","keywords":["classification","particle"],"string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000206","alias":"particle_class","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"part_org_carb":{"name":"part_org_carb","description":"Concentration of particulate organic carbon","title":"particulate organic carbon","examples":[{"value":"1.92 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","organic","particle","particulate"],"slot_uri":"MIXS:0000515","alias":"part_org_carb","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"perturbation":{"name":"perturbation","annotations":{"Expected_value":{"tag":"Expected_value","value":"perturbation type name;perturbation interval and duration"}},"description":"Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types","title":"perturbation","examples":[{"value":"antibiotic addition;R2/2018-05-11T14:30Z/2018-05-11T19:30Z/P1H30M"}],"from_schema":"https://w3id.org/mixs","keywords":["perturbation"],"string_serialization":"{text};{Rn/start_time/end_time/duration}","slot_uri":"MIXS:0000754","alias":"perturbation","owner":"MimsSedimentAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"petroleum_hydrocarb":{"name":"petroleum_hydrocarb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of petroleum hydrocarbon","title":"petroleum hydrocarbon","examples":[{"value":"0.05 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["hydrocarbon","petroleum"],"slot_uri":"MIXS:0000516","alias":"petroleum_hydrocarb","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"phaeopigments":{"name":"phaeopigments","annotations":{"Expected_value":{"tag":"Expected_value","value":"phaeopigment name;measurement value"},"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per cubic meter"}},"description":"Concentration of phaeopigments; can include multiple phaeopigments","title":"phaeopigments","examples":[{"value":"2.5 milligram per cubic meter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000180","alias":"phaeopigments","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"phosphate":{"name":"phosphate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of phosphate","title":"phosphate","examples":[{"value":"0.7 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["phosphate"],"slot_uri":"MIXS:0000505","alias":"phosphate","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"phosplipid_fatt_acid":{"name":"phosplipid_fatt_acid","annotations":{"Expected_value":{"tag":"Expected_value","value":"phospholipid fatty acid name;measurement value"}},"description":"Concentration of phospholipid fatty acids; can include multiple values","title":"phospholipid fatty acid","examples":[{"value":"2.98 milligram per liter"}],"from_schema":"https://w3id.org/mixs","string_serialization":"{text};{float} {unit}","slot_uri":"MIXS:0000181","alias":"phosplipid_fatt_acid","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","multivalued":true},"porosity":{"name":"porosity","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement value or range"},"Preferred_unit":{"tag":"Preferred_unit","value":"percentage"}},"description":"Porosity of deposited sediment is volume of voids divided by the total volume of sample","title":"porosity","from_schema":"https://w3id.org/mixs","keywords":["porosity"],"string_serialization":"{float} - {float} {unit}","slot_uri":"MIXS:0000211","alias":"porosity","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string"},"potassium":{"name":"potassium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, parts per million"}},"description":"Concentration of potassium in the sample","title":"potassium","examples":[{"value":"463 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000430","alias":"potassium","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"pressure":{"name":"pressure","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"atmosphere"}},"description":"Pressure to which the sample is subject to, in atmospheres","title":"pressure","examples":[{"value":"50 atmosphere"}],"from_schema":"https://w3id.org/mixs","keywords":["pressure"],"slot_uri":"MIXS:0000412","alias":"pressure","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"redox_potential":{"name":"redox_potential","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"millivolt"}},"description":"Redox potential, measured relative to a hydrogen cell, indicating oxidation or reduction potential","title":"redox potential","examples":[{"value":"300 millivolt"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000182","alias":"redox_potential","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"salinity":{"name":"salinity","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"practical salinity unit, percentage"}},"description":"The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater","title":"salinity","examples":[{"value":"25 practical salinity unit"}],"from_schema":"https://w3id.org/mixs","keywords":["salinity"],"slot_uri":"MIXS:0000183","alias":"salinity","owner":"MimsSedimentAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"samp_store_dur":{"name":"samp_store_dur","description":"Duration for which the sample was stored. Indicate the duration for which the sample was stored written in ISO 8601 format","title":"sample storage duration","examples":[{"value":"P1Y6M"}],"from_schema":"https://w3id.org/mixs","keywords":["duration","period","sample","storage"],"slot_uri":"MIXS:0000116","alias":"samp_store_dur","owner":"MimsSedimentAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^P(?:(?:\\\\d+D|\\\\d+M(?:\\\\d+D)?|\\\\d+Y(?:\\\\d+M(?:\\\\d+D)?)?)(?:T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S))?|T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S)|\\\\d+W)$","structured_pattern":{"syntax":"^{duration}$","interpolated":true,"partial_match":true}},"samp_store_loc":{"name":"samp_store_loc","annotations":{"Expected_value":{"tag":"Expected_value","value":"location name"}},"description":"Location at which sample was stored, usually name of a specific freezer/room","title":"sample storage location","examples":[{"value":"Freezer no:5"}],"from_schema":"https://w3id.org/mixs","keywords":["location","sample","storage"],"string_serialization":"{text}","slot_uri":"MIXS:0000755","alias":"samp_store_loc","owner":"MimsSedimentAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string"},"samp_store_temp":{"name":"samp_store_temp","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"degree Celsius"}},"description":"Temperature at which sample was stored, e.g. -80 degree Celsius","title":"sample storage temperature","examples":[{"value":"-80 degree Celsius"}],"from_schema":"https://w3id.org/mixs","keywords":["sample","storage","temperature"],"slot_uri":"MIXS:0000110","alias":"samp_store_temp","owner":"MimsSedimentAncient","domain_of":["HostAssociated","Sediment","MimsHostAssociated","MimsSediment","MimsHostAssociatedAncient"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sediment_type":{"name":"sediment_type","description":"Information about the sediment type based on major constituents","title":"sediment type","examples":[{"value":"biogenous"}],"from_schema":"https://w3id.org/mixs","keywords":["sediment","type"],"slot_uri":"MIXS:0001078","alias":"sediment_type","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"SEDIMENT_TYPE_ENUM"},"silicate":{"name":"silicate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter"}},"description":"Concentration of silicate","title":"silicate","examples":[{"value":"0.05 micromole per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000184","alias":"silicate","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sodium":{"name":"sodium","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"milligram per liter, parts per million"}},"description":"Sodium concentration in the sample","title":"sodium","examples":[{"value":"10.5 milligram per liter"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000428","alias":"sodium","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sulfate":{"name":"sulfate","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of sulfate in the sample","title":"sulfate","examples":[{"value":"5 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["sulfate"],"slot_uri":"MIXS:0000423","alias":"sulfate","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"sulfide":{"name":"sulfide","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"micromole per liter, milligram per liter, parts per million"}},"description":"Concentration of sulfide in the sample","title":"sulfide","examples":[{"value":"2 micromole per liter"}],"from_schema":"https://w3id.org/mixs","keywords":["sulfide"],"slot_uri":"MIXS:0000424","alias":"sulfide","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tidal_stage":{"name":"tidal_stage","description":"Stage of tide","title":"tidal stage","examples":[{"value":"high tide"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000750","alias":"tidal_stage","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"TIDAL_STAGE_ENUM"},"tot_carb":{"name":"tot_carb","description":"Total carbon content","title":"total carbon","from_schema":"https://w3id.org/mixs","keywords":["carbon","total"],"slot_uri":"MIXS:0000525","alias":"tot_carb","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tot_depth_water_col":{"name":"tot_depth_water_col","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"meter"}},"description":"Measurement of total depth of water column","title":"total depth of water column","examples":[{"value":"500 meter"}],"from_schema":"https://w3id.org/mixs","keywords":["depth","total","water"],"slot_uri":"MIXS:0000634","alias":"tot_depth_water_col","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tot_nitro_content":{"name":"tot_nitro_content","description":"Total nitrogen content of the sample","title":"total nitrogen content","examples":[{"value":"35 milligrams Nitrogen per kilogram of soil"}],"from_schema":"https://w3id.org/mixs","keywords":["content","nitrogen","total"],"slot_uri":"MIXS:0000530","alias":"tot_nitro_content","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"tot_org_carb":{"name":"tot_org_carb","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"gram Carbon per kilogram sample material"}},"description":"Total organic carbon content","title":"total organic carbon","examples":[{"value":"0.02"}],"from_schema":"https://w3id.org/mixs","keywords":["carbon","organic","total"],"slot_uri":"MIXS:0000533","alias":"tot_org_carb","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"turbidity":{"name":"turbidity","description":"Measure of the amount of cloudiness or haziness in water caused by individual particles","title":"turbidity","examples":[{"value":"0.3 nephelometric turbidity units"}],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:0000191","alias":"turbidity","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"water_content":{"name":"water_content","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"gram per gram or cubic centimeter per cubic centimeter"}},"description":"Water content measurement","title":"water content","from_schema":"https://w3id.org/mixs","keywords":["content","water"],"string_serialization":"{float} {unit}","slot_uri":"MIXS:0000185","alias":"water_content","owner":"MimsSedimentAncient","domain_of":["Sediment","MimsSediment"],"range":"string","pattern":"^[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?( *- *[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)? *([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)$","structured_pattern":{"syntax":"^{scientific_float}( *- *{scientific_float})? *{text}$","interpolated":true,"partial_match":true}},"collection_context_name":{"name":"collection_context_name","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of institution"}},"description":"Name of where sample originated and is typically stored. Typically will be \'owning institution\'.","title":"context name where location was collected","examples":[{"value":"Natural History Museum London"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["storage"],"slot_uri":"MIXS:999999901","alias":"collection_context_name","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"cultural_era":{"name":"cultural_era","annotations":{"Expected_value":{"tag":"Expected_value","value":"chronotology or PeriodO term; text"}},"description":"The cultural era approximating the period in which the individual lived from https://chronontology.dainst.org/ or PeriodO if possible. If not available, follow INSDC standard for reporting missing data (https://www.insdc.org/submitting-standards/missing-value-reporting/).","title":"cultural era or period","examples":[{"value":"Copper Age [Chronotology: NW6hofAScJSE]"},{"value":"Upper Middle Palaeolthic"},{"value":"Not collected"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient","age"],"string_serialization":"{termLabel} [{termID}]|{text}","slot_uri":"MIXS:999999902","alias":"cultural_era","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":true,"recommended":true,"multivalued":false},"damage_treatment":{"name":"damage_treatment","annotations":{"Expected_value":{"tag":"Expected_value","value":"enumeration"}},"description":"Indication of whether characteristic ancient DNA damage has been  altered or removed from a DNA extract in a laboratory","title":"Damage treatment type","examples":[{"value":"none"},{"value":"partial-udg"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"slot_uri":"MIXS:999999903","alias":"damage_treatment","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"DamageTreatmentEnum","required":true,"multivalued":true},"dna_extraction_date":{"name":"dna_extraction_date","description":"The date when the nucleic acid extraction was started from the sample material. In case no exact time is available, the date can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant","title":"date of extraction of nucleic acids from sample","examples":[{"value":"2023-12-01"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999904","alias":"dna_extraction_date","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"datetime","required":false,"recommended":true},"ethical_authority":{"name":"ethical_authority","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of authority providing ethical approval"}},"description":"Name of the authorit(ies) or institution(s) that awarded sampling and  analysis (e.g. human remains) and/or export permission  (e.g. animal remains)","title":"ethical authority","examples":[{"value":"University of Copenhagen"},{"value":"Federal Foreign Office (Germany)"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["ethics","location"],"slot_uri":"MIXS:999999905","alias":"ethical_authority","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"ethical_date":{"name":"ethical_date","description":"Date of award of ethical/export permission. The date can be right truncated  i.e. all of these are valid times: 2008-01-23; 2008-01; 2008;  Except: 2008-01; 2008 all are ISO8601 compliant","title":"date of ethical approval","examples":[{"value":"2023-12-01"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999906","alias":"ethical_date","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"datetime","required":false,"recommended":true},"ethical_id":{"name":"ethical_id","description":"The permissions code, ID, document, or similar provided by any authority  (socal, legal, academic etc.) associated with approval of the analysis of  this particular sample.","title":"ethical permit or approval ID","examples":[{"value":"DE-123-JK"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"slot_uri":"MIXS:999999906","alias":"ethical_id","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"experimental_procedures":{"name":"experimental_procedures","description":"Provide a DOI or URL to refer to the paper where the field report, nucleic acid extraction,  library construction, and other procedures are explained in more detail, e.g. the paper reporting the data.","title":"experimental procedures","examples":[{"value":"doi:10.1093/nar/gkr771"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999907","alias":"experimental_procedures","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_concentration":{"name":"lib_concentration","annotations":{"Expected_value":{"tag":"Expected_value","value":"measurement value in number of ng per l"}},"description":"Concentration of library in ng per l, as inferred by qPCR, digital PCR, Qubit etc.","title":"library concentration","examples":[{"value":"2"},{"value":"0.5"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","string_serialization":"{float}","slot_uri":"MIXS:999999908","alias":"lib_concentration","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"lib_index_polymerase":{"name":"lib_index_polymerase","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of polymerase used during library indexing"}},"description":"The DNA polymerase enzyme used for indexing DNA libraries","title":"library index polymerase","examples":[{"value":"Agilent PfuTurbo Cx HotStart"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["polymerase","library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999909","alias":"lib_index_polymerase","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true},"lib_preparation_protocol":{"name":"lib_preparation_protocol","description":"Citation(s) for the DNA library preparation protocol","title":"library preparation protocols","examples":[{"value":"doi:10.1093/nar/gkr771"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999907","alias":"lib_preparation_protocol","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"lib_reamplification_polymerase":{"name":"lib_reamplification_polymerase","annotations":{"Expected_value":{"tag":"Expected_value","value":"name of polymerase used during library indexing"}},"description":"The name of the DNA polymerase enzyme used for reamplifying DNA libraries post-indexing.","title":"library reamplification polymerase","examples":[{"value":"KAPA HiFi HotStart Uracil+"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["polymerase","library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999909","alias":"lib_reamplification_polymerase","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true},"lib_type":{"name":"lib_type","description":"The type of library created, i.e., amplicon, enriched, or shotgun.  Amplicon based is a library that has been amplified to target a specific single region of a genome (e.g. a specific gene).  An enriched library is a library has had a particular genome or genome regions \'captured\' or enriched typically via baits or probes. A shotgun library has had no type of targeted enrichment for a particular gene or genome.","title":"library type","examples":[{"value":"amplicon"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"slot_uri":"MIXS:999999910","alias":"lib_type","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"LibTypeEnum","required":false},"neg_cont_status":{"name":"neg_cont_status","description":"Specify whether the sample is a negative control or not.","title":"negative control status","examples":[{"value":"true"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["control"],"slot_uri":"MIXS:999999911","alias":"neg_cont_status","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"boolean"},"num_capture_amp_cycles":{"name":"num_capture_amp_cycles","description":"Number of amplification cycles after capture enrichment","title":"number of reamplification cycles after capture","examples":[{"value":"12"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999912","alias":"num_capture_amp_cycles","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer","multivalued":true},"num_reamp_cycles":{"name":"num_reamp_cycles","description":"Number of amplification cycles after library indexing PCR. If capture data, this refers to amplifications prior to the capture experiments.","title":"number of reamplification cycles","examples":[{"value":"12"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999913","alias":"num_reamp_cycles","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer"},"preservational_treatment":{"name":"preservational_treatment","description":"Description of any treatment applied to samples for the purpose of  maximising collection preservation that may influence downstream  DNA recovery or library construction, such as storage fluid or  reconstructive glue","title":"preservational treatment","examples":[{"value":"stored in formalin"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999914","alias":"preservational_treatment","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"prev_pubs":{"name":"prev_pubs","description":"Any publications that report data from the same body/skeleton/individual","title":"previous publications","examples":[{"value":"doi:10.1016/j.jas.2015.02.0181"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999915","alias":"prev_pubs","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"recovery_date":{"name":"recovery_date","annotations":{"Preferred_unit":{"tag":"Preferred_unit","value":"year"}},"description":"Date of excavation or retrieval from burial or depositional context, if known. In case no exact time is available, the date can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant. If excavations were done during a longer period, report its midpoint at a month or year level.","title":"date of retrieval from depositional context","examples":[{"value":"2023"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:9999999916","alias":"recovery_date","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"datetime","required":false,"recommended":true},"samp_decontam_pretreat":{"name":"samp_decontam_pretreat","description":"Method(s) employed for surface decontamination of samples of external  modern DNA; Treatment used on the samples. Depends on the sample type.  More relevant for bones than environmental samples. E.g. buffers, EDTA, etc.","title":"sample decontamination pretreatment","examples":[{"value":"doi:10.1016/j.jas.2015.02.0181"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999917","alias":"samp_decontam_pretreat","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true,"pattern":"^^PMID:\\\\d+$|^doi:10.\\\\d{2,9}/.*$|^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$$","structured_pattern":{"syntax":"^{PMID}|{DOI}|{URL}$","interpolated":true,"partial_match":true}},"samp_age_range_youngest_limit":{"name":"samp_age_range_youngest_limit","annotations":{"Expected_value":{"tag":"Expected_value","value":"age value corresponding to unit"}},"description":"The youngest possible age of the sample. Youngest refers to the time closest to present day. If a sample has only a single age value, the value from samp_age_range_oldest_limit can be repeated. Sample age refers to a period of time in the past, such as when an organism was living and then died, not the biological age.","title":"sample age range youngest limit","examples":[{"value":"100000"},{"value":"1700"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999918","alias":"samp_age_range_youngest_limit","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer","required":true,"recommended":true},"samp_age_range_youngest_unit":{"name":"samp_age_range_youngest_unit","description":"The unit of the youngest age in the sample age range.","title":"sample age range youngest unit","examples":[{"value":"cal BP"},{"value":"CE"},{"value":"Ma"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999919","alias":"samp_age_range_youngest_unit","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"SampAgeUnit","required":true,"recommended":true,"multivalued":true},"samp_age_range_oldest_limit":{"name":"samp_age_range_oldest_limit","annotations":{"Expected_value":{"tag":"Expected_value","value":"age value corresponding to unit"}},"description":"The oldest possible age of the sample. Oldest refers to the time furthest from present day. If a sample has only a single age value, the value from samp_age_range_youngest_limit can be repeated. Sample age refers to a period of time in the past, such as when an organism was living and then died, not the biological age.","title":"sample age range oldest limit","examples":[{"value":"120000"},{"value":"1900"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999920","alias":"samp_age_range_oldest_limit","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"integer","required":true,"recommended":true},"samp_age_range_oldest_unit":{"name":"samp_age_range_oldest_unit","description":"The unit of the oldest age in sample age range.","title":"sample age range oldest unit","examples":[{"value":"cal BP"},{"value":"CE"},{"value":"Ma"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999921","alias":"samp_age_range_oldest_unit","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"SampAgeUnit","required":true,"recommended":true,"multivalued":true},"samp_age_range_inference_methods":{"name":"samp_age_range_inference_methods","description":"The method used to infer the sample age. This corresponds to a fixed list of most commonly used dating methods, such as 14C, OSL, stratigraphic inference, historical records, other etc. If the method is not in the list, please provide a description in samp_age_range_inference_description.","title":"sample age inference methods","examples":[{"value":"radiocarbon dating, calibrated with OxCal v4.3"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999922","alias":"samp_age_range_inference_methods","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"SampAgeInferMethod","required":true,"recommended":true,"multivalued":true},"samp_age_range_inference_description":{"name":"samp_age_range_inference_description","description":"Additional general information about the inference method used to infer the sample age that can be useful for  understanding how the sample age was estimated. This can be things such as whether radiocarbon age estimated on AMS or conventional radiocarbon dating, or which calibration confidence interval was used.  It can also be used to describe what artefacts were used to age the stratigraphic layer of historical contexts.","title":"sample age inference description","examples":[{"value":"radiocarbon dating, calibrated with OxCal v4.3 with 95% confidence interval"},{"value":"based on proxy dating from other bone samples of stratigraphic layer"},{"value":"a coin found in the burial was from the 3rd century"},{"value":"age taken from previous publication Doe et al. 2019"},{"value":"radiocarbon age ID: OxA-12345"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","slot_uri":"MIXS:999999923","alias":"samp_age_range_inference_description","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":false},"samp_alt_lab_ids":{"name":"samp_alt_lab_ids","description":"An alternative sample or material ID related to the sample not already covered by terms samp_name and source_mat_id, including from associated other non-genetic analyses of the same sample (e.g. museum IDs, internal lab sample ID from sampling such a bone drilling). Can be specified multiple times for different ID contexts.","title":"alternative sample IDs","examples":[{"value":"ABC_24"}],"in_subset":["investigation"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:999999924","alias":"samp_alt_lab_ids","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","multivalued":true},"storage_conditions":{"name":"storage_conditions","description":"General conditions in which the material (before sampling for DNA analysis) was stored in long-term collection storage, that may influenced DNA recovery or library construction. For example, specify temperature, humidity, presence of microbial overgrowth etc..","title":"storage conditions","examples":[{"value":"climate-controlled"},{"value":"Stored in the museum from recovery_date to collection_date"},{"value":"Stored at -20oC until dna_extraction_date"}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","string_serialization":"{text}","slot_uri":"MIXS:999999925","alias":"storage_conditions","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string"},"palaeopath_status":{"name":"palaeopath_status","annotations":{"Expected_value":{"tag":"Expected_value","value":"description of health related observation on ancient remains"}},"description":"Describe briefly any relevant palaeopathological or health-related observations of the remains of the individual under study.","title":"palaeopathology status.","examples":[{"value":"Osteoporosis."},{"value":"Parasites found in pelvic area."},{"value":"Caries on right upper molar."}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["palaeopathology","host health","ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999926","alias":"palaeopath_status","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":false},"biocultural_label":{"name":"biocultural_label","annotations":{"Expected_value":{"tag":"Expected_value","value":"Relevant biocultural label from https://localcontexts.org/labels/biocultural-labels/"}},"description":"Relevant biocultural labels that apply to the sample under study, based on https://localcontexts.org/label/bc-provenance/. These help indicate how indigenous peoples and similar communities view in which contexts this data can be reused.","title":"biocultural label","examples":[{"value":"BC R"},{"value":"BC MC"}],"in_subset":["ethics"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"slot_uri":"MIXS:999999927","alias":"biocultural_label","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"BioCulturalLabel","required":false,"recommended":false,"multivalued":true},"ethics_perm_scope":{"name":"ethics_perm_scope","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the scope of ethical permissions for data use"}},"description":"Description of the original scope and permissions of the research on the genetic material, as was approved by the ethical authority (e.g. bacteria only, DNA only, bacteria and human, no host read analysis allowed).","title":"ethical Permissions & Scope","examples":[{"value":"Defined scope only includes the study of bacterial sequences and any human sequence is not covered under the agreement."}],"in_subset":["ethics"],"from_schema":"https://w3id.org/mixs","keywords":["ethics"],"string_serialization":"{text}","slot_uri":"MIXS:999999928","alias":"ethics_perm_scope","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true},"host_body_preservation_state":{"name":"host_body_preservation_state","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the preservation of the sampled (ancient) organism/host"}},"description":"Description of the state of the sampled (ancient) organism/host as originally preserved in the burial environment, e.g. if the the sample was artificially or naturally (fully/partially) mummified at the time of death.","title":"ancient body preservation state","examples":[{"value":"complete artificial mummification."},{"value":"natural partial mummification."},{"value":"fully skeletonised."}],"in_subset":["nucleic acid sequence source"],"from_schema":"https://w3id.org/mixs","keywords":["ancient"],"string_serialization":"{text}","slot_uri":"MIXS:999999929","alias":"host_body_preservation_state","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":false},"lib_strandedness":{"name":"lib_strandedness","annotations":{"Expected_value":{"tag":"Expected_value","value":"DNA library strandedness"}},"description":"The strandedness of the original template DNA molecules used for constructing the sequencing library","title":"DNA strandedness in library creation","examples":[{"value":"single"},{"value":"double"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"slot_uri":"MIXS:999999930","alias":"lib_strandedness","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"LibStrand","required":false,"recommended":true,"multivalued":true},"enrichment_probe_set":{"name":"enrichment_probe_set","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of probe set used for target enrichment/library selection"}},"description":"Description of target enrichment probe designs used (e.g., species included, sequences, type, company). This can include custom kits (please provide a general description and DOI if available) or commercially available kit (provide ID and company).","title":"enrichment probe design description","examples":[{"value":"Custom probe design (70 bp biotinylated RNA probes) covering 500 full E. Coli genomes; company TE-Science"},{"value":"Commercial kit ABC001-EC from company TE-Science, purchased 2020"}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","enrichment"],"string_serialization":"{text}","slot_uri":"MIXS:999999931","alias":"enrichment_probe_set","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":false,"multivalued":true},"mid_config":{"name":"mid_config","annotations":{"Expected_value":{"tag":"Expected_value","value":"Description of the indexing configuration of the library"}},"description":"Index/barcode/primer configuration used during library building for sequencing. This includes information such as the number, type and location of indexes, the index/primer kit/list, or if \'inline\' barcodes or UMIs were ligated directly onto the template molecules.","title":"multiplex identifiers or indexing configuration of libraries","examples":[{"value":"dual index with single internal barcode."},{"value":"UDI index sequences."}],"in_subset":["sequencing"],"from_schema":"https://w3id.org/mixs","keywords":["library","preparation"],"string_serialization":"{text}","slot_uri":"MIXS:999999932","alias":"mid_config","owner":"MimsSedimentAncient","domain_of":["Ancient","MimsHostAssociatedAncient"],"range":"string","required":false,"recommended":true,"multivalued":true}},"class_uri":"MIXS:0010007_0016011_9999903"}},"source_file":"minas.yml","settings":{"country":{"setting_key":"country","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"name":{"setting_key":"name","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"parameters":{"setting_key":"parameters","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"region":{"setting_key":"region","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"room_name":{"setting_key":"room_name","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"software":{"setting_key":"software","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"specific_location":{"setting_key":"specific_location","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"storage_condition_type":{"setting_key":"storage_condition_type","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"termLabel":{"setting_key":"termLabel","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"text":{"setting_key":"text","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"unit":{"setting_key":"unit","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"version":{"setting_key":"version","setting_value":"([^\\\\s-]{1,2}|[^\\\\s-]+.+[^\\\\s-]+)"},"DOI":{"setting_key":"DOI","setting_value":"^doi:10.\\\\d{2,9}/.*$"},"NCBItaxon_id":{"setting_key":"NCBItaxon_id","setting_value":"NCBITaxon:\\\\d+"},"PMID":{"setting_key":"PMID","setting_value":"^PMID:\\\\d+$"},"URL":{"setting_key":"URL","setting_value":"^https?:\\\\/\\\\/(?:www\\\\.)?[-a-zA-Z0-9@:%._\\\\+~#=]{1,256}\\\\.[a-zA-Z0-9()]{1,6}\\\\b(?:[-a-zA-Z0-9()@:%_\\\\+.~#?&\\\\/=]*)$"},"adapter":{"setting_key":"adapter","setting_value":"[ACGTRKSYMWBHDVN]+"},"ambiguous_nucleotides":{"setting_key":"ambiguous_nucleotides","setting_value":"[ACGTRKSYMWBHDVN]+"},"duration":{"setting_key":"duration","setting_value":"P(?:(?:\\\\d+D|\\\\d+M(?:\\\\d+D)?|\\\\d+Y(?:\\\\d+M(?:\\\\d+D)?)?)(?:T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S))?|T(?:\\\\d+H(?:\\\\d+M(?:\\\\d+S)?)?|\\\\d+M(?:\\\\d+S)?|\\\\d+S)|\\\\d+W)"},"float":{"setting_key":"float","setting_value":"[-+]?[0-9]*\\\\.?[0-9]+"},"integer":{"setting_key":"integer","setting_value":"[1-9][0-9]*"},"lat":{"setting_key":"lat","setting_value":"(-?((?:[0-8]?[0-9](?:\\\\.\\\\d{0,8})?)|90))"},"lon":{"setting_key":"lon","setting_value":"-?[0-9]+(?:\\\\.[0-9]{0,8})?$|^-?(1[0-7]{1,2})"},"room_number":{"setting_key":"room_number","setting_value":"[1-9][0-9]*"},"scientific_float":{"setting_key":"scientific_float","setting_value":"[-+]?[0-9]*\\\\.?[0-9]+(?:[eE][-+]?[0-9]+)?"},"termID":{"setting_key":"termID","setting_value":"[a-zA-Z]{2,}:[a-zA-Z0-9]\\\\d+"}},"@type":"SchemaDefinition"}')}}]);