<!doctype html>

<html lang="en">
<head>
  <meta charset="utf-8">
  <title>Guidelines for Biosample Metadata Collection and Submission</title>
  <meta name="description" content="Guidelines for submitting metadata regarding biosamples and their associated cases to sequence repositories.">
  <meta name="author" content="Hsiao Public Health Bioinformatics, Vancouver, BC, Canada">
  <link rel="stylesheet" href="./doc.css">

</head>

<body>
	
	<h2>Guidelines for Biosample metadata collection and submission</h2>
	<hr size="2"/>
	<p>Guidelines for submitting metadata regarding biosamples and their associated cases to sequence repositories.</p>

	<table>
		<thead>
			<tr>
				<th class="label">Field</th>
				<th class="description">Description</th>
				<th class="guidance">Guidance</th>
				<th class="example">Examples</th>
				<th class="data_status">Data Status</th>
			</tr>
		</thead>
		<tbody>
			<!-- python search and replace does this part -->	
  			
						<tr class="section">
							<td colspan="5"><h3>Sample collection and processing</h3></td>
						</tr>
						
						<tr>
							<td class="label">sample_collector_sample_ID</td>
							<td>The user-defined identifier for the sample, as provided by the laboratory that collected the sample.</td>
							<td>The sample_ID should represent the identifier assigned to the sample at time of collection, for which all the descriptive information applies. If the original sample_ID is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>ABCD123</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">alternative_sample_ID</td>
							<td>An alternative sample_ID assigned to the sample by another organization.</td>
							<td>Provide any alternative identifiers for the sample here, separated by commas. Alternative identifiers assigned to the sample should be tracked along with original IDs to establish chain of custody. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>bvn5567, RA1-HH789</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">collected_by_institution_name</td>
							<td>The name of the agency, organization or institution with which the sample collector is affiliated.</td>
							<td>Provide the name of the agency, organization or institution that collected the sample in full (avoid abbreviations). If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>Public Health Agency of Canada</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">collected_by_laboratory_name</td>
							<td>The specific laboratory affiliation of the sample collector.</td>
							<td>Provide the name of the specific laboratory that collected the sample (avoid abbreviations). If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>Topp Lab</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sample_collection_project_name</td>
							<td>The name of the project/initiative/program for which the sample was collected.</td>
							<td>Provide the name of the project and/or the project ID here. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>Watershed Project (HA-120)</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sample_plan_name</td>
							<td>The name of the study design for a surveillance project.</td>
							<td>Provide the name of the sample plan used for sample collection. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>National Microbiological Baseline Study in Broiler Chicken</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sample_plan_ID</td>
							<td>The identifier of the study design for a surveillance project.</td>
							<td>Provide the identifier of the sample plan used for sample collection. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>2001_M205</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sample_collector_contact_name</td>
							<td>The name or job title of the contact responsible for follow-up regarding the sample.</td>
							<td>Provide the name of an individual or their job title. As personnel turnover may render the contact's name obsolete, it is more preferable to provide a job title for ensuring accuracy of information and institutional memory. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>Enterics Lab Manager</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sample_collector_contact_email</td>
							<td>The email address of the contact responsible for follow-up regarding the sample.</td>
							<td>Provide the email associated with the listed contact. As personnel turnover may render an individual's email obsolete, it is more preferable to provide an address for a position or lab, to ensure accuracy of information and institutional memory. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>johnnyblogs@lab.ca</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">purpose_of_sampling</td>
							<td>The purpose of sample collection.</td>
							<td>Provide the reason for sample collection by selecting a value from the following pick list: Diagnostic testing, Surveillance, Monitoring, Clinical trial, Field experiment, Environmental testing. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>Surveillance</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">experimental_activity</td>
							<td>The experimental activities or variables that affected the sample collected.</td>
							<td>If there was experimental activity that would affect the sample prior to collection (this is different than sample processing), provide the experimental activities by selecting one or more values from the template pick list. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>Antimicrobial Pre-treatment</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">experimental_activity_details</td>
							<td>The details of the experimental activities or variables that affected the sample collected.</td>
							<td>Briefly describe the experimental details using free text.</td>
							<td>Chicken feed containing X amount of novobiocin was fed to chickens for 72 hours prior to collection of litter.</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sample_processing</td>
							<td>The processing applied to samples post-collection, prior to further testing, characterization, or isolation procedures.</td>
							<td>Provide the sample processing information by selecting a value from the template pick list. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>Samples pooled</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">geo_loc_name (country)</td>
							<td>The country where the sample was collected.</td>
							<td>Provide the name of the country where the sample was collected. Use the controlled vocabulary provided in the template pick list.  If the information is unknown or cannot be provided, provide a null value.</td>
							<td>Canada</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">geo_loc_name (state/province/region)</td>
							<td>The province/territory where the sample was collected.</td>
							<td>Provide the name of the province/state/region where the sample was collected.  If the information is unknown or cannot be provided, provide a null value.</td>
							<td>British Columbia</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">food_product_origin geo_loc_name (country)</td>
							<td>The country of origin of a food product.</td>
							<td>If a food product was sampled and the food product was manufactured outside of Canada, provide the name of the country where the food product originated by selecting a value from the template pick list. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>United States of America</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">host_origin geo_loc_name (country)</td>
							<td>The country of origin of the host.</td>
							<td>If a sample is from a human or animal host that originated from outside of Canada, provide the the name of the country where the host originated by selecting a value from the template pick list. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>South Africa</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">latitude_of_sample_collection</td>
							<td>The latitude coordinates of the geographical location of sample collection.</td>
							<td>If known, provide the degrees latitude.  Do NOT simply provide latitude of the institution if this is not where the sample was collected, nor the centre of the city/region where the sample was collected as this falsely implicates an existing geographical location and creates data inaccuracies. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>38.98 N</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">longitude_of_sample_collection</td>
							<td>The longitude coordinates of the geographical location of sample collection.</td>
							<td>If known, provide the degrees longitude.  Do NOT simply provide longitude of the institution if this is not where the sample was collected, nor the centre of the city/region where the sample was collected as this falsely implicates an existing geographical location and creates data inaccuracies. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>77.11 W</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sample_collection_date</td>
							<td>The date on which the sample was collected.</td>
							<td>Provide the date according to the ISO 8601 standard "YYYY-MM-DD", "YYYY-MM" or "YYYY".</td>
							<td>2020-10-30</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sample_received_date</td>
							<td>The date on which the sample was received by the laboratory.</td>
							<td>Provide the date according to the ISO 8601 standard "YYYY-MM-DD", "YYYY-MM" or "YYYY".</td>
							<td>2020-11-15</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">original_sample_description</td>
							<td>The original sample description provided by the sample collector.</td>
							<td>Provide the sample description provided by the original sample collector. The original description is useful as it may provide further details, or can be used to clarify higher level classifications.</td>
							<td>RTE Prosciutto from deli</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">environmental_site</td>
							<td>An environmental site is a location in the natural or built environment.</td>
							<td>If applicable, provide the standardized term and ontology ID for the environmental site. The standardized term can be sourced from these look-up services: https://www.ebi.ac.uk/ols/ontologies/envo or https://www.ebi.ac.uk/ols/ontologies/genepio.</td>
							<td>contact surface, production facility, floor drain, hatchery.</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">animal_or_plant_population</td>
							<td>The type of animal or plant population inhabiting an area.</td>
							<td>This field should be used when a sample is taken from an environmental location inhabited by many individuals of a specific type, rather than describing a sample taken from one particular host. If applicable, provide the standardized term and ontology ID for the animal or plant population name. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/genepio. If not applicable, leave blank.</td>
							<td>turkey</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">environmental_material</td>
							<td>A substance or object obtained from the natural or man-made environment .</td>
							<td>If applicable, provide the standardized term and ontology ID for the environmental material. The standardized term can be sourced from these look-up services: https://www.ebi.ac.uk/ols/ontologies/envo or https://www.ebi.ac.uk/ols/ontologies/genepio. If not applicable, leave blank.</td>
							<td>soil, water, sewage, tractor</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">body_product</td>
							<td>A substance excreted/secreted from an organism.</td>
							<td>If applicable, provide the standardized term and ontology ID for the body product. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/uberon. If not applicable, leave blank.</td>
							<td>feces, urine, sweat.</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">anatomical_part</td>
							<td>A substance obtained from an anatomical part of an organism.</td>
							<td>If applicable, provide the standardized term and ontology ID for the anatomical material. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/uberon. If not applicable, leave blank.</td>
							<td>tissue, blood</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">food_product</td>
							<td>A material consumed and digested for nutritional value or enjoyment.</td>
							<td>This field includes animal feed. If applicable, provide the standardized term and ontology ID for the anatomical material. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/foodon. If not applicable, leave blank.</td>
							<td>feather and bone meal, chicken breast</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">food_product_properties</td>
							<td>Any characteristic of the food product pertaining to its state, processing, a label claim, or implications for consumers.</td>
							<td>Provide any characteristics of the food product including whether it has been cooked, processed, preserved, any known information about its state (e.g. raw, ready-to-eat), any known information about its containment (e.g. canned), and any information about a label claim (e.g. organic, fat-free).</td>
							<td>smoked, chopped, ready-to-eat</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">animal_source_of_food</td>
							<td>The animal from which the food product was derived.</td>
							<td>Provide the common name of the animal. If not applicable, leave blank. Multiple entries can be provided, separated by a comma.</td>
							<td>chicken</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">food_packaging</td>
							<td>The type of packaging used to contain a food product.</td>
							<td>If known, provide information regarding how the food product was packaged.</td>
							<td>box, tray</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">collection_device</td>
							<td>The instrument or container used to collect the sample.</td>
							<td>This field includes animal feed. If applicable, provide the standardized term and ontology ID for the anatomical material. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/genepio. If not applicable, leave blank.</td>
							<td>swab</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">collection_method</td>
							<td>The process used to collect the sample.</td>
							<td>If applicable, provide the standardized term and ontology ID for the anatomical material. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/genepio. If not applicable, leave blank.</td>
							<td>phlebotomy, necropsy, rinsing</td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Host information</h3></td>
						</tr>
						
						<tr>
							<td class="label">host (common name)</td>
							<td>The commonly used name of the host.</td>
							<td>If the sample is directly from a host, either a common or scientific name must be provided (although both can be included, if known).  If known, provide the common name.</td>
							<td>cow, chicken, human</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">host (scientific name)</td>
							<td>The scientific name of the host.</td>
							<td>If the sample is directly from a host, either a common or scientific name must be provided (although both can be included, if known).  If known, provide the scientific name. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/ncbitaxon.</td>
							<td>Bos taurus, Homo sapiens</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">host_disease</td>
							<td>The name of the disease experienced by the host.</td>
							<td>This field is only required if the Pathogen.cl package was selected. If the host was sick, provide the name of the disease.The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/doid If the disease is not known, put “missing”.</td>
							<td>mastitis, gastroenteritis</td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Strain and isolation information</h3></td>
						</tr>
						
						<tr>
							<td class="label">microbiological_method</td>
							<td>The laboratory method used to grow, prepare, and/or isolate the microbial isolate.</td>
							<td>Provide the name and version number of the microbiological method. The ID of the method is also acceptable if the ID can be linked to the laboratory that created the procedure.</td>
							<td>MFHPB-30</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">strain</td>
							<td>The strain identifier.</td>
							<td>If the isolate represents or is derived from, a lab reference strain or strain from a type culture collection, provide the strain identifier.</td>
							<td>K12</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">isolate_ID</td>
							<td>The user-defined identifier for the isolate, as provided by the laboratory that originally isolated the isolate.</td>
							<td>Provide the isolate_ID created by the lab that first isolated the isolate (i.e. the original isolate ID). If the information is unknown or cannot be provided, leave blank or provide a null value. If only an alternate isolate ID is known (e.g. the ID from your lab, if your lab did not isolate the isolate from the original sample), make asure to include it in the alternative_isolate_ID field.</td>
							<td>SA20131043</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">alternative_isolate_ID</td>
							<td>An alternative isolate_ID assigned to the isolate by another organization.</td>
							<td>If one or more alternative isolate_IDs are available, record them all here, separated by a comma.</td>
							<td>PHAC_45678</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">progeny_isolate_ID</td>
							<td>The identifier assigned to a progenitor isolate derived from an isolate that was directly obtained from a sample.</td>
							<td>If your sequence data pertains to progeny of an original isolate, provide the progeny_isolate_ID.</td>
							<td>SUB_ON_1526</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">IRIDA_isolate_ID</td>
							<td>The identifier of the isolate in the IRIDA platform.</td>
							<td>Provide the "sample ID" used to track information linked to the isolate in IRIDA. IRIDA sample IDs should be unqiue to avoid ID clash. This is very important in large Projects, especially when samples are shared from different organizations. Download the IRIDA sample ID and add it to the sample data in your spreadsheet as part of good data management practices.</td>
							<td>GRDI_LL_12345</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">IRIDA_project_ID</td>
							<td>The identifier of the Project in the iRIDA platform.</td>
							<td>Provide the IRIDA "project ID".</td>
							<td>666</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">isolated_by_institution_name</td>
							<td>The name of the agency, organization or institution with which the individual who performed the isolation procedure is affiliated.</td>
							<td>Provide the name of the agency, organization or institution that isolated the original isolate in full (avoid abbreviations). If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>Public Health Agency of Canada</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">isolated_by_laboratory_name</td>
							<td>The specific laboratory affiliation of the individual who performed the isolation procedure.</td>
							<td>Provide the name of the specific laboratory that that isolated the original isolate (avoid abbreviations). If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>Topp Lab</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">isolated_by_contact_name</td>
							<td>The name or title of the contact responsible for follow-up regarding the isolate.</td>
							<td>Provide the name of an individual or their job title. As personnel turnover may render the contact's name obsolete, it is prefereable to provide a job title for ensuring accuracy of information and institutional memory. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>Enterics Lab Manager</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">isolated_by_contact_email</td>
							<td>The email address of the contact responsible for follow-up regarding the isolate.</td>
							<td>Provide the email associated with the listed contact. As personnel turnover may render an individual's email obsolete, it is more prefereable to provide an address for a position or lab, to ensure accuracy of information and institutional memory. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>enterics@lab.ca</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">isolation_date</td>
							<td>The date on which the isolate was isolated from a sample.</td>
							<td>Provide the date according to the ISO 8601 standard "YYYY-MM-DD", "YYYY-MM" or "YYYY".</td>
							<td>2020-10-30</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">isolate_received_date</td>
							<td>The date on which the isolate was received by the laboratory.</td>
							<td>Provide the date according to the ISO 8601 standard "YYYY-MM-DD", "YYYY-MM" or "YYYY".</td>
							<td>2020-11-15</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">organism</td>
							<td>The taxonomic name of the organism.</td>
							<td>Put the genus and species (and subspecies if applicable) of the bacteria, if known. The standardized term can be sourced from this look-up service: https://www.ebi.ac.uk/ols/ontologies/ncbitaxon.</td>
							<td>Salmonella enterica subspecies enterica</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">serovar</td>
							<td>The serovar of the organism.</td>
							<td>Only include this information if it has been determined by traditional serological methods or a validated in silico prediction tool e.g. SISTR.</td>
							<td>Heidelberg</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">serotyping_method</td>
							<td>The method used to determine the serovar.</td>
							<td>If the serovar was determined via traditional serotyping methods, put “Traditional serotyping”. If the serovar was determined via in silico methods, provide the name and version number of the software.</td>
							<td>SISTR 1.0.1</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">phagetype</td>
							<td>The phagetype of the organism.</td>
							<td>Provide if known. If unknown, put “missing”.</td>
							<td>47</td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Sequence information</h3></td>
						</tr>
						
						<tr>
							<td class="label">library_ID</td>
							<td>The identifier of the library sequenced.</td>
							<td>Every "library ID" from a single submitter must be unique. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible.</td>
							<td>LS_2010_NP_123446</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sequenced_by_institution_name</td>
							<td>The name of the agency, organization or institution responsible for sequencing the isolate's genome.</td>
							<td>Provide the name of the agency, organization or institution that performed the sequencing in full (avoid abbreviations). If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>Public Health Agency of Canada</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sequenced_by_laboratory_name</td>
							<td>The specific laboratory affiliation of the responsible for sequencing the isolate's genome.</td>
							<td>Provide the name of the specific laboratory that that performed the sequencing in full (avoid abbreviations). If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>Topp Lab</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sequenced_by_contact_name</td>
							<td>The name or title of the contact responsible for follow-up regarding the sequence.</td>
							<td>Provide the name of an individual or their job title. As personnel turnover may render the contact's name obsolete, it is more prefereable to provide a job title for ensuring accuracy of information and institutional memory. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>Enterics Lab Manager</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sequenced_by_contact_email</td>
							<td>The email address of the contact responsible for follow-up regarding the sequence.</td>
							<td>Provide the email associated with the listed contact. As personnel turnover may render an individual's email obsolete, it is more prefereable to provide an address for a position or lab, to ensure accuracy of information and institutional memory. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>enterics@lab.ca</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">purpose_of_sequencing</td>
							<td>The purpose of sequencing.</td>
							<td>Provide the reason for sequencing by selecting a value from the following pick list: Diagnostic testing, Surveillance, Monitoring, Clinical trial, Field experiment, Environmental testing. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>Research</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sequencing_project_name</td>
							<td>The name of the project/initiative/program for which sequencing was performed.</td>
							<td>Provide the name of the project and/or the project ID here. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>AMR-GRDI (PA-1356)</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sequencing_platform</td>
							<td>The platform technology used to perform the sequencing.</td>
							<td>Provide the name of the company that created the sequencing instrument by selecting a value from the template pick list. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>Illumina</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sequencing_instrument</td>
							<td>The instrument type used to perform the sequencing.</td>
							<td>Provide the model sequencing instrument by selecting a value from the template pick list. If the information is unknown or cannot be provided, leave blank or provide a null value.</td>
							<td>MiSeq</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sequencing_method</td>
							<td>The protocol or method used for sequencing.</td>
							<td>Provide the name and version of the procedure or protocol used for sequencing. You can also provide a link to a protocol online.</td>
							<td>https://www.protocols.io/view/ncov-2019-sequencing-protocol-bbmuik6w?version_warning=no</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">r1_fastq_filename</td>
							<td>The user-specified filename of the r1 FASTQ file.</td>
							<td>Provide the r1 FASTQ filename.</td>
							<td>ABC123_S1_L001_R1_001.fastq.gz</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">r1_fastq_filename</td>
							<td>The user-specified filename of the r2 FASTQ file.</td>
							<td>Provide the r2 FASTQ filename.</td>
							<td>ABC123_S1_L001_R2_001.fastq.gz</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">fast5_filename</td>
							<td>The user-specified filename of the FAST5 file.</td>
							<td>Provide the FAST5 filename.</td>
							<td>batch1a_sequences.fast5</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">assembly_filename</td>
							<td>The user-defined filename of the FASTA file.</td>
							<td>Provide the FASTA filename.</td>
							<td>pathogenassembly123.fasta</td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Public repository information</h3></td>
						</tr>
						
						<tr>
							<td class="label">publication_ID</td>
							<td>The identifier for a publication.</td>
							<td>If the isolate is associated with a published work which can provide additional information, provide the PubMed identifier of the publication. Other types of identifiers (e.g. DOI) are also acceptable.</td>
							<td>PMID: 33205991</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">attribute_package</td>
							<td>The attribute package used to structure metadata in an INSDC BioSample.</td>
							<td>If the sample is from a specific human or animal, put “Pathogen.cl”. If the sample is from an environmental sample including food, feed, production facility, farm, water source, manure etc, put “Pathogen.env”.</td>
							<td>Pathogen.env</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">biosample_accession</td>
							<td>The identifier assigned to a BioSample in INSDC archives.</td>
							<td>Store the accession returned from the BioSample submission. NCBI BioSamples will have the prefix SAMN, whileEMBL- EBI BioSamples will have the prefix SAMEA.</td>
							<td>SAMN14180202</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">SRA_accession</td>
							<td>The Sequence Read Archive (SRA), European Nucleotide Archive (ENA) or DDBJ Sequence Read Archive (DRA) identifier linking raw read data, methodological metadata and quality control metrics submitted to the INSDC.</td>
							<td>Store the accession assigned to the submitted "run". NCBI-SRA accessions start with SRR, EBI-ENA runs start with ERR and DRA accessions start with DRR.</td>
							<td>SRR11177792</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">GenBank_accession</td>
							<td>The GenBank/ENA/DDBJ identifier assigned to the sequence in the INSDC archives.</td>
							<td>Store the accession returned from a GenBank/ENA/DDBJ submission.</td>
							<td>MN908947.3</td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Risk assessment information</h3></td>
						</tr>
						
						<tr>
							<td class="label">prevalence_metrics</td>
							<td>Metrics regarding the prevalence of the pathogen of interest obtained from a surveillance project.</td>
							<td>Risk assessment requires detailed information regarding the quantities of a pathogen in a specified location, commodity, or environment. As such, it is useful for risk assessors to know what types of information are available through documented methods and results. Provide the metric types that are available in the surveillance project sample plan by selecting them from the pick list. The metrics of interest are " Number of total samples collected", "Number of positive samples", "Average count of hazard organism", "Average count of indicator organism". You do not need to provide the actual values, just indicate that the information is available.</td>
							<td>Number of total samples collected, Number of positive samples</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">prevalence_metrics_details</td>
							<td>The details pertaining to the prevalence metrics from a surveillance project.</td>
							<td>If there are details pertaining to samples or organism counts in the sample plan that might be informative, provide details using free text.</td>
							<td>Hazard organism counts (i.e. Salmonella) do not distinguish between serovars.</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">stage_of_production</td>
							<td>The stage of food production.</td>
							<td>Provide the stage of food production from which the sample was taken. Select a value form the pick list provided.</td>
							<td>abattoir</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">experimental_intervention</td>
							<td>The category of the experimental intervention applied in the food production system.</td>
							<td>In some surveys, a particular intervention in the food supply chain in studied. If there was an intervention specified in the sample plan, select the intervention category from the pick list provided.</td>
							<td>Vaccination</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">experiment_intervention_details</td>
							<td>The details of the experimental intervention applied in the food production system.</td>
							<td>If an experimental intervention was applied in the survey, provide details in this field as free text.</td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr class="section">
							<td colspan="5"><h3>Antimicrobial resistance</h3></td>
						</tr>
						
						<tr>
							<td class="label">AMR_laboratory_typing_method</td>
							<td>Type of method used for antibiotic susceptibility testing.</td>
							<td>“MIC”, “agar dilution”, “disk diffusion”, or “missing”</td>
							<td>MIC</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">AMR_laboratory_typing_platform</td>
							<td>The brand/platform used for antibiotic susceptibility testing</td>
							<td>“Microscan”, “Phoenix”, “Sensititre”, or “Vitek”</td>
							<td>Sensititre</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">AMR_laboratory_typing_platform_version</td>
							<td>The specific name and version of the plate, panel, or other platform used for antibiotic susceptibility testing.</td>
							<td>Any additional information about the nature of the antimicrobial susceptibility test can go in this field.</td>
							<td>CMV3AGNF</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">AMR_measurement_units</td>
							<td>Units used for measuring antibiotic susceptibility</td>
							<td>“mg/L” or “mm”</td>
							<td>mg/L</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">amoxicillin-clavulanic_acid_resistance_phenotype</td>
							<td>Antimicrobial resistance phenotype, as determined by the antibiotic susceptibility measurement and testing standard for this antibiotic</td>
							<td>“resistant”, “susceptible”, “intermediate”, “nonsusceptible”, “suceptible-dose dependent”, or “not defined”</td>
							<td>sensitive</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">amoxicillin-clavulanic_acid_measurement_sign</td>
							<td>Qualifier associated with the antibiotic susceptibility measurement</td>
							<td>“<”, “<=”, “==”, “>=”, or “>". If the susceptibility measurement for this antibiotic and for this sample is exact, use “==”.</td>
							<td>#ERROR!</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">amoxicillin-clavulanic_acid_measurement</td>
							<td>Antibiotic susceptibility measurement, measured in the units specified in the “AMR_measurement_units” field.</td>
							<td>This field should only contain a number (either an integer or a number with decimals).</td>
							<td>4</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ampicillin_resistance_phenotype</td>
							<td></td>
							<td></td>
							<td>CLSI</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ampicillin_measurement_sign</td>
							<td></td>
							<td></td>
							<td>M100</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ampicillin_measurement</td>
							<td></td>
							<td></td>
							<td>27th ed. Wayne, PA: Clinical and Laboratory Standards Institute; 2017.</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">azithromycin_resistance_phenotype</td>
							<td></td>
							<td></td>
							<td>8</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">azithromycin_measurement_sign</td>
							<td></td>
							<td></td>
							<td>16</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">azithromycin_measurement</td>
							<td></td>
							<td></td>
							<td>32</td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">cefoxitin_resistance_phenotype</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">cefoxitin_measurement_sign</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">cefoxitin_measurement</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ceftriaxone_resistance_phenotype</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ceftriaxone_measurement_sign</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ceftriaxone_measurement</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">chloramphenicol_resistance_phenotype</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">chloramphenicol_measurement_sign</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">chloramphenicol_measurement</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ciprofloxacin_resistance_phenotype</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ciprofloxacin_measurement_sign</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ciprofloxacin_measurement</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">gentamicin_resistance_phenotype</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">gentamicin_measurement_sign</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">gentamicin_measurement</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">meropenem_resistance_phenotype</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">meropenem_measurement_sign</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">meropenem_measurement</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">nalidixic_acid_resistance_phenotype</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">nalidixic_acid_measurement_sign</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">nalidixic_acid_measurement</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">streptomycin_resistance_phenotype</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">streptomycin_measurement_sign</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">streptomycin_measurement</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sulfisoxazole_resistance_phenotype</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sulfisoxazole_measurement_sign</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sulfisoxazole_measurement</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">tetracycline_resistance_phenotype</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">tetracycline_measurement_sign</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">tetracycline_measurement</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">amoxicillin-clavulanic_acid_testing_standard</td>
							<td>Testing standard used for determination of resistance phenotype</td>
							<td>“CLSI”, “NARMS”, “BSAC”, “DIN”, “EUCAST”, “NCCLS”, “SFM”, “SIR”, “WRG”, or “missing”</td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">amoxicillin-clavulanic_acid_testing_standard_version</td>
							<td>Version number associated with the testing standard used for determination of resistance phenotype</td>
							<td>If applicable, include a version number for the testing standard used.</td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">amoxicillin-clavulanic_acid_testing_standard_details</td>
							<td>Additional details associated with the testing standard used for determination of resistance phenotype</td>
							<td>This information may include the year or location where the testing standard was published. If not applicable, leave blank.</td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">amoxicillin-clavulanic_acid_susceptible_breakpoint</td>
							<td>Maximum measurement, in the units specified in the “AMR_measurement_units” field, for a sample to be considered “sensitive” to this antibiotic</td>
							<td>This field should only contain a number (either an integer or a number with decimals), since the “<=” qualifier is implied.</td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">amoxicillin-clavulanic_acid_intermediate_breakpoint</td>
							<td>Intermediate measurement(s), in the units specified in the “AMR_measurement_units” field, where a sample would be considered to have an “intermediate” phenotype for this antibiotic</td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">amoxicillin-clavulanic_acid_resistant_breakpoint</td>
							<td>Minimum measurement, in the units specified in the “AMR_measurement_units” field, for a sample to be considered “resistant” to this antibiotic</td>
							<td>This field should only contain a number (either an integer or a number with decimals), since the “>=” qualifier is implied.</td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ampicillin_testing_standard</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ampicillin_testing_standard_version</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ampicillin_testing_standard_details</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ampicillin_susceptible_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ampicillin_intermediate_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ampicillin_resistant_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">azithromycin_testing_standard</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">azithromycin_testing_standard_version</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">azithromycin_testing_standard_details</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">azithromycin_susceptible_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">azithromycin_intermediate_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">azithromycin_resistant_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">cefoxitin_testing_standard</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">cefoxitin_testing_standard_version</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">cefoxitin_testing_standard_details</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">cefoxitin_susceptible_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">cefoxitin_intermediate_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">cefoxitin_resistant_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ceftriaxone_testing_standard</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ceftriaxone_testing_standard_version</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ceftriaxone_testing_standard_details</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ceftriaxone_susceptible_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ceftriaxone_intermediate_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ceftriaxone_resistant_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">chloramphenicol_testing_standard</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">chloramphenicol_testing_standard_version</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">chloramphenicol_testing_standard_details</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">chloramphenicol_susceptible_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">chloramphenicol_intermediate_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">chloramphenicol_resistant_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ciprofloxacin_testing_standard</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ciprofloxacin_testing_standard_version</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ciprofloxacin_testing_standard_details</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ciprofloxacin_susceptible_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ciprofloxacin_intermediate_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">ciprofloxacin_resistant_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">gentamicin_testing_standard</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">gentamicin_testing_standard_version</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">gentamicin_testing_standard_details</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">gentamicin_susceptible_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">gentamicin_intermediate_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">gentamicin_resistant_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">meropenem_testing_standard</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">meropenem_testing_standard_version</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">meropenem_testing_standard_details</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">meropenem_susceptible_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">meropenem_intermediate_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">meropenem_resistant_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">nalidixic_acid_testing_standard</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">nalidixic_acid_testing_standard_version</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">nalidixic_acid_testing_standard_details</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">nalidixic_acid_susceptible_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">nalidixic_acid_intermediate_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">nalidixic_acid_resistant_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">streptomycin_testing_standard</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">streptomycin_testing_standard_version</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">streptomycin_testing_standard_details</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">streptomycin_susceptible_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">streptomycin_intermediate_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">streptomycin_resistant_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sulfisoxazole_testing_standard</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sulfisoxazole_testing_standard_version</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sulfisoxazole_testing_standard_details</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sulfisoxazole_susceptible_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sulfisoxazole_intermediate_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">sulfisoxazole_resistant_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">tetracycline_testing_standard</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">tetracycline_testing_standard_version</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">tetracycline_testing_standard_details</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">tetracycline_susceptible_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">tetracycline_intermediate_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						
						<tr>
							<td class="label">tetracycline_resistant_breakpoint</td>
							<td></td>
							<td></td>
							<td></td>
							<td>None</td>
						</tr>

						</table>

		</tbody>
	</table>
</body>
</html>